--- GrayDB
+++ 共産党公式
@@ -8,28 +8,27 @@
     ns1:facebook "https://www.facebook.com/profile.php?id=100006085722650" ;
     ns1:twitter "chiduko916" ;
     ns1:せい "たかはし" ;
-    ns1:めい "ちづこ" ;
-    ns1:候補名 "高橋千鶴子" ;
-    ns1:公式サイト "http://www.chiduko.gr.jp/" ;
+    ns1:めい "ちづ" ;
+    ns1:候補名 "高橋ちづ子" ;
     ns1:前回 "前" ;
-    ns1:名 "千鶴子" ;
-    ns1:名前 "高橋千鶴子" ;
+    ns1:名 "子千鶴子" ;
+    ns1:名前 "高橋子千鶴子" ;
     ns1:姓 "高橋" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "58" ;
+    ns1:年齢 "57" ;
     ns1:比例区 "東北" .
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/profile.php?id=100014876527777" ;
     ns1:twitter "nisida_tosiko" ;
-    ns1:せい "にしだ" ;
-    ns1:めい "としこ" ;
-    ns1:候補名 "西田敏子" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "敏子" ;
-    ns1:名前 "西田敏子" ;
-    ns1:姓 "西田" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
+    ns1:候補名 "西田とし子" ;
+    ns1:公式サイト "http://jcpweb.net/nishida/" ;
+    ns1:前回 "新" ;
+    ns1:名 "" ;
+    ns1:名前 "西田とし子" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "愛知4区" ;
     ns1:年齢 "63" ;
     ns1:比例区 "" .
@@ -37,16 +36,15 @@
 [] ns1:blog "" ;
     ns1:facebook "" ;
     ns1:twitter "" ;
-    ns1:せい "こみやま" ;
-    ns1:めい "えみこ" ;
-    ns1:候補名 "籠山恵美子" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "恵美子" ;
-    ns1:名前 "籠山恵美子" ;
-    ns1:姓 "籠山" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
+    ns1:候補名 "こみやま恵美子" ;
+    ns1:前回 "新" ;
+    ns1:名 "" ;
+    ns1:名前 "こみやま恵美子" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "岐阜4区" ;
-    ns1:年齢 "63" ;
+    ns1:年齢 "62" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
@@ -55,13 +53,12 @@
     ns1:せい "しばやま" ;
     ns1:めい "のぼる" ;
     ns1:候補名 "しば山のぼる" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "昇" ;
-    ns1:名前 "柴山昇" ;
-    ns1:姓 "柴山" ;
+    ns1:名前 "山柴昇" ;
+    ns1:姓 "山柴" ;
     ns1:小選挙区 "大阪1区" ;
-    ns1:年齢 "64" ;
+    ns1:年齢 "63" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
@@ -69,42 +66,42 @@
     ns1:twitter "sima_omi" ;
     ns1:せい "しまさき" ;
     ns1:めい "やすおみ" ;
-    ns1:候補名 "島崎保臣" ;
+    ns1:候補名 "島﨑やすおみ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "保臣" ;
-    ns1:名前 "島崎保臣" ;
-    ns1:姓 "島崎" ;
+    ns1:名前 "島﨑保臣" ;
+    ns1:姓 "島﨑" ;
     ns1:小選挙区 "" ;
     ns1:年齢 "34" ;
     ns1:比例区 "四国" .
 
-[] ns1:blog "http://blog.canpan.info/tak1381" ;
-    ns1:facebook "https://www.facebook.com/profile.php?id=100004754730502" ;
-    ns1:twitter "tak1381" ;
-    ns1:せい "たかむら" ;
-    ns1:めい "なおや" ;
+[] ns1:blog "" ;
+    ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
     ns1:候補名 "たかむら直也" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
-    ns1:名 "直也" ;
-    ns1:名前 "高村直也" ;
-    ns1:姓 "高村" ;
+    ns1:名 "" ;
+    ns1:名前 "たかむら直也" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "宮城4区" ;
     ns1:年齢 "34" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://blog.goo.ne.jp/tame15" ;
+[] ns1:blog "" ;
     ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "ため" ;
     ns1:めい "きみひと" ;
     ns1:候補名 "ため仁史" ;
-    ns1:公式サイト "" ;
+    ns1:公式サイト "http://blog.goo.ne.jp/tame15" ;
     ns1:前回 "新" ;
     ns1:名 "仁史" ;
-    ns1:名前 "為仁史" ;
-    ns1:姓 "為" ;
+    ns1:名前 "爲仁史" ;
+    ns1:姓 "爲" ;
     ns1:小選挙区 "大阪15区" ;
     ns1:年齢 "68" ;
     ns1:比例区 "" .
@@ -112,14 +109,14 @@
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/tagire.power.of.the.fish" ;
     ns1:twitter "jkabucp" ;
-    ns1:せい "のざわ" ;
-    ns1:めい "やすゆき" ;
-    ns1:候補名 "野沢康幸" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
+    ns1:候補名 "のざわ康幸" ;
     ns1:公式サイト "http://jcpweb.net/nozawa/" ;
     ns1:前回 "新" ;
-    ns1:名 "康幸" ;
-    ns1:名前 "野沢康幸" ;
-    ns1:姓 "野沢" ;
+    ns1:名 "" ;
+    ns1:名前 "のざわ康幸" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "愛知15区" ;
     ns1:年齢 "29" ;
     ns1:比例区 "" .
@@ -129,14 +126,44 @@
     ns1:twitter "" ;
     ns1:せい "のざわ" ;
     ns1:めい "まさじ" ;
-    ns1:候補名 "野沢正司" ;
+    ns1:候補名 "のざわ正司" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "正司" ;
-    ns1:名前 "野沢正司" ;
-    ns1:姓 "野沢" ;
+    ns1:名前 "野澤正司" ;
+    ns1:姓 "野澤" ;
     ns1:小選挙区 "静岡7区" ;
     ns1:年齢 "68" ;
+    ns1:比例区 "" .
+
+[] ns1:blog "" ;
+    ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:せい "テルヤ" ;
+    ns1:めい "かんとく" ;
+    ns1:候補名 "テルヤ寛徳" ;
+    ns1:公式サイト "http://www.terukan.com/" ;
+    ns1:前回 "前" ;
+    ns1:名 "寛徳" ;
+    ns1:名前 "照屋寛徳" ;
+    ns1:姓 "照屋" ;
+    ns1:小選挙区 "沖縄2区(推薦、オール沖縄)" ;
+    ns1:年齢 "72" ;
+    ns1:比例区 "" .
+
+[] ns1:blog "" ;
+    ns1:facebook "" ;
+    ns1:twitter "nakasatookinawa" ;
+    ns1:せい "ナカザト" ;
+    ns1:めい "としのぶ" ;
+    ns1:候補名 "ナカザト利信" ;
+    ns1:公式サイト "http://nakasato.okinawa/" ;
+    ns1:前回 "前" ;
+    ns1:名 "利信" ;
+    ns1:名前 "仲里利信" ;
+    ns1:姓 "仲里" ;
+    ns1:小選挙区 "沖縄4区(推薦、オール沖縄)" ;
+    ns1:年齢 "80" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
@@ -154,13 +181,12 @@
     ns1:年齢 "67" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://shoot128.exblog.jp" ;
+[] ns1:blog "http://shoot128.exblog.jp/" ;
     ns1:facebook "https://www.facebook.com/tomoyuki.miwa3" ;
     ns1:twitter "1975_miwa" ;
     ns1:せい "みわ" ;
     ns1:めい "ともゆき" ;
     ns1:候補名 "みわ智之" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "智之" ;
     ns1:名前 "三和智之" ;
@@ -170,13 +196,12 @@
     ns1:比例区 "近畿" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/kamijyo.ryoichi",
-        "https://www.facebook.com/ryoichikamijo" ;
+    ns1:facebook "https://www.facebook.com/kamijyo.ryoichi" ;
     ns1:twitter "ryoichikamijo" ;
     ns1:せい "かみじょう" ;
     ns1:めい "りょういち" ;
     ns1:候補名 "かみじょう亮一" ;
-    ns1:公式サイト "http://ryoichikamijo.com/" ;
+    ns1:公式サイト "http://www.jcp-kyoto.jp/2017shuin/candidate06.html" ;
     ns1:前回 "新" ;
     ns1:名 "亮一" ;
     ns1:名前 "上條亮一" ;
@@ -190,22 +215,21 @@
     ns1:twitter "satiueda29" ;
     ns1:せい "うえだ" ;
     ns1:めい "さちこ" ;
-    ns1:候補名 "上田幸子" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "幸子" ;
-    ns1:名前 "上田幸子" ;
+    ns1:候補名 "上田さち子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子幸" ;
+    ns1:名前 "上田子幸" ;
     ns1:姓 "上田" ;
     ns1:小選挙区 "兵庫7区" ;
     ns1:年齢 "69" ;
     ns1:比例区 "" .
 
-[] ns1:blog "https://ameblo.jp/hirotsugu1216/" ;
-    ns1:facebook "https://www.facebook.com/ueno.hirotsugu" ;
+[] ns1:blog "" ;
+    ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "うえの" ;
     ns1:めい "ひろつぐ" ;
-    ns1:候補名 "上野洋嗣" ;
+    ns1:候補名 "上野ひろつぐ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "洋嗣" ;
@@ -221,7 +245,6 @@
     ns1:せい "しもじま" ;
     ns1:めい "りょういち" ;
     ns1:候補名 "下島良一" ;
-    ns1:公式サイト "http://jcpweb.net/shimojima/" ;
     ns1:前回 "新" ;
     ns1:名 "良一" ;
     ns1:名前 "下島良一" ;
@@ -232,10 +255,10 @@
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/shimomura.masahiro.54" ;
-    ns1:twitter "wvzurewxzk7lisw" ;
+    ns1:twitter "" ;
     ns1:せい "しもむら" ;
     ns1:めい "まさひろ" ;
-    ns1:候補名 "下村雅洋" ;
+    ns1:候補名 "下村まさひろ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "雅洋" ;
@@ -247,10 +270,10 @@
 
 [] ns1:blog "" ;
     ns1:facebook "" ;
-    ns1:twitter "ojskdc90t9hnbra" ;
+    ns1:twitter "" ;
     ns1:せい "なかいし" ;
     ns1:めい "ひとし" ;
-    ns1:候補名 "中石仁" ;
+    ns1:候補名 "中石ひとし" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "仁" ;
@@ -265,23 +288,21 @@
     ns1:twitter "" ;
     ns1:せい "なかの" ;
     ns1:めい "たけし" ;
-    ns1:候補名 "中野武史" ;
-    ns1:公式サイト "http://jcpweb.net/nakano/" ;
+    ns1:候補名 "中野たけし" ;
     ns1:前回 "新" ;
     ns1:名 "武史" ;
     ns1:名前 "中野武史" ;
     ns1:姓 "中野" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "43" ;
+    ns1:年齢 "42" ;
     ns1:比例区 "東海" .
 
-[] ns1:blog "http://jcpnakanowatari.blogspot.jp" ;
+[] ns1:blog "http://jcpnakanowatari.blogspot.jp/" ;
     ns1:facebook "https://www.facebook.com/jun.nakanowatari.9" ;
     ns1:twitter "" ;
     ns1:せい "なかのわたり" ;
     ns1:めい "じゅん" ;
-    ns1:候補名 "中野渡旬" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "中野渡じゅん" ;
     ns1:前回 "新" ;
     ns1:名 "旬" ;
     ns1:名前 "中野渡旬" ;
@@ -295,7 +316,7 @@
     ns1:twitter "" ;
     ns1:せい "くぼ" ;
     ns1:めい "たかゆき" ;
-    ns1:候補名 "久保孝之" ;
+    ns1:候補名 "久保たかゆき" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "孝之" ;
@@ -340,7 +361,7 @@
     ns1:twitter "inoue_9" ;
     ns1:せい "いのうえ" ;
     ns1:めい "たかし" ;
-    ns1:候補名 "井上宣" ;
+    ns1:候補名 "井上たかし" ;
     ns1:公式サイト "http://peaceforat.com/it/" ;
     ns1:前回 "新" ;
     ns1:名 "宣" ;
@@ -351,12 +372,12 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/inoueyoshiko122" ;
+    ns1:facebook "https://www.facebook.com/profile.php?id=100008607883601" ;
     ns1:twitter "inoueyoshiko122" ;
     ns1:せい "いのうえ" ;
     ns1:めい "よしこ" ;
     ns1:候補名 "井上良子" ;
-    ns1:公式サイト "http://jcp-nara.jugem.jp/?eid=1384" ;
+    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "良子" ;
     ns1:名前 "井上良子" ;
@@ -371,7 +392,6 @@
     ns1:せい "いのうえ" ;
     ns1:めい "さとし" ;
     ns1:候補名 "井上さとし" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "諭" ;
     ns1:名前 "井上諭" ;
@@ -411,7 +431,7 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/6kuayumi" ;
+    ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "いまむら" ;
     ns1:めい "あゆみ" ;
@@ -427,10 +447,10 @@
 
 [] ns1:blog "" ;
     ns1:facebook "" ;
-    ns1:twitter "itou_makoto1974" ;
+    ns1:twitter "" ;
     ns1:せい "いとう" ;
     ns1:めい "まこと" ;
-    ns1:候補名 "伊藤誠" ;
+    ns1:候補名 "いとう誠" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "誠" ;
@@ -445,7 +465,7 @@
     ns1:twitter "tatsuya_jcp" ;
     ns1:せい "いとう" ;
     ns1:めい "たつや" ;
-    ns1:候補名 "伊藤達也" ;
+    ns1:候補名 "伊藤たつや" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "達也" ;
@@ -457,11 +477,10 @@
 
 [] ns1:blog "http://sasakiakemi-taine.blogspot.jp/" ;
     ns1:facebook "https://www.facebook.com/akemi.sasaki.1656" ;
-    ns1:twitter "peace12wave" ;
+    ns1:twitter "" ;
     ns1:せい "ささき" ;
     ns1:めい "あけみ" ;
-    ns1:候補名 "佐々木明美" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "佐々木あけみ" ;
     ns1:前回 "新" ;
     ns1:名 "明美" ;
     ns1:名前 "佐々木明美" ;
@@ -505,7 +524,7 @@
     ns1:twitter "" ;
     ns1:せい "ふじたに" ;
     ns1:めい "かえこ" ;
-    ns1:候補名 "冨士谷香恵子" ;
+    ns1:候補名 "ふじたに香恵子" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "香恵子" ;
@@ -520,7 +539,7 @@
     ns1:twitter "" ;
     ns1:せい "とみおか" ;
     ns1:めい "あきら" ;
-    ns1:候補名 "冨岡昭" ;
+    ns1:候補名 "冨岡あきら" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "昭" ;
@@ -530,12 +549,12 @@
     ns1:年齢 "67" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://blog.goo.ne.jp/tomita-naoki" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/naoki.tomita.503" ;
     ns1:twitter "tomita_naoki" ;
     ns1:せい "とみた" ;
     ns1:めい "なおき" ;
-    ns1:候補名 "冨田直樹" ;
+    ns1:候補名 "冨田なおき" ;
     ns1:公式サイト "http://www.tomita-naoki.com/" ;
     ns1:前回 "新" ;
     ns1:名 "直樹" ;
@@ -546,12 +565,11 @@
     ns1:比例区 "東京" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/profile.php?id=100006856698727" ;
-    ns1:twitter "rikiriki1005",
-        "rikitomorenntai" ;
+    ns1:facebook "" ;
+    ns1:twitter "rikitomorenntai" ;
     ns1:せい "りきしげ" ;
     ns1:めい "ともゆき" ;
-    ns1:候補名 "力重智之" ;
+    ns1:候補名 "りきしげ智之" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "智之" ;
@@ -561,15 +579,12 @@
     ns1:年齢 "39" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://ameblo.jp/katohrika/",
-        "http://katohrika.blog.fc2.com" ;
-    ns1:facebook "https://www.facebook.com/profile.php?id=100010277866022",
-        "https://www.facebook.com/rikakatoh" ;
+[] ns1:blog "http://ameblo.jp/katohrika/" ;
+    ns1:facebook "https://www.facebook.com/profile.php?id=100010277866022" ;
     ns1:twitter "" ;
     ns1:せい "かとう" ;
-    ns1:めい "りか" ;
+    ns1:めい "リカ" ;
     ns1:候補名 "加藤リカ" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "リカ" ;
     ns1:名前 "加藤リカ" ;
@@ -579,8 +594,8 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/taichi.katou.7" ;
-    ns1:twitter "nonbiriyo" ;
+    ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:せい "かとう" ;
     ns1:めい "たいち" ;
     ns1:候補名 "加藤太一" ;
@@ -599,7 +614,6 @@
     ns1:せい "きたやま" ;
     ns1:めい "りょうぞう" ;
     ns1:候補名 "北山良三" ;
-    ns1:公式サイト "http://kunishige-toru.com/" ;
     ns1:前回 "新" ;
     ns1:名 "良三" ;
     ns1:名前 "北山良三" ;
@@ -608,14 +622,12 @@
     ns1:年齢 "65" ;
     ns1:比例区 "" .
 
-[] ns1:blog "https://ameblo.jp/miraini-kibou/" ;
-    ns1:facebook "https://www.facebook.com/kitamura.miki19",
-        "https://www.facebook.com/miki.t.kitamura" ;
+[] ns1:blog "http://ameblo.jp/miraini-kibou/" ;
+    ns1:facebook "https://www.facebook.com/miki.t.kitamura" ;
     ns1:twitter "0620_appletree" ;
     ns1:せい "きたむら" ;
     ns1:めい "みき" ;
     ns1:候補名 "北村みき" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "みき" ;
     ns1:名前 "北村みき" ;
@@ -626,11 +638,10 @@
 
 [] ns1:blog "http://kiihanto.exblog.jp/" ;
     ns1:facebook "https://www.facebook.com/profile.php?id=1209855673" ;
-    ns1:twitter "syogatani" ;
+    ns1:twitter "" ;
     ns1:せい "はら" ;
     ns1:めい "やすひさ" ;
-    ns1:候補名 "原矢寸久" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "原やすひさ" ;
     ns1:前回 "新" ;
     ns1:名 "矢寸久" ;
     ns1:名前 "原矢寸久" ;
@@ -645,7 +656,6 @@
     ns1:せい "はら" ;
     ns1:めい "じゅんこ" ;
     ns1:候補名 "原純子" ;
-    ns1:公式サイト "http://hara-junko.com/" ;
     ns1:前回 "新" ;
     ns1:名 "純子" ;
     ns1:名前 "原純子" ;
@@ -659,7 +669,7 @@
     ns1:twitter "" ;
     ns1:せい "ふるかわ" ;
     ns1:めい "よしのり" ;
-    ns1:候補名 "古川芳憲" ;
+    ns1:候補名 "古川よしのり" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "芳憲" ;
@@ -674,7 +684,7 @@
     ns1:twitter "" ;
     ns1:せい "ふるさわ" ;
     ns1:めい "よしゆき" ;
-    ns1:候補名 "古沢喜幸" ;
+    ns1:候補名 "古沢よしゆき" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "喜幸" ;
@@ -685,8 +695,7 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/5555yoshida",
-        "https://www.facebook.com/yoshida.go.1" ;
+    ns1:facebook "https://www.facebook.com/5555yoshida" ;
     ns1:twitter "gogoyoshidago" ;
     ns1:せい "よしだ" ;
     ns1:めい "ごう" ;
@@ -697,15 +706,15 @@
     ns1:名前 "吉田剛" ;
     ns1:姓 "吉田" ;
     ns1:小選挙区 "宮城3区" ;
-    ns1:年齢 "36" ;
+    ns1:年齢 "35" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/jcp.yosida.toshio" ;
-    ns1:twitter "yoshida_toshio" ;
+    ns1:twitter "" ;
     ns1:せい "よしだ" ;
     ns1:めい "としお" ;
-    ns1:候補名 "吉田年男" ;
+    ns1:候補名 "吉田としお" ;
     ns1:公式サイト "http://www.koto-minpo.jp/jcp-yoshidatoshio/" ;
     ns1:前回 "新" ;
     ns1:名 "年男" ;
@@ -721,7 +730,7 @@
     ns1:せい "よしだ" ;
     ns1:めい "こういち" ;
     ns1:候補名 "吉田幸一" ;
-    ns1:公式サイト "" ;
+    ns1:公式サイト "http://www.jcp-kyoto.jp/2017shuin/candidate04.html" ;
     ns1:前回 "新" ;
     ns1:名 "幸一" ;
     ns1:名前 "吉田幸一" ;
@@ -745,7 +754,7 @@
     ns1:年齢 "36" ;
     ns1:比例区 "東北" .
 
-[] ns1:blog "https://ameblo.jp/smn-mukose/" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/shinichi.mukose" ;
     ns1:twitter "mukose3" ;
     ns1:せい "むこせ" ;
@@ -761,11 +770,11 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/cyuubutiku" ;
+    ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "よつや" ;
     ns1:めい "めぐみ" ;
-    ns1:候補名 "四ツ谷恵" ;
+    ns1:候補名 "よつや恵" ;
     ns1:公式サイト "https://meg-yotsuya.jimdo.com/" ;
     ns1:前回 "新" ;
     ns1:名 "恵" ;
@@ -787,12 +796,11 @@
     ns1:名前 "国田睦" ;
     ns1:姓 "国田" ;
     ns1:小選挙区 "愛媛3区" ;
-    ns1:年齢 "66" ;
-    ns1:比例区 "" .
-
-[] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/chisaka.takuaki",
-        "https://www.facebook.com/takuaki.chisaka" ;
+    ns1:年齢 "65" ;
+    ns1:比例区 "" .
+
+[] ns1:blog "" ;
+    ns1:facebook "https://www.facebook.com/chisaka.takuaki" ;
     ns1:twitter "chitak21" ;
     ns1:せい "ちさか" ;
     ns1:めい "たくあき" ;
@@ -812,18 +820,16 @@
     ns1:せい "さかい" ;
     ns1:めい "わかこ" ;
     ns1:候補名 "坂井和歌子" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "和歌子" ;
     ns1:名前 "坂井和歌子" ;
     ns1:姓 "坂井" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "40" ;
+    ns1:年齢 "39" ;
     ns1:比例区 "東京" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/55hiroshi",
-        "https://www.facebook.com/hiskm" ;
+    ns1:facebook "https://www.facebook.com/55hiroshi" ;
     ns1:twitter "hi_sakamoto" ;
     ns1:せい "さかもと" ;
     ns1:めい "ひろし" ;
@@ -837,13 +843,12 @@
     ns1:年齢 "47" ;
     ns1:比例区 "北陸信越" .
 
-[] ns1:blog "http://kyomi-k.cocolog-nifty.com/" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/gu.jingmei" ;
     ns1:twitter "haruka_tubasa" ;
     ns1:せい "かきうち" ;
     ns1:めい "きょうみ" ;
     ns1:候補名 "垣内京美" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "京美" ;
     ns1:名前 "垣内京美" ;
@@ -858,13 +863,12 @@
     ns1:せい "ほりうち" ;
     ns1:めい "てるふみ" ;
     ns1:候補名 "堀内照文" ;
-    ns1:公式サイト "http://www.teru-horiuchi.com/" ;
     ns1:前回 "前" ;
     ns1:名 "照文" ;
     ns1:名前 "堀内照文" ;
     ns1:姓 "堀内" ;
     ns1:小選挙区 "兵庫8区" ;
-    ns1:年齢 "45" ;
+    ns1:年齢 "44" ;
     ns1:比例区 "近畿" .
 
 [] ns1:blog "" ;
@@ -872,7 +876,7 @@
     ns1:twitter "" ;
     ns1:せい "ほりえ" ;
     ns1:めい "かくじ" ;
-    ns1:候補名 "堀江鶴治" ;
+    ns1:候補名 "堀江かくじ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "鶴治" ;
@@ -884,10 +888,10 @@
 
 [] ns1:blog "" ;
     ns1:facebook "" ;
-    ns1:twitter "yuzuruhori_jcp" ;
+    ns1:twitter "" ;
     ns1:せい "ほり" ;
     ns1:めい "ゆずる" ;
-    ns1:候補名 "堀譲" ;
+    ns1:候補名 "堀ゆずる" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "譲" ;
@@ -902,7 +906,7 @@
     ns1:twitter "aozoranaru" ;
     ns1:せい "つかだ" ;
     ns1:めい "なるゆき" ;
-    ns1:候補名 "塚田成幸" ;
+    ns1:候補名 "塚田なるゆき" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "成幸" ;
@@ -918,18 +922,17 @@
     ns1:せい "しおかわ" ;
     ns1:めい "てつや" ;
     ns1:候補名 "塩川鉄也" ;
-    ns1:公式サイト "http://www.shiokawa-tetsuya.jp/" ;
     ns1:前回 "前" ;
     ns1:名 "鉄也" ;
     ns1:名前 "塩川鉄也" ;
     ns1:姓 "塩川" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "55" ;
+    ns1:年齢 "54" ;
     ns1:比例区 "北関東" .
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/reiko.ookubo.33" ;
-    ns1:twitter "rei500313" ;
+    ns1:twitter "" ;
     ns1:せい "おおくぼ" ;
     ns1:めい "れいこ" ;
     ns1:候補名 "大久保令子" ;
@@ -942,34 +945,32 @@
     ns1:年齢 "67" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://blog.livedoor.jp/oouchikumiko/" ;
+[] ns1:blog "" ;
     ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "おおうち" ;
     ns1:めい "くみこ" ;
-    ns1:候補名 "大内久美子" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "久美子" ;
-    ns1:名前 "大内久美子" ;
+    ns1:候補名 "大内くみ子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子久美" ;
+    ns1:名前 "大内子久美" ;
     ns1:姓 "大内" ;
     ns1:小選挙区 "茨城1区" ;
     ns1:年齢 "68" ;
     ns1:比例区 "北関東" .
 
-[] ns1:blog "http://yoshinobuu.exblog.jp/" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/ohirayoshinobu" ;
     ns1:twitter "tekuteku_diary" ;
     ns1:せい "おおひら" ;
     ns1:めい "よしのぶ" ;
-    ns1:候補名 "大平喜信" ;
-    ns1:公式サイト "http://www.yoshinobu-ohira.net/" ;
+    ns1:候補名 "大平よしのぶ" ;
     ns1:前回 "前" ;
     ns1:名 "喜信" ;
     ns1:名前 "大平喜信" ;
     ns1:姓 "大平" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "39" ;
+    ns1:年齢 "38" ;
     ns1:比例区 "中国" .
 
 [] ns1:blog "" ;
@@ -992,7 +993,7 @@
     ns1:twitter "" ;
     ns1:せい "おおすぎ" ;
     ns1:めい "てつお" ;
-    ns1:候補名 "大椙鉄夫" ;
+    ns1:候補名 "おおすぎ鉄夫" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "鉄夫" ;
@@ -1002,12 +1003,12 @@
     ns1:年齢 "69" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://kakushinhiroshima.hatenablog.com" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/osamu.onishi.739" ;
-    ns1:twitter "peaceosanishi" ;
+    ns1:twitter "" ;
     ns1:せい "おおにし" ;
-    ns1:めい "おさむ" ;
-    ns1:候補名 "大西理" ;
+    ns1:めい "オサム" ;
+    ns1:候補名 "大西オサム" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "理" ;
@@ -1017,12 +1018,12 @@
     ns1:年齢 "51" ;
     ns1:比例区 "" .
 
-[] ns1:blog "https://ameblo.jp/oonishiwataru/" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/wataame.hare" ;
     ns1:twitter "wataame_hare" ;
     ns1:せい "おおにし" ;
     ns1:めい "わたる" ;
-    ns1:候補名 "大西航" ;
+    ns1:候補名 "大西わたる" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "航" ;
@@ -1032,13 +1033,12 @@
     ns1:年齢 "36" ;
     ns1:比例区 "南関東" .
 
-[] ns1:blog "http://jcpoonuki.blogspot.jp" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/ohnukikiyofumi" ;
     ns1:twitter "kiyofumi_ohnuki" ;
     ns1:せい "おおぬき" ;
     ns1:めい "きよふみ" ;
     ns1:候補名 "おおぬき清文" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "清文" ;
     ns1:名前 "大貫清文" ;
@@ -1052,7 +1052,7 @@
     ns1:twitter "" ;
     ns1:せい "おおの" ;
     ns1:めい "ひろみつ" ;
-    ns1:候補名 "大野宙光" ;
+    ns1:候補名 "大野ひろみつ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "宙光" ;
@@ -1063,12 +1063,11 @@
     ns1:比例区 "東海" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/oono.takasi.jcp",
-        "https://www.facebook.com/takasi.oono" ;
+    ns1:facebook "" ;
     ns1:twitter "koredeiinoka" ;
     ns1:せい "おおの" ;
     ns1:めい "たかし" ;
-    ns1:候補名 "大野隆" ;
+    ns1:候補名 "大野たかし" ;
     ns1:公式サイト "http://korede.iinoka.net/" ;
     ns1:前回 "新" ;
     ns1:名 "隆" ;
@@ -1080,11 +1079,10 @@
 
 [] ns1:blog "" ;
     ns1:facebook "" ;
-    ns1:twitter "okumotonaomi",
-        "pekoandnaomi" ;
+    ns1:twitter "" ;
     ns1:せい "おくもと" ;
     ns1:めい "なおみ" ;
-    ns1:候補名 "奥本菜保巳" ;
+    ns1:候補名 "奥本なおみ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "菜保巳" ;
@@ -1099,7 +1097,7 @@
     ns1:twitter "genmiyauchi1982" ;
     ns1:せい "みやうち" ;
     ns1:めい "げん" ;
-    ns1:候補名 "宮内現" ;
+    ns1:候補名 "宮内げん" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "現" ;
@@ -1114,8 +1112,7 @@
     ns1:twitter "" ;
     ns1:せい "みやち" ;
     ns1:めい "いさお" ;
-    ns1:候補名 "宮地勲" ;
-    ns1:公式サイト "http://jcpweb.net/miyachi/" ;
+    ns1:候補名 "宮地いさお" ;
     ns1:前回 "新" ;
     ns1:名 "勲" ;
     ns1:名前 "宮地勲" ;
@@ -1128,12 +1125,12 @@
     ns1:facebook "https://www.facebook.com/tsuyako.miyazima" ;
     ns1:twitter "tsuyako1127" ;
     ns1:せい "みやじま" ;
-    ns1:めい "つやこ" ;
+    ns1:めい "つやつやこ" ;
     ns1:候補名 "宮嶋つや子" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
-    ns1:名 "つや子" ;
-    ns1:名前 "宮嶋つや子" ;
+    ns1:名 "子" ;
+    ns1:名前 "宮嶋子" ;
     ns1:姓 "宮嶋" ;
     ns1:小選挙区 "福岡8区" ;
     ns1:年齢 "66" ;
@@ -1144,14 +1141,13 @@
     ns1:twitter "" ;
     ns1:せい "みやもと" ;
     ns1:めい "たけし" ;
-    ns1:候補名 "宮本岳志" ;
-    ns1:公式サイト "http://www.miyamoto-net.net/" ;
+    ns1:候補名 "宮本たけし" ;
     ns1:前回 "前" ;
     ns1:名 "岳志" ;
     ns1:名前 "宮本岳志" ;
     ns1:姓 "宮本" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "57" ;
+    ns1:年齢 "56" ;
     ns1:比例区 "近畿" .
 
 [] ns1:blog "" ;
@@ -1160,7 +1156,6 @@
     ns1:せい "みやもと" ;
     ns1:めい "とおる" ;
     ns1:候補名 "宮本徹" ;
-    ns1:公式サイト "http://miyamototooru.info/" ;
     ns1:前回 "前" ;
     ns1:名 "徹" ;
     ns1:名前 "宮本徹" ;
@@ -1174,21 +1169,21 @@
     ns1:twitter "" ;
     ns1:せい "てらだ" ;
     ns1:めい "あきみつ" ;
-    ns1:候補名 "寺田明充" ;
+    ns1:候補名 "寺田あきみつ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "明充" ;
     ns1:名前 "寺田明充" ;
     ns1:姓 "寺田" ;
     ns1:小選挙区 "広島6区" ;
-    ns1:年齢 "66" ;
+    ns1:年齢 "65" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/azuma.kozutsumi" ;
     ns1:twitter "azuma_united_fc" ;
     ns1:せい "こづつみ" ;
-    ns1:めい "あずま" ;
+    ns1:めい "あづま" ;
     ns1:候補名 "小堤東" ;
     ns1:公式サイト "http://senkyo.jcp-tokyo.net/kozutsumi/" ;
     ns1:前回 "新" ;
@@ -1234,11 +1229,11 @@
     ns1:twitter "tokikobayashi" ;
     ns1:せい "こばやし" ;
     ns1:めい "ときこ" ;
-    ns1:候補名 "小林解子" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "解子" ;
-    ns1:名前 "小林解子" ;
+    ns1:候補名 "小林とき子" ;
+    ns1:公式サイト "" ;
+    ns1:前回 "新" ;
+    ns1:名 "子解" ;
+    ns1:名前 "小林子解" ;
     ns1:姓 "小林" ;
     ns1:小選挙区 "福岡6区" ;
     ns1:年齢 "37" ;
@@ -1246,11 +1241,10 @@
 
 [] ns1:blog "" ;
     ns1:facebook "" ;
-    ns1:twitter "koike_kazunori" ;
+    ns1:twitter "" ;
     ns1:せい "こいけ" ;
     ns1:めい "かずのり" ;
     ns1:候補名 "小池一徳" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "一徳" ;
     ns1:名前 "小池一徳" ;
@@ -1260,11 +1254,11 @@
     ns1:比例区 "北関東" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/profile.php?id=100008467158841" ;
+    ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "おのざと" ;
     ns1:めい "ていりょう" ;
-    ns1:候補名 "小野里定良" ;
+    ns1:候補名 "小野里ていりょう" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "定良" ;
@@ -1279,7 +1273,7 @@
     ns1:twitter "koganezawayuka" ;
     ns1:せい "こがねざわ" ;
     ns1:めい "ゆか" ;
-    ns1:候補名 "小金沢由佳" ;
+    ns1:候補名 "小金沢ゆか" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "由佳" ;
@@ -1309,11 +1303,11 @@
     ns1:twitter "" ;
     ns1:せい "おざき" ;
     ns1:めい "ひろこ" ;
-    ns1:候補名 "尾崎宏子" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "宏子" ;
-    ns1:名前 "尾崎宏子" ;
+    ns1:候補名 "尾崎ひろ子" ;
+    ns1:公式サイト "" ;
+    ns1:前回 "新" ;
+    ns1:名 "子宏" ;
+    ns1:名前 "尾崎子宏" ;
     ns1:姓 "尾崎" ;
     ns1:小選挙区 "岡山3区" ;
     ns1:年齢 "61" ;
@@ -1331,16 +1325,16 @@
     ns1:名前 "尾崎光" ;
     ns1:姓 "尾崎" ;
     ns1:小選挙区 "広島5区" ;
-    ns1:年齢 "65" ;
-    ns1:比例区 "" .
-
-[] ns1:blog "https://ameblo.jp/sobo-katamuki/" ;
+    ns1:年齢 "64" ;
+    ns1:比例区 "" .
+
+[] ns1:blog "" ;
     ns1:facebook "" ;
     ns1:twitter "kai02yamashita" ;
     ns1:せい "やました" ;
     ns1:めい "かい" ;
-    ns1:候補名 "山下魁" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "山下かい" ;
+    ns1:公式サイト "http://profile.ameba.jp/sobo-katamuki" ;
     ns1:前回 "新" ;
     ns1:名 "魁" ;
     ns1:名前 "山下魁" ;
@@ -1366,14 +1360,14 @@
 
 [] ns1:blog "" ;
     ns1:facebook "" ;
-    ns1:twitter "yuutakagoshima3" ;
+    ns1:twitter "" ;
     ns1:せい "やまぐち" ;
     ns1:めい "ゆうた" ;
     ns1:候補名 "山口ゆうた" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
-    ns1:名 "勇太" ;
-    ns1:名前 "山口勇太" ;
+    ns1:名 "ゆうた" ;
+    ns1:名前 "山口ゆうた" ;
     ns1:姓 "山口" ;
     ns1:小選挙区 "鹿児島3区" ;
     ns1:年齢 "26" ;
@@ -1384,14 +1378,14 @@
     ns1:twitter "" ;
     ns1:せい "やまざき" ;
     ns1:めい "としひこ" ;
-    ns1:候補名 "山崎寿彦" ;
+    ns1:候補名 "山崎としひこ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "寿彦" ;
     ns1:名前 "山崎寿彦" ;
     ns1:姓 "山崎" ;
     ns1:小選挙区 "栃木4区" ;
-    ns1:年齢 "65" ;
+    ns1:年齢 "64" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
@@ -1426,11 +1420,10 @@
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/jcp20141214okazaki" ;
-    ns1:twitter "info_okazaki" ;
+    ns1:twitter "" ;
     ns1:せい "おかざき" ;
     ns1:めい "ゆたか" ;
-    ns1:候補名 "岡崎裕" ;
-    ns1:公式サイト "http://jcpokazaki.com/" ;
+    ns1:候補名 "岡崎ゆたか" ;
     ns1:前回 "新" ;
     ns1:名 "裕" ;
     ns1:名前 "岡崎裕" ;
@@ -1444,7 +1437,7 @@
     ns1:twitter "" ;
     ns1:せい "いわもと" ;
     ns1:めい "こうじ" ;
-    ns1:候補名 "岩本康嗣" ;
+    ns1:候補名 "岩本こうじ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "康嗣" ;
@@ -1459,7 +1452,7 @@
     ns1:twitter "" ;
     ns1:せい "いわた" ;
     ns1:めい "たけし" ;
-    ns1:候補名 "岩田剛" ;
+    ns1:候補名 "岩田たけし" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "剛" ;
@@ -1474,8 +1467,7 @@
     ns1:twitter "kisskisi" ;
     ns1:せい "きし" ;
     ns1:めい "たけし" ;
-    ns1:候補名 "岸武志" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "岸たけし" ;
     ns1:前回 "新" ;
     ns1:名 "武志" ;
     ns1:名前 "岸武志" ;
@@ -1484,9 +1476,8 @@
     ns1:年齢 "46" ;
     ns1:比例区 "東京" .
 
-[] ns1:blog "http://senkyo.jcp-tokyo.net/kishi/blog/" ;
-    ns1:facebook "https://www.facebook.com/kishiyoshinobu",
-        "https://www.facebook.com/profile.php?id=100010161334611" ;
+[] ns1:blog "" ;
+    ns1:facebook "https://www.facebook.com/kishiyoshinobu" ;
     ns1:twitter "jcptokyo10" ;
     ns1:せい "きし" ;
     ns1:めい "よしのぶ" ;
@@ -1506,17 +1497,16 @@
     ns1:せい "しまづ" ;
     ns1:めい "ゆきひろ" ;
     ns1:候補名 "しまづ幸広" ;
-    ns1:公式サイト "http://shimazu-yukihiro.jp/" ;
     ns1:前回 "前" ;
     ns1:名 "幸広" ;
     ns1:名前 "島津幸広" ;
     ns1:姓 "島津" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "61" ;
+    ns1:年齢 "60" ;
     ns1:比例区 "東海" .
 
 [] ns1:blog "" ;
-    ns1:facebook "" ;
+    ns1:facebook "https://www.facebook.com/jcpshizu.seibu" ;
     ns1:twitter "" ;
     ns1:せい "しまだ" ;
     ns1:めい "はつえ" ;
@@ -1535,7 +1525,7 @@
     ns1:twitter "" ;
     ns1:せい "かわかみ" ;
     ns1:めい "ひとし" ;
-    ns1:候補名 "川上均" ;
+    ns1:候補名 "川上ひとし" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "均" ;
@@ -1545,27 +1535,27 @@
     ns1:年齢 "61" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://kawasakiatsuko.seesaa.net/" ;
-    ns1:facebook "https://www.facebook.com/profile.php?id=100005773051957" ;
+[] ns1:blog "" ;
+    ns1:facebook "" ;
     ns1:twitter "jyuuouumare" ;
     ns1:せい "かわさき" ;
     ns1:めい "あつこ" ;
-    ns1:候補名 "川崎篤子" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "篤子" ;
-    ns1:名前 "川崎篤子" ;
+    ns1:候補名 "川崎あつ子" ;
+    ns1:公式サイト "" ;
+    ns1:前回 "新" ;
+    ns1:名 "子篤" ;
+    ns1:名前 "川崎子篤" ;
     ns1:姓 "川崎" ;
     ns1:小選挙区 "茨城5区" ;
     ns1:年齢 "64" ;
     ns1:比例区 "" .
 
-[] ns1:blog "https://kawaberyuji.blogspot.jp/" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/profile.php?id=100005961273973" ;
     ns1:twitter "ryujikawabe" ;
     ns1:せい "かわべ" ;
     ns1:めい "りゅうじ" ;
-    ns1:候補名 "川部竜二" ;
+    ns1:候補名 "かわべ竜二" ;
     ns1:前回 "新" ;
     ns1:名 "竜二" ;
     ns1:名前 "川部竜二" ;
@@ -1579,7 +1569,7 @@
     ns1:twitter "" ;
     ns1:せい "たいら" ;
     ns1:めい "よしひこ" ;
-    ns1:候補名 "平善彦" ;
+    ns1:候補名 "たいら善彦" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "善彦" ;
@@ -1594,7 +1584,7 @@
     ns1:twitter "jcpsaitamananbu" ;
     ns1:せい "ひらかわ" ;
     ns1:めい "みちや" ;
-    ns1:候補名 "平川道也" ;
+    ns1:候補名 "平川みちや" ;
     ns1:公式サイト "http://www.cupolatown.net/" ;
     ns1:前回 "新" ;
     ns1:名 "道也" ;
@@ -1620,7 +1610,7 @@
     ns1:比例区 "近畿" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/profile.php?id=100015517326981" ;
+    ns1:facebook "" ;
     ns1:twitter "3bihv3x5oh9us09" ;
     ns1:せい "ひらばやし" ;
     ns1:めい "あきなり" ;
@@ -1634,16 +1624,15 @@
     ns1:年齢 "62" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://setuko2525.cocolog-nifty.com" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/setuko2525" ;
-    ns1:twitter "setuko2525de15" ;
+    ns1:twitter "" ;
     ns1:せい "たなはし" ;
     ns1:めい "せつこ" ;
-    ns1:候補名 "店橋世津子" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "世津子" ;
-    ns1:名前 "店橋世津子" ;
+    ns1:候補名 "たなはしせつ子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子世津" ;
+    ns1:名前 "店橋子世津" ;
     ns1:姓 "店橋" ;
     ns1:小選挙区 "群馬1区" ;
     ns1:年齢 "56" ;
@@ -1661,11 +1650,11 @@
     ns1:名前 "志位和夫" ;
     ns1:姓 "志位" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "63" ;
+    ns1:年齢 "62" ;
     ns1:比例区 "南関東" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/profile.php?id=100014155634094" ;
+    ns1:facebook "https://www.facebook.com/saichi.toguchi" ;
     ns1:twitter "" ;
     ns1:せい "とぐち" ;
     ns1:めい "さいち" ;
@@ -1684,7 +1673,7 @@
     ns1:twitter "" ;
     ns1:せい "ところ" ;
     ns1:めい "すすむ" ;
-    ns1:候補名 "所進" ;
+    ns1:候補名 "所すすむ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "進" ;
@@ -1700,7 +1689,6 @@
     ns1:せい "さいとう" ;
     ns1:めい "かずこ" ;
     ns1:候補名 "さいとう和子" ;
-    ns1:公式サイト "http://saitokazuko.jp/" ;
     ns1:前回 "前" ;
     ns1:名 "和子" ;
     ns1:名前 "斉藤和子" ;
@@ -1714,14 +1702,14 @@
     ns1:twitter "" ;
     ns1:せい "さいとう" ;
     ns1:めい "さちこ" ;
-    ns1:候補名 "斉藤幸子" ;
+    ns1:候補名 "さいとう幸子" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "幸子" ;
     ns1:名前 "斉藤幸子" ;
     ns1:姓 "斉藤" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "60" ;
+    ns1:年齢 "59" ;
     ns1:比例区 "近畿" .
 
 [] ns1:blog "" ;
@@ -1729,7 +1717,7 @@
     ns1:twitter "nodokajcp" ;
     ns1:せい "さいとう" ;
     ns1:めい "のどか" ;
-    ns1:候補名 "斉藤温" ;
+    ns1:候補名 "斉藤のどか" ;
     ns1:公式サイト "http://s-nodoka.net/" ;
     ns1:前回 "新" ;
     ns1:名 "温" ;
@@ -1740,12 +1728,11 @@
     ns1:比例区 "" .
 
 [] ns1:blog "http://overwash.blog7.fc2.com/" ;
-    ns1:facebook "https://www.facebook.com/sugio.arai" ;
+    ns1:facebook "" ;
     ns1:twitter "over_wash" ;
     ns1:せい "あらい" ;
     ns1:めい "すぎお" ;
     ns1:候補名 "新井杉生" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "杉生" ;
     ns1:名前 "新井杉生" ;
@@ -1771,10 +1758,10 @@
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/jcp.munakasu" ;
-    ns1:twitter "sindome7" ;
+    ns1:twitter "" ;
     ns1:せい "しんどめ" ;
     ns1:めい "きよたか" ;
-    ns1:候補名 "新留清隆" ;
+    ns1:候補名 "しんどめ清隆" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "清隆" ;
@@ -1787,14 +1774,14 @@
 [] ns1:blog "" ;
     ns1:facebook "" ;
     ns1:twitter "" ;
-    ns1:せい "ほしの" ;
-    ns1:めい "ふみお" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
     ns1:候補名 "星野文雄" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
-    ns1:名 "文雄" ;
+    ns1:名 "" ;
     ns1:名前 "星野文雄" ;
-    ns1:姓 "星野" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "茨城2区" ;
     ns1:年齢 "69" ;
     ns1:比例区 "" .
@@ -1804,7 +1791,7 @@
     ns1:twitter "jcpniiza" ;
     ns1:せい "あさか" ;
     ns1:めい "ひでよし" ;
-    ns1:候補名 "朝賀英義" ;
+    ns1:候補名 "あさか英義" ;
     ns1:公式サイト "http://www.jcp-niiza.com/asaka/" ;
     ns1:前回 "新" ;
     ns1:名 "英義" ;
@@ -1819,30 +1806,28 @@
     ns1:twitter "hondanobuhiro" ;
     ns1:せい "ほんだ" ;
     ns1:めい "のぶひろ" ;
-    ns1:候補名 "本多信弘" ;
+    ns1:候補名 "本多のぶひろ" ;
     ns1:公式サイト "http://jcpweb.net/honda/" ;
     ns1:前回 "新" ;
     ns1:名 "信弘" ;
     ns1:名前 "本多信弘" ;
     ns1:姓 "本多" ;
     ns1:小選挙区 "愛知11区" ;
-    ns1:年齢 "42" ;
-    ns1:比例区 "" .
-
-[] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/nobuko.motomura.3",
-        "https://www.facebook.com/sogasogabega" ;
+    ns1:年齢 "41" ;
+    ns1:比例区 "" .
+
+[] ns1:blog "" ;
+    ns1:facebook "https://www.facebook.com/nobuko.motomura.3" ;
     ns1:twitter "motomura_nobuko" ;
     ns1:せい "もとむら" ;
     ns1:めい "のぶこ" ;
     ns1:候補名 "もとむら伸子" ;
-    ns1:公式サイト "http://motomura.jcp-aichi.jp/" ;
     ns1:前回 "前" ;
     ns1:名 "伸子" ;
     ns1:名前 "本村伸子" ;
     ns1:姓 "本村" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "45" ;
+    ns1:年齢 "44" ;
     ns1:比例区 "東海" .
 
 [] ns1:blog "" ;
@@ -1861,14 +1846,11 @@
     ns1:比例区 "" .
 
 [] ns1:blog "http://honmakatsumi.blogspot.jp/" ;
-    ns1:facebook "https://www.facebook.com/honmakatsumi",
-        "https://www.facebook.com/katsumi.honma.1",
-        "https://www.facebook.com/profile.php?id=100010830348450" ;
-    ns1:twitter "qn93vr" ;
+    ns1:facebook "https://www.facebook.com/honmakatsumi" ;
+    ns1:twitter "" ;
     ns1:せい "ほんま" ;
     ns1:めい "かつみ" ;
-    ns1:候補名 "本間勝美" ;
-    ns1:公式サイト "http://www.katumi.jp/" ;
+    ns1:候補名 "本間かつみ" ;
     ns1:前回 "新" ;
     ns1:名 "勝美" ;
     ns1:名前 "本間勝美" ;
@@ -1883,34 +1865,31 @@
     ns1:せい "すぎした" ;
     ns1:めい "しげお" ;
     ns1:候補名 "杉下茂雄" ;
-    ns1:公式サイト "http://senkyo.jcp-tokyo.net/sugishita/profile/" ;
+    ns1:公式サイト "http://senkyo.jcp-tokyo.net/sugishita/" ;
     ns1:前回 "新" ;
     ns1:名 "茂雄" ;
     ns1:名前 "杉下茂雄" ;
     ns1:姓 "杉下" ;
     ns1:小選挙区 "東京19区" ;
-    ns1:年齢 "69" ;
-    ns1:比例区 "" .
-
-[] ns1:blog "http://muraguti-kumiko.cocolog-nifty.com" ;
-    ns1:facebook "https://www.facebook.com/jcpsuitasettsukuntan",
-        "https://www.facebook.com/kumiko.muraguchi.7" ;
+    ns1:年齢 "68" ;
+    ns1:比例区 "" .
+
+[] ns1:blog "" ;
+    ns1:facebook "https://www.facebook.com/jcpsuitasettsukuntan" ;
     ns1:twitter "hanahanaexpress" ;
     ns1:せい "むらぐち" ;
     ns1:めい "くみこ" ;
-    ns1:候補名 "村口久美子" ;
-    ns1:公式サイト "https://jcp-suitasettsu.wixsite.com/kuntan" ;
-    ns1:前回 "新" ;
-    ns1:名 "久美子" ;
-    ns1:名前 "村口久美子" ;
+    ns1:候補名 "村口くみ子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子久美" ;
+    ns1:名前 "村口子久美" ;
     ns1:姓 "村口" ;
     ns1:小選挙区 "大阪7区" ;
     ns1:年齢 "45" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/hideaki.matsui.9081",
-        "https://www.facebook.com/matsuihideakijimusyo" ;
+    ns1:facebook "https://www.facebook.com/matsuihideakijimusyo" ;
     ns1:twitter "" ;
     ns1:せい "まつい" ;
     ns1:めい "ひであき" ;
@@ -1936,7 +1915,7 @@
     ns1:名前 "松原聡" ;
     ns1:姓 "松原" ;
     ns1:小選挙区 "静岡4区" ;
-    ns1:年齢 "56" ;
+    ns1:年齢 "55" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
@@ -1944,11 +1923,11 @@
     ns1:twitter "matsuo_ritsuko" ;
     ns1:せい "まつお" ;
     ns1:めい "りつこ" ;
-    ns1:候補名 "松尾律子" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "律子" ;
-    ns1:名前 "松尾律子" ;
+    ns1:候補名 "松尾りつ子" ;
+    ns1:公式サイト "" ;
+    ns1:前回 "新" ;
+    ns1:名 "子律" ;
+    ns1:名前 "松尾子律" ;
     ns1:姓 "松尾" ;
     ns1:小選挙区 "福岡2区" ;
     ns1:年齢 "45" ;
@@ -1956,7 +1935,7 @@
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/profile.php?id=100016365635786" ;
-    ns1:twitter "matsuo_masanori" ;
+    ns1:twitter "" ;
     ns1:せい "まつお" ;
     ns1:めい "まさのり" ;
     ns1:候補名 "松尾まさのり" ;
@@ -1969,13 +1948,12 @@
     ns1:年齢 "58" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://matuokamasaru.blogspot.jp" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/mtok48" ;
     ns1:twitter "monsmsg2" ;
     ns1:せい "まつおか" ;
     ns1:めい "まさる" ;
     ns1:候補名 "松岡勝" ;
-    ns1:公式サイト "http://matuoka44.webcrow.jp/" ;
     ns1:前回 "新" ;
     ns1:名 "勝" ;
     ns1:名前 "松岡勝" ;
@@ -1985,12 +1963,11 @@
     ns1:比例区 "九州" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/matsumotokenji.kochi" ;
+    ns1:facebook "https://www.facebook.com/profile.php?id=100013854768106" ;
     ns1:twitter "matsu_en" ;
     ns1:せい "まつもと" ;
     ns1:めい "けんじ" ;
     ns1:候補名 "松本けんじ" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "顕治" ;
     ns1:名前 "松本顕治" ;
@@ -2004,8 +1981,8 @@
     ns1:twitter "akahigepapa" ;
     ns1:せい "まつむら" ;
     ns1:めい "りょうすけ" ;
-    ns1:候補名 "松村亮佑" ;
-    ns1:公式サイト "http://senkyo.jcp-tokyo.net/matsumura/" ;
+    ns1:候補名 "松村りょうすけ" ;
+    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "亮佑" ;
     ns1:名前 "松村亮佑" ;
@@ -2014,13 +1991,12 @@
     ns1:年齢 "37" ;
     ns1:比例区 "" .
 
-[] ns1:blog "https://ameblo.jp/matsuhashi-chiharu/" ;
+[] ns1:blog "http://ameblo.jp/matsuhashi-chiharu/" ;
     ns1:facebook "https://www.facebook.com/profile.php?id=100008887746844" ;
     ns1:twitter "" ;
     ns1:せい "まつはし" ;
     ns1:めい "ちはる" ;
-    ns1:候補名 "松橋千春" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "松橋ちはる" ;
     ns1:前回 "新" ;
     ns1:名 "千春" ;
     ns1:名前 "松橋千春" ;
@@ -2029,13 +2005,13 @@
     ns1:年齢 "35" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://blog.livedoor.jp/kazushi53/" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/KazushiMatsuda53" ;
-    ns1:twitter "jcpkazushi53" ;
+    ns1:twitter "" ;
     ns1:せい "まつだ" ;
     ns1:めい "かずし" ;
     ns1:候補名 "松田一志" ;
-    ns1:公式サイト "http://www.matsudakazushi53.com/" ;
+    ns1:公式サイト "http://blog.livedoor.jp/kazushi53/" ;
     ns1:前回 "新" ;
     ns1:名 "一志" ;
     ns1:名前 "松田一志" ;
@@ -2044,14 +2020,12 @@
     ns1:年齢 "60" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://itakuramasahumi.blog69.fc2.com/" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/masafumi.itakura.5" ;
-    ns1:twitter "2b28a3370fae4f9",
-        "itakurama" ;
+    ns1:twitter "2b28a3370fae4f9" ;
     ns1:せい "いたくら" ;
     ns1:めい "まさふみ" ;
     ns1:候補名 "いたくら正文" ;
-    ns1:公式サイト "http://jcpweb.net/itakura/" ;
     ns1:前回 "新" ;
     ns1:名 "正文" ;
     ns1:名前 "板倉正文" ;
@@ -2065,7 +2039,7 @@
     ns1:twitter "" ;
     ns1:せい "はやし" ;
     ns1:めい "みやこ" ;
-    ns1:候補名 "林京" ;
+    ns1:候補名 "林みやこ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "京" ;
@@ -2076,11 +2050,11 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/profile.php?id=100006941132398" ;
+    ns1:facebook "" ;
     ns1:twitter "hayashihide22" ;
     ns1:せい "はやし" ;
     ns1:めい "ひでひろ" ;
-    ns1:候補名 "林秀洋" ;
+    ns1:候補名 "林ひでひろ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "秀洋" ;
@@ -2095,7 +2069,7 @@
     ns1:twitter "shibaoka_yuma" ;
     ns1:せい "しばおか" ;
     ns1:めい "ゆうま" ;
-    ns1:候補名 "柴岡祐真" ;
+    ns1:候補名 "柴岡ゆうま" ;
     ns1:公式サイト "http://www.shibaoka-yuma.jp/" ;
     ns1:前回 "新" ;
     ns1:名 "祐真" ;
@@ -2106,29 +2080,26 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/kakinumaharuki" ;
-    ns1:twitter "140829haruking" ;
-    ns1:せい "かきぬま" ;
-    ns1:めい "はるき" ;
-    ns1:候補名 "柿沼遥輝" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "遥輝" ;
-    ns1:名前 "柿沼遥輝" ;
-    ns1:姓 "柿沼" ;
+    ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
+    ns1:候補名 "柿沼はるき" ;
+    ns1:公式サイト "" ;
+    ns1:前回 "新" ;
+    ns1:名 "" ;
+    ns1:名前 "柿沼はるき" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "埼玉10区" ;
     ns1:年齢 "34" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/saeko.umemura.3",
-        "https://www.facebook.com/umemurasaeko" ;
+    ns1:facebook "https://www.facebook.com/umemurasaeko" ;
     ns1:twitter "umemura_saeko" ;
     ns1:せい "うめむら" ;
     ns1:めい "さえこ" ;
-    ns1:候補名 "梅村さえこ",
-        "関口早江子" ;
-    ns1:公式サイト "http://www.jcp-umemura.jp/" ;
+    ns1:候補名 "梅村さえこ" ;
     ns1:前回 "前" ;
     ns1:名 "早江子" ;
     ns1:名前 "梅村早江子" ;
@@ -2143,7 +2114,6 @@
     ns1:せい "もりざくら" ;
     ns1:めい "ふさよし" ;
     ns1:候補名 "森桜ふさよし" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "房義" ;
     ns1:名前 "森桜房義" ;
@@ -2172,7 +2142,7 @@
     ns1:twitter "" ;
     ns1:せい "くすもと" ;
     ns1:めい "ふみろう" ;
-    ns1:候補名 "楠本文郎" ;
+    ns1:候補名 "くすもと文郎" ;
     ns1:公式サイト "https://humirou-kusumoto.jimdo.com/" ;
     ns1:前回 "新" ;
     ns1:名 "文郎" ;
@@ -2187,7 +2157,7 @@
     ns1:twitter "" ;
     ns1:せい "まき" ;
     ns1:めい "しょうぞう" ;
-    ns1:候補名 "槙昌三" ;
+    ns1:候補名 "まき昌三" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "昌三" ;
@@ -2202,8 +2172,7 @@
     ns1:twitter "yokoyamaseigo" ;
     ns1:せい "よこやま" ;
     ns1:めい "せいご" ;
-    ns1:候補名 "横山征吾" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "横山せいご" ;
     ns1:前回 "新" ;
     ns1:名 "征吾" ;
     ns1:名前 "横山征吾" ;
@@ -2227,13 +2196,12 @@
     ns1:年齢 "73" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://jcpyokota.blogspot.jp" ;
-    ns1:facebook "https://www.facebook.com/jcpyokotaeiji" ;
+[] ns1:blog "http://jcpyokota.blogspot.jp/" ;
+    ns1:facebook "https://www.facebook.com/profile.php?id=100004574320067" ;
     ns1:twitter "yokotaeiji17" ;
     ns1:せい "よこた" ;
     ns1:めい "えいじ" ;
     ns1:候補名 "横田英司" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "英司" ;
     ns1:名前 "横田英司" ;
@@ -2247,7 +2215,7 @@
     ns1:twitter "" ;
     ns1:せい "はしもと" ;
     ns1:めい "けんじ" ;
-    ns1:候補名 "橋本健二" ;
+    ns1:候補名 "橋本けんじ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "健二" ;
@@ -2263,7 +2231,6 @@
     ns1:せい "もうり" ;
     ns1:めい "えいこ" ;
     ns1:候補名 "もうり栄子" ;
-    ns1:公式サイト "http://plaza.rakuten.co.jp/mourieiko/" ;
     ns1:前回 "新" ;
     ns1:名 "栄子" ;
     ns1:名前 "毛利栄子" ;
@@ -2277,7 +2244,7 @@
     ns1:twitter "omuta_eguchi" ;
     ns1:せい "えぐち" ;
     ns1:めい "まなぶ" ;
-    ns1:候補名 "江口学" ;
+    ns1:候補名 "江口まなぶ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "学" ;
@@ -2288,14 +2255,11 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/saolyn.ikeuchi",
-        "https://www.facebook.com/saorioshitai" ;
-    ns1:twitter "ikeuchi_saori",
-        "saorijimusyo" ;
+    ns1:facebook "https://www.facebook.com/saolyn.ikeuchi" ;
+    ns1:twitter "ikeuchi_saori" ;
     ns1:せい "いけうち" ;
     ns1:めい "さおり" ;
     ns1:候補名 "池内さおり" ;
-    ns1:公式サイト "http://www.saori-ikeuchi.com/" ;
     ns1:前回 "前" ;
     ns1:名 "沙織" ;
     ns1:名前 "池内沙織" ;
@@ -2304,13 +2268,12 @@
     ns1:年齢 "35" ;
     ns1:比例区 "東京" .
 
-[] ns1:blog "http://jcpikeda.blogspot.jp" ;
+[] ns1:blog "http://jcpikeda.blogspot.jp/" ;
     ns1:facebook "https://www.facebook.com/profile.php?id=100008285657728" ;
     ns1:twitter "" ;
     ns1:せい "いけだ" ;
     ns1:めい "ひろひで" ;
     ns1:候補名 "池田博英" ;
-    ns1:公式サイト "http://jcp.kir.jp/09.html" ;
     ns1:前回 "新" ;
     ns1:名 "博英" ;
     ns1:名前 "池田博英" ;
@@ -2324,7 +2287,7 @@
     ns1:twitter "" ;
     ns1:せい "かわむら" ;
     ns1:めい "ただし" ;
-    ns1:候補名 "河村整" ;
+    ns1:候補名 "河村ただし" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "整" ;
@@ -2339,7 +2302,7 @@
     ns1:twitter "" ;
     ns1:せい "ふかつ" ;
     ns1:めい "としろう" ;
-    ns1:候補名 "深津俊郎" ;
+    ns1:候補名 "深津としろう" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "俊郎" ;
@@ -2350,13 +2313,11 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/shimizu.tadashi.jcp",
-        "https://www.facebook.com/tadashi.shimizu.90" ;
+    ns1:facebook "https://www.facebook.com/tadashi.shimizu.90" ;
     ns1:twitter "tadashishimizu" ;
     ns1:せい "しみず" ;
     ns1:めい "ただし" ;
     ns1:候補名 "清水ただし" ;
-    ns1:公式サイト "http://www.shimizu-tadashi.jp/" ;
     ns1:前回 "前" ;
     ns1:名 "忠史" ;
     ns1:名前 "清水忠史" ;
@@ -2366,12 +2327,11 @@
     ns1:比例区 "近畿" .
 
 [] ns1:blog "" ;
-    ns1:facebook "" ;
+    ns1:facebook "https://www.facebook.com/watanabe.hiroshi.79" ;
     ns1:twitter "" ;
     ns1:せい "わたなべ" ;
     ns1:めい "ひろし" ;
-    ns1:候補名 "渡辺裕" ;
-    ns1:公式サイト "http://jcpweb.net/watanabe/" ;
+    ns1:候補名 "渡辺ひろし" ;
     ns1:前回 "新" ;
     ns1:名 "裕" ;
     ns1:名前 "渡辺裕" ;
@@ -2386,7 +2346,6 @@
     ns1:せい "わたなべ" ;
     ns1:めい "ゆい" ;
     ns1:候補名 "わたなべ結" ;
-    ns1:公式サイト "http://www.watanabeyui.net/" ;
     ns1:前回 "新" ;
     ns1:名 "結" ;
     ns1:名前 "渡部結" ;
@@ -2400,7 +2359,7 @@
     ns1:twitter "" ;
     ns1:せい "わたなべ" ;
     ns1:めい "たかお" ;
-    ns1:候補名 "渡部隆夫" ;
+    ns1:候補名 "わたなべ隆夫" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "隆夫" ;
@@ -2415,8 +2374,7 @@
     ns1:twitter "" ;
     ns1:せい "せと" ;
     ns1:めい "かずひろ" ;
-    ns1:候補名 "瀬戸和弘" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "せと和弘" ;
     ns1:前回 "新" ;
     ns1:名 "和弘" ;
     ns1:名前 "瀬戸和弘" ;
@@ -2426,19 +2384,18 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/kumagaitomo3",
-        "https://www.facebook.com/kumatomo3" ;
+    ns1:facebook "https://www.facebook.com/kumagaitomo3" ;
     ns1:twitter "h0use3" ;
     ns1:せい "くまがい" ;
     ns1:めい "とも" ;
-    ns1:候補名 "熊谷智" ;
-    ns1:公式サイト "http://kumatomo.hateblo.jp/" ;
+    ns1:候補名 "くまがい智" ;
+    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "智" ;
     ns1:名前 "熊谷智" ;
     ns1:姓 "熊谷" ;
     ns1:小選挙区 "福島5区" ;
-    ns1:年齢 "38" ;
+    ns1:年齢 "37" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
@@ -2461,7 +2418,7 @@
     ns1:twitter "" ;
     ns1:せい "さるはし" ;
     ns1:めい "たくみ" ;
-    ns1:候補名 "猿橋たくみ" ;
+    ns1:候補名 "さるはし巧" ;
     ns1:公式サイト "http://jcp-fukui.jp/" ;
     ns1:前回 "新" ;
     ns1:名 "巧" ;
@@ -2472,12 +2429,26 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
+    ns1:facebook "https://www.facebook.com/denny.tamaki" ;
+    ns1:twitter "tamakidenny" ;
+    ns1:せい "たまき" ;
+    ns1:めい "デニー" ;
+    ns1:候補名 "玉城デニー" ;
+    ns1:公式サイト "http://d21tamaki.com/" ;
+    ns1:前回 "前" ;
+    ns1:名 "デニー" ;
+    ns1:名前 "玉城デニー" ;
+    ns1:姓 "玉城" ;
+    ns1:小選挙区 "沖縄3区(推薦、オール沖縄)" ;
+    ns1:年齢 "58" ;
+    ns1:比例区 "" .
+
+[] ns1:blog "" ;
     ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "たがみ" ;
     ns1:めい "みつのり" ;
-    ns1:候補名 "田上光徳" ;
-    ns1:公式サイト "http://www.jcp-aichi.jp/kokusei/tagami" ;
+    ns1:候補名 "田上みつのり" ;
     ns1:前回 "新" ;
     ns1:名 "光徳" ;
     ns1:名前 "田上光徳" ;
@@ -2491,7 +2462,7 @@
     ns1:twitter "tanakayoji" ;
     ns1:せい "たなか" ;
     ns1:めい "ようじ" ;
-    ns1:候補名 "田中陽二" ;
+    ns1:候補名 "田中ようじ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "陽二" ;
@@ -2502,13 +2473,11 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/takaaki.tamura.16",
-        "https://www.facebook.com/tamuratakaaki" ;
+    ns1:facebook "https://www.facebook.com/tamuratakaaki" ;
     ns1:twitter "tamuratakaaki" ;
     ns1:せい "たむら" ;
     ns1:めい "たかあき" ;
-    ns1:候補名 "田村貴昭" ;
-    ns1:公式サイト "http://tamura-takaaki.com/" ;
+    ns1:候補名 "田村たかあき" ;
     ns1:前回 "前" ;
     ns1:名 "貴昭" ;
     ns1:名前 "田村貴昭" ;
@@ -2523,7 +2492,6 @@
     ns1:せい "はたの" ;
     ns1:めい "きみえ" ;
     ns1:候補名 "はたの君枝" ;
-    ns1:公式サイト "http://www.hatano-kimie.jp/" ;
     ns1:前回 "前" ;
     ns1:名 "君枝" ;
     ns1:名前 "畑野君枝" ;
@@ -2532,50 +2500,47 @@
     ns1:年齢 "60" ;
     ns1:比例区 "南関東" .
 
-[] ns1:blog "https://hatarogu.blogspot.jp/" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/kazuya.hatayama.1" ;
     ns1:twitter "hatayama_kazuya" ;
     ns1:せい "はたやま" ;
     ns1:めい "かずや" ;
     ns1:候補名 "はたやま和也" ;
-    ns1:公式サイト "http://hatayamakazuya.com/" ;
     ns1:前回 "前" ;
     ns1:名 "和也" ;
     ns1:名前 "畠山和也" ;
     ns1:姓 "畠山" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "46" ;
+    ns1:年齢 "45" ;
     ns1:比例区 "北海道" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/SirakawaYouko",
-        "https://www.facebook.com/profile.php?id=100001986880844" ;
-    ns1:twitter "youkosirakawa" ;
+    ns1:facebook "https://www.facebook.com/SirakawaYouko" ;
+    ns1:twitter "" ;
     ns1:せい "しらかわ" ;
     ns1:めい "ようこ" ;
-    ns1:候補名 "白川容子" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "容子" ;
-    ns1:名前 "白川容子" ;
+    ns1:候補名 "白川よう子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子容" ;
+    ns1:名前 "白川子容" ;
     ns1:姓 "白川" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "51" ;
+    ns1:年齢 "50" ;
     ns1:比例区 "四国" .
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/masuda.jcp" ;
-    ns1:twitter "masudajcp" ;
+    ns1:twitter "" ;
     ns1:せい "ますだ" ;
     ns1:めい "まきこ" ;
-    ns1:候補名 "益田牧子" ;
+    ns1:候補名 "ますだ牧子" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "牧子" ;
     ns1:名前 "益田牧子" ;
     ns1:姓 "益田" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "67" ;
+    ns1:年齢 "66" ;
     ns1:比例区 "九州" .
 
 [] ns1:blog "" ;
@@ -2584,7 +2549,6 @@
     ns1:せい "まじま" ;
     ns1:めい "しょうぞう" ;
     ns1:候補名 "まじま省三" ;
-    ns1:公式サイト "http://jcp-majimasyouzo.jp/" ;
     ns1:前回 "前" ;
     ns1:名 "省三" ;
     ns1:名前 "真島省三" ;
@@ -2610,7 +2574,7 @@
 
 [] ns1:blog "" ;
     ns1:facebook "" ;
-    ns1:twitter "jcpishimaru" ;
+    ns1:twitter "" ;
     ns1:せい "いしまる" ;
     ns1:めい "かんじ" ;
     ns1:候補名 "石丸完治" ;
@@ -2628,7 +2592,7 @@
     ns1:twitter "jcp_ishidoh" ;
     ns1:せい "いしどう" ;
     ns1:めい "あつし" ;
-    ns1:候補名 "石堂淳士" ;
+    ns1:候補名 "石堂あつし" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "淳士" ;
@@ -2643,14 +2607,14 @@
     ns1:twitter "" ;
     ns1:せい "いしじま" ;
     ns1:めい "いわお" ;
-    ns1:候補名 "石嶋巌" ;
+    ns1:候補名 "石嶋いわお" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "巌" ;
     ns1:名前 "石嶋巌" ;
     ns1:姓 "石嶋" ;
     ns1:小選挙区 "茨城7区" ;
-    ns1:年齢 "58" ;
+    ns1:年齢 "63" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
@@ -2668,7 +2632,7 @@
     ns1:年齢 "65" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://blog.goo.ne.jp/ishiakiishi8740" ;
+[] ns1:blog "" ;
     ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "いしかわ" ;
@@ -2685,7 +2649,7 @@
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/shoishikawa1973" ;
-    ns1:twitter "sho_ishikawa38" ;
+    ns1:twitter "" ;
     ns1:せい "いしかわ" ;
     ns1:めい "しょう" ;
     ns1:候補名 "石川渉" ;
@@ -2714,11 +2678,11 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/noriyuki.ishizawa.7" ;
+    ns1:facebook "" ;
     ns1:twitter "noriyuki_ishi" ;
     ns1:せい "いしざわ" ;
     ns1:めい "のりゆき" ;
-    ns1:候補名 "石沢憲之" ;
+    ns1:候補名 "石沢のりゆき" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "憲之" ;
@@ -2728,13 +2692,13 @@
     ns1:年齢 "30" ;
     ns1:比例区 "東京" .
 
-[] ns1:blog "http://blog.goo.ne.jp/motokisobue" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/profile.php?id=100015441914773" ;
     ns1:twitter "sobuemotoki" ;
     ns1:せい "そぶえ" ;
     ns1:めい "もとき" ;
     ns1:候補名 "そぶえ元希" ;
-    ns1:公式サイト "" ;
+    ns1:公式サイト "http://blog.goo.ne.jp/motokisobue" ;
     ns1:前回 "新" ;
     ns1:名 "元希" ;
     ns1:名前 "祖父江元希" ;
@@ -2793,7 +2757,7 @@
     ns1:twitter "inasan1730724" ;
     ns1:せい "いなむら" ;
     ns1:めい "さとる" ;
-    ns1:候補名 "稲村知" ;
+    ns1:候補名 "稲村さとる" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "知" ;
@@ -2808,8 +2772,7 @@
     ns1:twitter "kokutakeiji" ;
     ns1:せい "こくた" ;
     ns1:めい "けいじ" ;
-    ns1:候補名 "穀田恵二" ;
-    ns1:公式サイト "http://www.kokuta-keiji.jp/" ;
+    ns1:候補名 "こくた恵二" ;
     ns1:前回 "前" ;
     ns1:名 "恵二" ;
     ns1:名前 "穀田恵二" ;
@@ -2839,13 +2802,12 @@
     ns1:せい "かさい" ;
     ns1:めい "あきら" ;
     ns1:候補名 "笠井亮" ;
-    ns1:公式サイト "http://www.kasai-akira.jp/" ;
     ns1:前回 "前" ;
     ns1:名 "亮" ;
     ns1:名前 "笠井亮" ;
     ns1:姓 "笠井" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "65" ;
+    ns1:年齢 "64" ;
     ns1:比例区 "東京" .
 
 [] ns1:blog "" ;
@@ -2868,7 +2830,7 @@
     ns1:twitter "" ;
     ns1:せい "みいみ" ;
     ns1:めい "よしあき" ;
-    ns1:候補名 "美見芳明" ;
+    ns1:候補名 "みいみ芳明" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "芳明" ;
@@ -2883,19 +2845,18 @@
     ns1:twitter "" ;
     ns1:せい "ふなやま" ;
     ns1:めい "ゆみ" ;
-    ns1:候補名 "舩山由美" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "ふなやま由美" ;
     ns1:前回 "新" ;
     ns1:名 "由美" ;
     ns1:名前 "舩山由美" ;
     ns1:姓 "舩山" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "49" ;
+    ns1:年齢 "48" ;
     ns1:比例区 "東北" .
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/kyoko.nae" ;
-    ns1:twitter "kyokonae" ;
+    ns1:twitter "" ;
     ns1:せい "なえむら" ;
     ns1:めい "きょうこ" ;
     ns1:候補名 "苗村京子" ;
@@ -2908,13 +2869,12 @@
     ns1:年齢 "58" ;
     ns1:比例区 "" .
 
-[] ns1:blog "https://jcpwakabayashi.blogspot.jp" ;
+[] ns1:blog "https://jcpwakabayashi.blogspot.jp/" ;
     ns1:facebook "https://www.facebook.com/wakabayashi8" ;
     ns1:twitter "wakayasu1985" ;
     ns1:せい "わかばやし" ;
     ns1:めい "やすひさ" ;
     ns1:候補名 "若林やすひさ" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "靖久" ;
     ns1:名前 "若林靖久" ;
@@ -2923,13 +2883,13 @@
     ns1:年齢 "32" ;
     ns1:比例区 "南関東" .
 
-[] ns1:blog "http://sugawara.jcpkitami.jp/" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/makoto.sugawara.33" ;
     ns1:twitter "sugawara_makoto" ;
     ns1:せい "すがわら" ;
     ns1:めい "まこと" ;
-    ns1:候補名 "菅原誠" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "菅原まこと" ;
+    ns1:公式サイト "http://sugawara.jcpkitami.jp/" ;
     ns1:前回 "新" ;
     ns1:名 "誠" ;
     ns1:名前 "菅原誠" ;
@@ -2943,7 +2903,7 @@
     ns1:twitter "" ;
     ns1:せい "はぎわら" ;
     ns1:めい "さだお" ;
-    ns1:候補名 "萩原貞夫" ;
+    ns1:候補名 "はぎわら貞夫" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "貞夫" ;
@@ -2958,8 +2918,8 @@
     ns1:twitter "1218fuji" ;
     ns1:せい "ふじもと" ;
     ns1:めい "かずのり" ;
-    ns1:候補名 "藤本一規" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "藤本かずのり" ;
+    ns1:公式サイト "http://ikki.jcp-web.net/" ;
     ns1:前回 "新" ;
     ns1:名 "一規" ;
     ns1:名前 "藤本一規" ;
@@ -2989,13 +2949,13 @@
     ns1:せい "ふじもと" ;
     ns1:めい "さちこ" ;
     ns1:候補名 "藤本さちこ" ;
-    ns1:公式サイト "http://fujimoto.jcp-sakai.jp/" ;
+    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "幸子" ;
     ns1:名前 "藤本幸子" ;
     ns1:姓 "藤本" ;
     ns1:小選挙区 "大阪17区" ;
-    ns1:年齢 "43" ;
+    ns1:年齢 "42" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
@@ -3018,14 +2978,13 @@
     ns1:twitter "fujinofujinooo" ;
     ns1:せい "ふじの" ;
     ns1:めい "やすふみ" ;
-    ns1:候補名 "藤野保史" ;
-    ns1:公式サイト "http://fujino.jcpweb.net/" ;
+    ns1:候補名 "藤野やすふみ" ;
     ns1:前回 "前" ;
     ns1:名 "保史" ;
     ns1:名前 "藤野保史" ;
     ns1:姓 "藤野" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "47" ;
+    ns1:年齢 "46" ;
     ns1:比例区 "北陸信越" .
 
 [] ns1:blog "" ;
@@ -3033,7 +2992,7 @@
     ns1:twitter "" ;
     ns1:せい "にしなか" ;
     ns1:めい "たかお" ;
-    ns1:候補名 "西中孝男" ;
+    ns1:候補名 "西中たかお" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "孝男" ;
@@ -3074,12 +3033,11 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/jcp.nishizawa.k",
-        "https://www.facebook.com/koichi.nishizawa.758" ;
+    ns1:facebook "https://www.facebook.com/koichi.nishizawa.758" ;
     ns1:twitter "nissyzaaaawa" ;
     ns1:せい "にしざわ" ;
     ns1:めい "こういち" ;
-    ns1:候補名 "西沢耕一" ;
+    ns1:候補名 "西沢こういち" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "耕一" ;
@@ -3089,19 +3047,18 @@
     ns1:年齢 "39" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://blog.goo.ne.jp/melf4511" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/hiroshi.nishizawa.3910" ;
     ns1:twitter "melf4511" ;
     ns1:せい "にしざわ" ;
     ns1:めい "ひろし" ;
     ns1:候補名 "にしざわ博" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "博" ;
     ns1:名前 "西澤博" ;
     ns1:姓 "西澤" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "37" ;
+    ns1:年齢 "36" ;
     ns1:比例区 "北陸信越" .
 
 [] ns1:blog "" ;
@@ -3109,24 +3066,22 @@
     ns1:twitter "" ;
     ns1:せい "たになか" ;
     ns1:めい "みよし" ;
-    ns1:候補名 "谷中三好" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "三好" ;
-    ns1:名前 "谷中三好" ;
+    ns1:候補名 "谷中みよし" ;
+    ns1:公式サイト "" ;
+    ns1:前回 "新" ;
+    ns1:名 "みよし" ;
+    ns1:名前 "谷中みよし" ;
     ns1:姓 "谷中" ;
     ns1:小選挙区 "三重4区" ;
     ns1:年齢 "57" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/JCP.TANet",
-        "https://www.facebook.com/dr.tanigawa" ;
+    ns1:facebook "https://www.facebook.com/profile.php?id=100001997661120" ;
     ns1:twitter "ttanigawa3" ;
     ns1:せい "たにがわ" ;
     ns1:めい "ともゆき" ;
     ns1:候補名 "谷川智行" ;
-    ns1:公式サイト "http://senkyo.jcp-tokyo.net/tanigawa/" ;
     ns1:前回 "新" ;
     ns1:名 "智行" ;
     ns1:名前 "谷川智行" ;
@@ -3136,27 +3091,25 @@
     ns1:比例区 "東京" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/akamineseiken",
-        "https://www.facebook.com/seikenn.akamine" ;
+    ns1:facebook "https://www.facebook.com/akamineseiken" ;
     ns1:twitter "akamineseiken" ;
     ns1:せい "あかみね" ;
     ns1:めい "せいけん" ;
     ns1:候補名 "あかみね政賢" ;
-    ns1:公式サイト "http://akamine-seiken.jp/" ;
     ns1:前回 "前" ;
     ns1:名 "政賢" ;
     ns1:名前 "赤嶺政賢" ;
     ns1:姓 "赤嶺" ;
-    ns1:小選挙区 "沖縄1区" ;
+    ns1:小選挙区 "沖縄1区(党公認、オール沖縄)" ;
     ns1:年齢 "69" ;
     ns1:比例区 "九州" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/yuto.akahira" ;
-    ns1:twitter "akahira1990" ;
+    ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:せい "あかひら" ;
     ns1:めい "ゆうと" ;
-    ns1:候補名 "赤平勇人" ;
+    ns1:候補名 "赤平ゆうと" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "勇人" ;
@@ -3170,8 +3123,8 @@
     ns1:facebook "https://www.facebook.com/jcp.sai" ;
     ns1:twitter "tsujijcp" ;
     ns1:せい "つじ" ;
-    ns1:めい "もとみ" ;
-    ns1:候補名 "辻源巳" ;
+    ns1:めい "モトミ" ;
+    ns1:候補名 "つじモトミ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "源巳" ;
@@ -3200,9 +3153,8 @@
     ns1:facebook "https://www.facebook.com/kentaro.sakai.12" ;
     ns1:twitter "iakas_oratnek" ;
     ns1:せい "さかい" ;
-    ns1:めい "けんたろう" ;
-    ns1:候補名 "酒井健太朗" ;
-    ns1:公式サイト "http://jcpweb.net/sakai/" ;
+    ns1:めい "ケンタロー" ;
+    ns1:候補名 "酒井ケンタロー" ;
     ns1:前回 "新" ;
     ns1:名 "健太朗" ;
     ns1:名前 "酒井健太朗" ;
@@ -3211,13 +3163,12 @@
     ns1:年齢 "31" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://kojishigemura.exblog.jp" ;
+[] ns1:blog "http://kojishigemura.exblog.jp/" ;
     ns1:facebook "" ;
     ns1:twitter "koji_1951" ;
     ns1:せい "しげむら" ;
     ns1:めい "こうじ" ;
-    ns1:候補名 "重村幸司" ;
-    ns1:公式サイト "" ;
+    ns1:候補名 "しげむら幸司" ;
     ns1:前回 "新" ;
     ns1:名 "幸司" ;
     ns1:名前 "重村幸司" ;
@@ -3231,7 +3182,7 @@
     ns1:twitter "" ;
     ns1:せい "のむら" ;
     ns1:めい "まさひろ" ;
-    ns1:候補名 "野村真弘" ;
+    ns1:候補名 "野村まさひろ" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "真弘" ;
@@ -3241,9 +3192,8 @@
     ns1:年齢 "61" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://www.kanakura.net/" ;
-    ns1:facebook "https://www.facebook.com/kanakuramasatoshi",
-        "https://www.facebook.com/kanakuramasatosioffice" ;
+[] ns1:blog "" ;
+    ns1:facebook "https://www.facebook.com/kanakuramasatoshi" ;
     ns1:twitter "kanakuramasato" ;
     ns1:せい "かなくら" ;
     ns1:めい "まさとし" ;
@@ -3262,7 +3212,7 @@
     ns1:twitter "jcp_555" ;
     ns1:せい "かねもと" ;
     ns1:めい "ゆきえ" ;
-    ns1:候補名 "金元幸枝" ;
+    ns1:候補名 "かねもと幸枝" ;
     ns1:公式サイト "http://jcp-fukui.jp/" ;
     ns1:前回 "新" ;
     ns1:名 "幸枝" ;
@@ -3274,11 +3224,10 @@
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/nobuyuki.kinpara" ;
-    ns1:twitter "kpf9" ;
+    ns1:twitter "" ;
     ns1:せい "きんぱら" ;
     ns1:めい "のぶゆき" ;
-    ns1:候補名 "金原信之" ;
-    ns1:公式サイト "http://jcpweb.net/kinpara/" ;
+    ns1:候補名 "金原のぶゆき" ;
     ns1:前回 "新" ;
     ns1:名 "信之" ;
     ns1:名前 "金原信之" ;
@@ -3293,7 +3242,7 @@
     ns1:せい "かなもり" ;
     ns1:めい "とおる" ;
     ns1:候補名 "金森とおる" ;
-    ns1:公式サイト "" ;
+    ns1:公式サイト "http://www.jcp-kyoto.jp/2017shuin/candidate03.html" ;
     ns1:前回 "新" ;
     ns1:名 "亨" ;
     ns1:名前 "金森亨" ;
@@ -3317,7 +3266,7 @@
     ns1:年齢 "52" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://kugimaru.blogspot.jp/" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/susumu.kugimaru" ;
     ns1:twitter "kugimarususumu" ;
     ns1:せい "くぎまる" ;
@@ -3332,13 +3281,12 @@
     ns1:年齢 "44" ;
     ns1:比例区 "" .
 
-[] ns1:blog "https://ameblo.jp/chikachika-gogo/" ;
+[] ns1:blog "http://ameblo.jp/chikachika-gogo/" ;
     ns1:facebook "https://www.facebook.com/chikachikagogo" ;
     ns1:twitter "chikachika_gogo" ;
     ns1:せい "すずき" ;
     ns1:めい "ちか" ;
     ns1:候補名 "鈴木ちか" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "千佳" ;
     ns1:名前 "鈴木千佳" ;
@@ -3348,7 +3296,7 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/kota.suzuki.1656" ;
+    ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "すずき" ;
     ns1:めい "こうた" ;
@@ -3362,9 +3310,8 @@
     ns1:年齢 "36" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://senkyo.jcp-tokyo.net/osanai/blog/" ;
-    ns1:facebook "https://www.facebook.com/fumiko.osanai",
-        "https://www.facebook.com/osanai235" ;
+[] ns1:blog "" ;
+    ns1:facebook "https://www.facebook.com/osanai235" ;
     ns1:twitter "osanai235" ;
     ns1:せい "おさない" ;
     ns1:めい "ふみこ" ;
@@ -3375,7 +3322,7 @@
     ns1:名前 "長内史子" ;
     ns1:姓 "長内" ;
     ns1:小選挙区 "東京8区" ;
-    ns1:年齢 "30" ;
+    ns1:年齢 "29" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
@@ -3383,8 +3330,7 @@
     ns1:twitter "clfcde1eljomceq" ;
     ns1:せい "ながとも" ;
     ns1:めい "ただひろ" ;
-    ns1:候補名 "長友忠弘" ;
-    ns1:公式サイト "http://jcpweb.net/nagatomo/" ;
+    ns1:候補名 "長友ただひろ" ;
     ns1:前回 "新" ;
     ns1:名 "忠弘" ;
     ns1:名前 "長友忠弘" ;
@@ -3410,14 +3356,14 @@
 
 [] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/tine.naganuma" ;
-    ns1:twitter "tinenaganuma" ;
+    ns1:twitter "" ;
     ns1:せい "ながぬま" ;
     ns1:めい "ちね" ;
-    ns1:候補名 "長沼チネ" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "チネ" ;
-    ns1:名前 "長沼チネ" ;
+    ns1:候補名 "長沼ちね" ;
+    ns1:公式サイト "" ;
+    ns1:前回 "新" ;
+    ns1:名 "ちね" ;
+    ns1:名前 "長沼ちね" ;
     ns1:姓 "長沼" ;
     ns1:小選挙区 "埼玉7区" ;
     ns1:年齢 "67" ;
@@ -3429,7 +3375,6 @@
     ns1:せい "ながせ" ;
     ns1:めい "ゆきこ" ;
     ns1:候補名 "ながせ由希子" ;
-    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "由希子" ;
     ns1:名前 "長瀬由希子" ;
@@ -3468,7 +3413,7 @@
     ns1:年齢 "27" ;
     ns1:比例区 "" .
 
-[] ns1:blog "https://www.kamoshita.gr.jp/media/" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/kazuyuki.atou" ;
     ns1:twitter "atoukazuyuki" ;
     ns1:せい "あとう" ;
@@ -3484,13 +3429,12 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/Tokyo22abemakoto",
-        "https://www.facebook.com/makoto.abe.22" ;
+    ns1:facebook "https://www.facebook.com/makoto.abe.22" ;
     ns1:twitter "rhein2011" ;
     ns1:せい "あべ" ;
     ns1:めい "まこと" ;
     ns1:候補名 "阿部真" ;
-    ns1:公式サイト "http://senkyo.jcp-tokyo.net/abe/" ;
+    ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "真" ;
     ns1:名前 "阿部真" ;
@@ -3500,12 +3444,11 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/junichi.shimotori.1",
-        "https://www.facebook.com/nara2shimotori" ;
+    ns1:facebook "https://www.facebook.com/nara2shimotori" ;
     ns1:twitter "" ;
     ns1:せい "しもとり" ;
     ns1:めい "じゅんいち" ;
-    ns1:候補名 "霜鳥純一" ;
+    ns1:候補名 "しもとり純一" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "純一" ;
@@ -3516,16 +3459,16 @@
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "" ;
+    ns1:facebook "https://www.facebook.com/ryosuke.aoyama.779" ;
     ns1:twitter "aoyama_ryosuke" ;
-    ns1:せい "あおやま" ;
-    ns1:めい "りょうすけ" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
     ns1:候補名 "青山了介" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
-    ns1:名 "了介" ;
+    ns1:名 "" ;
     ns1:名前 "青山了介" ;
-    ns1:姓 "青山" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "富山1区" ;
     ns1:年齢 "42" ;
     ns1:比例区 "" .
@@ -3534,8 +3477,8 @@
     ns1:facebook "https://www.facebook.com/kohei.aoyama.90" ;
     ns1:twitter "iyaiya_55" ;
     ns1:せい "あおやま" ;
-    ns1:めい "こうへい" ;
-    ns1:候補名 "青山昂平" ;
+    ns1:めい "コウヘイ" ;
+    ns1:候補名 "青山コウヘイ" ;
     ns1:公式サイト "http://kouhei.jcp-ota.jp/" ;
     ns1:前回 "新" ;
     ns1:名 "昂平" ;
@@ -3550,7 +3493,7 @@
     ns1:twitter "" ;
     ns1:せい "あおき" ;
     ns1:めい "ひろし" ;
-    ns1:候補名 "青木弘" ;
+    ns1:候補名 "青木ひろし" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "弘" ;
@@ -3560,14 +3503,12 @@
     ns1:年齢 "56" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://senkyo.jcp-tokyo.net/iida/blog/" ;
-    ns1:facebook "https://www.facebook.com/miyako.iida.9",
-        "https://www.facebook.com/八法亭-みややっこ-729898467032255" ;
+[] ns1:blog "" ;
+    ns1:facebook "" ;
     ns1:twitter "iidamiyako1" ;
     ns1:せい "いいだ" ;
     ns1:めい "みやこ" ;
-    ns1:候補名 "飯田美弥子" ;
-    ns1:公式サイト "http://senkyo.jcp-tokyo.net/iida/" ;
+    ns1:候補名 "飯田みやこ" ;
     ns1:前回 "新" ;
     ns1:名 "美弥子" ;
     ns1:名前 "飯田美弥子" ;
@@ -3581,22 +3522,21 @@
     ns1:twitter "kouzai2007" ;
     ns1:せい "こうざい" ;
     ns1:めい "かつすけ" ;
-    ns1:候補名 "香西克介" ;
-    ns1:公式サイト "" ;
-    ns1:前回 "新" ;
-    ns1:名 "克介" ;
-    ns1:名前 "香西克介" ;
+    ns1:候補名 "香西かつ介" ;
+    ns1:前回 "新" ;
+    ns1:名 "介克" ;
+    ns1:名前 "香西介克" ;
     ns1:姓 "香西" ;
     ns1:小選挙区 "東京3区" ;
     ns1:年齢 "41" ;
     ns1:比例区 "" .
 
 [] ns1:blog "" ;
-    ns1:facebook "https://www.facebook.com/profile.php?id=100016054359692" ;
+    ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "たかやなぎ" ;
     ns1:めい "ひろあき" ;
-    ns1:候補名 "高柳博明" ;
+    ns1:候補名 "高柳ひろあき" ;
     ns1:公式サイト "" ;
     ns1:前回 "新" ;
     ns1:名 "博明" ;
@@ -3666,7 +3606,7 @@
     ns1:年齢 "69" ;
     ns1:比例区 "" .
 
-[] ns1:blog "http://hanayutadaichan.strikingly.com/#_blog" ;
+[] ns1:blog "" ;
     ns1:facebook "https://www.facebook.com/s.k.daigo" ;
     ns1:twitter "hana_daichan" ;
     ns1:せい "さいとう" ;
