--- GrayDB
+++ 共産党公式
@@ -4,93 +4,106 @@
 @prefix xml: <http://www.w3.org/XML/1998/namespace> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100006085722650" ;
     ns1:twitter "chiduko916" ;
     ns1:せい "たかはし" ;
-    ns1:めい "ちづこ" ;
-    ns1:候補名 "高橋千鶴子" ;
+    ns1:めい "ちづ" ;
+    ns1:候補名 "高橋ちづ子" ;
     ns1:前回 "前" ;
-    ns1:名 "千鶴子" ;
-    ns1:名前 "高橋千鶴子" ;
+    ns1:名 "子千鶴子" ;
+    ns1:名前 "高橋子千鶴子" ;
     ns1:姓 "高橋" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "58" ;
+    ns1:年齢 "57" ;
     ns1:比例区 "東北" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/tsuyako.miyazima" ;
     ns1:twitter "tsuyako1127" ;
     ns1:せい "みやじま" ;
-    ns1:めい "つやこ" ;
+    ns1:めい "つやつやこ" ;
     ns1:候補名 "宮嶋つや子" ;
     ns1:前回 "新" ;
-    ns1:名 "つや子" ;
-    ns1:名前 "宮嶋つや子" ;
+    ns1:名 "子" ;
+    ns1:名前 "宮嶋子" ;
     ns1:姓 "宮嶋" ;
     ns1:小選挙区 "福岡8区" ;
     ns1:年齢 "66" ;
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
+    ns1:候補名 "こみやま恵美子" ;
+    ns1:前回 "新" ;
+    ns1:名 "" ;
+    ns1:名前 "こみやま恵美子" ;
+    ns1:姓 "" ;
+    ns1:小選挙区 "岐阜4区" ;
+    ns1:年齢 "62" ;
+    ns1:比例区 "" .
+
+[] ns1:facebook "https://www.facebook.com/s.k.daigo" ;
     ns1:twitter "hana_daichan" ;
     ns1:せい "さいとう" ;
     ns1:めい "だいご" ;
     ns1:候補名 "さいとう大悟" ;
     ns1:前回 "新" ;
     ns1:名 "大悟" ;
-    ns1:名前 "斉藤大悟" ;
-    ns1:姓 "斉藤" ;
+    ns1:名前 "齊藤大悟" ;
+    ns1:姓 "齊藤" ;
     ns1:小選挙区 "秋田1区" ;
-    ns1:年齢 "37" ;
+    ns1:年齢 "38" ;
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "さいとう" ;
     ns1:めい "のどか" ;
-    ns1:候補名 "斎藤温" ;
+    ns1:候補名 "斉藤のどか" ;
     ns1:前回 "新" ;
     ns1:名 "温" ;
-    ns1:名前 "斎藤温" ;
-    ns1:姓 "斎藤" ;
+    ns1:名前 "斉藤温" ;
+    ns1:姓 "斉藤" ;
     ns1:小選挙区 "神奈川9区" ;
     ns1:年齢 "26" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "ikunotennohji" ;
+[] ns1:facebook "https://www.facebook.com/noboru.shibayama.9" ;
+    ns1:twitter "" ;
     ns1:せい "しばやま" ;
     ns1:めい "のぼる" ;
     ns1:候補名 "しば山のぼる" ;
     ns1:前回 "新" ;
     ns1:名 "昇" ;
-    ns1:名前 "柴山昇" ;
-    ns1:姓 "柴山" ;
+    ns1:名前 "山柴昇" ;
+    ns1:姓 "山柴" ;
     ns1:小選挙区 "大阪1区" ;
     ns1:年齢 "63" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/yasuomi.shimasaki" ;
     ns1:twitter "sima_omi" ;
     ns1:せい "しまさき" ;
     ns1:めい "やすおみ" ;
-    ns1:候補名 "島崎保臣" ;
+    ns1:候補名 "島﨑やすおみ" ;
     ns1:前回 "新" ;
     ns1:名 "保臣" ;
-    ns1:名前 "島崎保臣" ;
-    ns1:姓 "島崎" ;
+    ns1:名前 "島﨑保臣" ;
+    ns1:姓 "島﨑" ;
     ns1:小選挙区 "" ;
     ns1:年齢 "34" ;
     ns1:比例区 "四国" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "tak1381" ;
-    ns1:せい "たかむら" ;
-    ns1:めい "なおや" ;
+    ns1:twitter "" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
     ns1:候補名 "たかむら直也" ;
     ns1:前回 "新" ;
-    ns1:名 "直也" ;
-    ns1:名前 "高村直也" ;
-    ns1:姓 "高村" ;
+    ns1:名 "" ;
+    ns1:名前 "たかむら直也" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "宮城4区" ;
     ns1:年齢 "34" ;
     ns1:比例区 "" .
@@ -102,40 +115,79 @@
     ns1:候補名 "ため仁史" ;
     ns1:前回 "新" ;
     ns1:名 "仁史" ;
-    ns1:名前 "為仁史" ;
-    ns1:姓 "為" ;
+    ns1:名前 "爲仁史" ;
+    ns1:姓 "爲" ;
     ns1:小選挙区 "大阪15区" ;
     ns1:年齢 "68" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/hiroshi.nishizawa.3910" ;
     ns1:twitter "melf4511" ;
     ns1:せい "にしざわ" ;
     ns1:めい "ひろし" ;
     ns1:候補名 "にしざわ博" ;
     ns1:前回 "新" ;
     ns1:名 "博" ;
-    ns1:名前 "西沢博" ;
-    ns1:姓 "西沢" ;
-    ns1:小選挙区 "" ;
-    ns1:年齢 "37" ;
+    ns1:名前 "西澤博" ;
+    ns1:姓 "西澤" ;
+    ns1:小選挙区 "" ;
+    ns1:年齢 "36" ;
     ns1:比例区 "北陸信越" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
+    ns1:候補名 "のざわ康幸" ;
+    ns1:前回 "新" ;
+    ns1:名 "" ;
+    ns1:名前 "のざわ康幸" ;
+    ns1:姓 "" ;
+    ns1:小選挙区 "愛知15区" ;
+    ns1:年齢 "29" ;
+    ns1:比例区 "" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "のざわ" ;
     ns1:めい "まさじ" ;
-    ns1:候補名 "野沢正司" ;
+    ns1:候補名 "のざわ正司" ;
     ns1:前回 "新" ;
     ns1:名 "正司" ;
-    ns1:名前 "野沢正司" ;
-    ns1:姓 "野沢" ;
+    ns1:名前 "野澤正司" ;
+    ns1:姓 "野澤" ;
     ns1:小選挙区 "静岡7区" ;
     ns1:年齢 "68" ;
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "issikikazumasa" ;
+    ns1:twitter "" ;
+    ns1:せい "テルヤ" ;
+    ns1:めい "かんとく" ;
+    ns1:候補名 "テルヤ寛徳" ;
+    ns1:前回 "前" ;
+    ns1:名 "寛徳" ;
+    ns1:名前 "照屋寛徳" ;
+    ns1:姓 "照屋" ;
+    ns1:小選挙区 "沖縄2区(推薦、オール沖縄)" ;
+    ns1:年齢 "72" ;
+    ns1:比例区 "" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:せい "ナカザト" ;
+    ns1:めい "としのぶ" ;
+    ns1:候補名 "ナカザト利信" ;
+    ns1:前回 "前" ;
+    ns1:名 "利信" ;
+    ns1:名前 "仲里利信" ;
+    ns1:姓 "仲里" ;
+    ns1:小選挙区 "沖縄4区(推薦、オール沖縄)" ;
+    ns1:年齢 "80" ;
+    ns1:比例区 "" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:せい "いっしき" ;
     ns1:めい "かずまさ" ;
     ns1:候補名 "一色一正" ;
@@ -147,7 +199,7 @@
     ns1:年齢 "67" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/tomoyuki.miwa3" ;
     ns1:twitter "1975_miwa" ;
     ns1:せい "みわ" ;
     ns1:めい "ともゆき" ;
@@ -174,13 +226,13 @@
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "satiueda29" ;
+    ns1:twitter "" ;
     ns1:せい "うえだ" ;
     ns1:めい "さちこ" ;
-    ns1:候補名 "上田幸子" ;
-    ns1:前回 "新" ;
-    ns1:名 "幸子" ;
-    ns1:名前 "上田幸子" ;
+    ns1:候補名 "上田さち子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子幸" ;
+    ns1:名前 "上田子幸" ;
     ns1:姓 "上田" ;
     ns1:小選挙区 "兵庫7区" ;
     ns1:年齢 "69" ;
@@ -190,7 +242,7 @@
     ns1:twitter "" ;
     ns1:せい "うえの" ;
     ns1:めい "ひろつぐ" ;
-    ns1:候補名 "上野洋嗣" ;
+    ns1:候補名 "上野ひろつぐ" ;
     ns1:前回 "新" ;
     ns1:名 "洋嗣" ;
     ns1:名前 "上野洋嗣" ;
@@ -212,11 +264,11 @@
     ns1:年齢 "71" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "wvzurewxzk7lisw" ;
+[] ns1:facebook "https://www.facebook.com/shimomura.masahiro.54" ;
+    ns1:twitter "" ;
     ns1:せい "しもむら" ;
     ns1:めい "まさひろ" ;
-    ns1:候補名 "下村雅洋" ;
+    ns1:候補名 "下村まさひろ" ;
     ns1:前回 "新" ;
     ns1:名 "雅洋" ;
     ns1:名前 "下村雅洋" ;
@@ -226,10 +278,10 @@
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "ojskdc90t9hnbra" ;
+    ns1:twitter "" ;
     ns1:せい "なかいし" ;
     ns1:めい "ひとし" ;
-    ns1:候補名 "中石仁" ;
+    ns1:候補名 "中石ひとし" ;
     ns1:前回 "新" ;
     ns1:名 "仁" ;
     ns1:名前 "中石仁" ;
@@ -242,20 +294,20 @@
     ns1:twitter "" ;
     ns1:せい "なかの" ;
     ns1:めい "たけし" ;
-    ns1:候補名 "中野武史" ;
+    ns1:候補名 "中野たけし" ;
     ns1:前回 "新" ;
     ns1:名 "武史" ;
     ns1:名前 "中野武史" ;
     ns1:姓 "中野" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "43" ;
+    ns1:年齢 "42" ;
     ns1:比例区 "東海" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "なかのわたり" ;
     ns1:めい "じゅん" ;
-    ns1:候補名 "中野渡旬" ;
+    ns1:候補名 "中野渡じゅん" ;
     ns1:前回 "新" ;
     ns1:名 "旬" ;
     ns1:名前 "中野渡旬" ;
@@ -264,11 +316,11 @@
     ns1:年齢 "69" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/jcp.tokushima" ;
     ns1:twitter "" ;
     ns1:せい "くぼ" ;
     ns1:めい "たかゆき" ;
-    ns1:候補名 "久保孝之" ;
+    ns1:候補名 "久保たかゆき" ;
     ns1:前回 "新" ;
     ns1:名 "孝之" ;
     ns1:名前 "久保孝之" ;
@@ -277,8 +329,8 @@
     ns1:年齢 "54" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "efisgfiseeedhde" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100004099751374" ;
+    ns1:twitter "" ;
     ns1:せい "いがらし" ;
     ns1:めい "たけひこ" ;
     ns1:候補名 "五十嵐たけひこ" ;
@@ -303,11 +355,11 @@
     ns1:年齢 "61" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/takashi.inoue.jcp" ;
     ns1:twitter "inoue_9" ;
     ns1:せい "いのうえ" ;
     ns1:めい "たかし" ;
-    ns1:候補名 "井上宣" ;
+    ns1:候補名 "井上たかし" ;
     ns1:前回 "新" ;
     ns1:名 "宣" ;
     ns1:名前 "井上宣" ;
@@ -316,7 +368,7 @@
     ns1:年齢 "43" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100008607883601" ;
     ns1:twitter "" ;
     ns1:せい "いのうえ" ;
     ns1:めい "よしこ" ;
@@ -342,7 +394,7 @@
     ns1:年齢 "50" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/iguchi.shizuoka" ;
     ns1:twitter "" ;
     ns1:せい "いぐち" ;
     ns1:めい "まさひこ" ;
@@ -385,7 +437,7 @@
     ns1:twitter "" ;
     ns1:せい "いとう" ;
     ns1:めい "まこと" ;
-    ns1:候補名 "伊藤誠" ;
+    ns1:候補名 "いとう誠" ;
     ns1:前回 "新" ;
     ns1:名 "誠" ;
     ns1:名前 "伊藤誠" ;
@@ -394,11 +446,11 @@
     ns1:年齢 "43" ;
     ns1:比例区 "北陸信越" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/tatsuya.jcp.g" ;
     ns1:twitter "tatsuya_jcp" ;
     ns1:せい "いとう" ;
     ns1:めい "たつや" ;
-    ns1:候補名 "伊藤達也" ;
+    ns1:候補名 "伊藤たつや" ;
     ns1:前回 "新" ;
     ns1:名 "達也" ;
     ns1:名前 "伊藤達也" ;
@@ -407,11 +459,11 @@
     ns1:年齢 "34" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "peace12wave" ;
+[] ns1:facebook "https://www.facebook.com/akemi.sasaki.1656" ;
+    ns1:twitter "" ;
     ns1:せい "ささき" ;
     ns1:めい "あけみ" ;
-    ns1:候補名 "佐々木明美" ;
+    ns1:候補名 "佐々木あけみ" ;
     ns1:前回 "新" ;
     ns1:名 "明美" ;
     ns1:名前 "佐々木明美" ;
@@ -450,7 +502,7 @@
     ns1:twitter "" ;
     ns1:せい "ふじたに" ;
     ns1:めい "かえこ" ;
-    ns1:候補名 "冨士谷香恵子" ;
+    ns1:候補名 "ふじたに香恵子" ;
     ns1:前回 "新" ;
     ns1:名 "香恵子" ;
     ns1:名前 "冨士谷香恵子" ;
@@ -463,7 +515,7 @@
     ns1:twitter "" ;
     ns1:せい "とみおか" ;
     ns1:めい "あきら" ;
-    ns1:候補名 "冨岡昭" ;
+    ns1:候補名 "冨岡あきら" ;
     ns1:前回 "新" ;
     ns1:名 "昭" ;
     ns1:名前 "冨岡昭" ;
@@ -472,11 +524,11 @@
     ns1:年齢 "67" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/naoki.tomita.503" ;
     ns1:twitter "tomita_naoki" ;
     ns1:せい "とみた" ;
     ns1:めい "なおき" ;
-    ns1:候補名 "冨田直樹" ;
+    ns1:候補名 "冨田なおき" ;
     ns1:前回 "新" ;
     ns1:名 "直樹" ;
     ns1:名前 "冨田直樹" ;
@@ -486,10 +538,10 @@
     ns1:比例区 "東京" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "rikiriki1005" ;
+    ns1:twitter "" ;
     ns1:せい "りきしげ" ;
     ns1:めい "ともゆき" ;
-    ns1:候補名 "力重智之" ;
+    ns1:候補名 "りきしげ智之" ;
     ns1:前回 "新" ;
     ns1:名 "智之" ;
     ns1:名前 "力重智之" ;
@@ -498,10 +550,10 @@
     ns1:年齢 "39" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/rikakatoh/" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100010277866022" ;
     ns1:twitter "" ;
     ns1:せい "かとう" ;
-    ns1:めい "りか" ;
+    ns1:めい "リカ" ;
     ns1:候補名 "加藤リカ" ;
     ns1:前回 "新" ;
     ns1:名 "リカ" ;
@@ -524,7 +576,7 @@
     ns1:年齢 "66" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/kitayamasupport" ;
     ns1:twitter "jcpkitayama" ;
     ns1:せい "きたやま" ;
     ns1:めい "りょうぞう" ;
@@ -537,7 +589,7 @@
     ns1:年齢 "65" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/miki.t.kitamura" ;
     ns1:twitter "0620_appletree" ;
     ns1:せい "きたむら" ;
     ns1:めい "みき" ;
@@ -550,20 +602,20 @@
     ns1:年齢 "51" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=1209855673" ;
     ns1:twitter "" ;
     ns1:せい "はら" ;
     ns1:めい "やすひさ" ;
-    ns1:候補名 "原矢寸久" ;
+    ns1:候補名 "原やすひさ" ;
     ns1:前回 "新" ;
     ns1:名 "矢寸久" ;
     ns1:名前 "原矢寸久" ;
     ns1:姓 "原" ;
     ns1:小選挙区 "和歌山1区" ;
-    ns1:年齢 "65" ;
+    ns1:年齢 "66" ;
     ns1:比例区 "近畿" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/kukure.hara" ;
     ns1:twitter "harachanj" ;
     ns1:せい "はら" ;
     ns1:めい "じゅんこ" ;
@@ -580,7 +632,7 @@
     ns1:twitter "" ;
     ns1:せい "ふるかわ" ;
     ns1:めい "よしのり" ;
-    ns1:候補名 "古川芳憲" ;
+    ns1:候補名 "古川よしのり" ;
     ns1:前回 "新" ;
     ns1:名 "芳憲" ;
     ns1:名前 "古川芳憲" ;
@@ -593,7 +645,7 @@
     ns1:twitter "" ;
     ns1:せい "ふるさわ" ;
     ns1:めい "よしゆき" ;
-    ns1:候補名 "古沢喜幸" ;
+    ns1:候補名 "古沢よしゆき" ;
     ns1:前回 "新" ;
     ns1:名 "喜幸" ;
     ns1:名前 "古沢喜幸" ;
@@ -602,7 +654,7 @@
     ns1:年齢 "71" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/5555yoshida/" ;
     ns1:twitter "gogoyoshidago" ;
     ns1:せい "よしだ" ;
     ns1:めい "ごう" ;
@@ -615,11 +667,11 @@
     ns1:年齢 "35" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/jcp.yosida.toshio/" ;
-    ns1:twitter "yoshida_toshio" ;
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:せい "よしだ" ;
     ns1:めい "としお" ;
-    ns1:候補名 "吉田年男" ;
+    ns1:候補名 "吉田としお" ;
     ns1:前回 "新" ;
     ns1:名 "年男" ;
     ns1:名前 "吉田年男" ;
@@ -654,7 +706,7 @@
     ns1:年齢 "36" ;
     ns1:比例区 "東北" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/shinichi.mukose" ;
     ns1:twitter "mukose3" ;
     ns1:せい "むこせ" ;
     ns1:めい "しんいち" ;
@@ -667,11 +719,11 @@
     ns1:年齢 "46" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/cyuubutiku" ;
     ns1:twitter "" ;
     ns1:せい "よつや" ;
     ns1:めい "めぐみ" ;
-    ns1:候補名 "四ツ谷恵" ;
+    ns1:候補名 "よつや恵" ;
     ns1:前回 "新" ;
     ns1:名 "恵" ;
     ns1:名前 "四ツ谷恵" ;
@@ -707,7 +759,7 @@
     ns1:比例区 "近畿" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "waka929" ;
+    ns1:twitter "" ;
     ns1:せい "さかい" ;
     ns1:めい "わかこ" ;
     ns1:候補名 "坂井和歌子" ;
@@ -716,7 +768,7 @@
     ns1:名前 "坂井和歌子" ;
     ns1:姓 "坂井" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "40" ;
+    ns1:年齢 "39" ;
     ns1:比例区 "東京" .
 
 [] ns1:facebook "https://www.facebook.com/55hiroshi/" ;
@@ -732,7 +784,7 @@
     ns1:年齢 "47" ;
     ns1:比例区 "北陸信越" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/gu.jingmei" ;
     ns1:twitter "haruka_tubasa" ;
     ns1:せい "かきうち" ;
     ns1:めい "きょうみ" ;
@@ -742,10 +794,10 @@
     ns1:名前 "垣内京美" ;
     ns1:姓 "垣内" ;
     ns1:小選挙区 "岡山2区" ;
-    ns1:年齢 "50" ;
+    ns1:年齢 "51" ;
     ns1:比例区 "中国" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/terufumi.horiuchi" ;
     ns1:twitter "teruhoriuchi" ;
     ns1:せい "ほりうち" ;
     ns1:めい "てるふみ" ;
@@ -762,7 +814,7 @@
     ns1:twitter "" ;
     ns1:せい "ほりえ" ;
     ns1:めい "かくじ" ;
-    ns1:候補名 "堀江鶴治" ;
+    ns1:候補名 "堀江かくじ" ;
     ns1:前回 "新" ;
     ns1:名 "鶴治" ;
     ns1:名前 "堀江鶴治" ;
@@ -775,7 +827,7 @@
     ns1:twitter "" ;
     ns1:せい "ほり" ;
     ns1:めい "ゆずる" ;
-    ns1:候補名 "堀譲" ;
+    ns1:候補名 "堀ゆずる" ;
     ns1:前回 "新" ;
     ns1:名 "譲" ;
     ns1:名前 "堀譲" ;
@@ -788,7 +840,7 @@
     ns1:twitter "aozoranaru" ;
     ns1:せい "つかだ" ;
     ns1:めい "なるゆき" ;
-    ns1:候補名 "塚田成幸" ;
+    ns1:候補名 "塚田なるゆき" ;
     ns1:前回 "新" ;
     ns1:名 "成幸" ;
     ns1:名前 "塚田成幸" ;
@@ -797,7 +849,7 @@
     ns1:年齢 "53" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/shiokawa.jcp" ;
     ns1:twitter "shiokawatetsuya" ;
     ns1:せい "しおかわ" ;
     ns1:めい "てつや" ;
@@ -807,7 +859,7 @@
     ns1:名前 "塩川鉄也" ;
     ns1:姓 "塩川" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "55" ;
+    ns1:年齢 "54" ;
     ns1:比例区 "北関東" .
 
 [] ns1:facebook "" ;
@@ -827,10 +879,10 @@
     ns1:twitter "" ;
     ns1:せい "おおうち" ;
     ns1:めい "くみこ" ;
-    ns1:候補名 "大内久美子" ;
-    ns1:前回 "新" ;
-    ns1:名 "久美子" ;
-    ns1:名前 "大内久美子" ;
+    ns1:候補名 "大内くみ子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子久美" ;
+    ns1:名前 "大内子久美" ;
     ns1:姓 "大内" ;
     ns1:小選挙区 "茨城1区" ;
     ns1:年齢 "68" ;
@@ -840,13 +892,13 @@
     ns1:twitter "tekuteku_diary" ;
     ns1:せい "おおひら" ;
     ns1:めい "よしのぶ" ;
-    ns1:候補名 "大平喜信" ;
+    ns1:候補名 "大平よしのぶ" ;
     ns1:前回 "前" ;
     ns1:名 "喜信" ;
     ns1:名前 "大平喜信" ;
     ns1:姓 "大平" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "39" ;
+    ns1:年齢 "38" ;
     ns1:比例区 "中国" .
 
 [] ns1:facebook "" ;
@@ -866,7 +918,7 @@
     ns1:twitter "" ;
     ns1:せい "おおすぎ" ;
     ns1:めい "てつお" ;
-    ns1:候補名 "大椙鉄夫" ;
+    ns1:候補名 "おおすぎ鉄夫" ;
     ns1:前回 "新" ;
     ns1:名 "鉄夫" ;
     ns1:名前 "大椙鉄夫" ;
@@ -878,8 +930,8 @@
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "おおにし" ;
-    ns1:めい "おさむ" ;
-    ns1:候補名 "大西理" ;
+    ns1:めい "オサム" ;
+    ns1:候補名 "大西オサム" ;
     ns1:前回 "新" ;
     ns1:名 "理" ;
     ns1:名前 "大西理" ;
@@ -888,11 +940,11 @@
     ns1:年齢 "51" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "http://www.facebook.com/wataame.hare" ;
     ns1:twitter "wataame_hare" ;
     ns1:せい "おおにし" ;
     ns1:めい "わたる" ;
-    ns1:候補名 "大西航" ;
+    ns1:候補名 "大西わたる" ;
     ns1:前回 "新" ;
     ns1:名 "航" ;
     ns1:名前 "大西航" ;
@@ -902,7 +954,7 @@
     ns1:比例区 "南関東" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "kiyofumi_ohnuki" ;
+    ns1:twitter "" ;
     ns1:せい "おおぬき" ;
     ns1:めい "きよふみ" ;
     ns1:候補名 "おおぬき清文" ;
@@ -914,11 +966,11 @@
     ns1:年齢 "60" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/hiromitsu.ohno.7" ;
     ns1:twitter "" ;
     ns1:せい "おおの" ;
     ns1:めい "ひろみつ" ;
-    ns1:候補名 "大野宙光" ;
+    ns1:候補名 "大野ひろみつ" ;
     ns1:前回 "新" ;
     ns1:名 "宙光" ;
     ns1:名前 "大野宙光" ;
@@ -927,11 +979,11 @@
     ns1:年齢 "54" ;
     ns1:比例区 "東海" .
 
-[] ns1:facebook "https://www.facebook.com/oono.takasi.jcp/" ;
+[] ns1:facebook "" ;
     ns1:twitter "koredeiinoka" ;
     ns1:せい "おおの" ;
     ns1:めい "たかし" ;
-    ns1:候補名 "大野隆" ;
+    ns1:候補名 "大野たかし" ;
     ns1:前回 "新" ;
     ns1:名 "隆" ;
     ns1:名前 "大野隆" ;
@@ -941,11 +993,10 @@
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "okumotonaomi",
-        "pekoandnaomi" ;
+    ns1:twitter "" ;
     ns1:せい "おくもと" ;
     ns1:めい "なおみ" ;
-    ns1:候補名 "奥本菜保巳" ;
+    ns1:候補名 "奥本なおみ" ;
     ns1:前回 "新" ;
     ns1:名 "菜保巳" ;
     ns1:名前 "奥本菜保巳" ;
@@ -954,11 +1005,11 @@
     ns1:年齢 "57" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/gen.miyauchi" ;
     ns1:twitter "genmiyauchi1982" ;
     ns1:せい "みやうち" ;
     ns1:めい "げん" ;
-    ns1:候補名 "宮内現" ;
+    ns1:候補名 "宮内げん" ;
     ns1:前回 "新" ;
     ns1:名 "現" ;
     ns1:名前 "宮内現" ;
@@ -971,7 +1022,7 @@
     ns1:twitter "" ;
     ns1:せい "みやち" ;
     ns1:めい "いさお" ;
-    ns1:候補名 "宮地勲" ;
+    ns1:候補名 "宮地いさお" ;
     ns1:前回 "新" ;
     ns1:名 "勲" ;
     ns1:名前 "宮地勲" ;
@@ -980,17 +1031,17 @@
     ns1:年齢 "63" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/takeshi.miyamoto.5661" ;
     ns1:twitter "" ;
     ns1:せい "みやもと" ;
     ns1:めい "たけし" ;
-    ns1:候補名 "宮本岳志" ;
+    ns1:候補名 "宮本たけし" ;
     ns1:前回 "前" ;
     ns1:名 "岳志" ;
     ns1:名前 "宮本岳志" ;
     ns1:姓 "宮本" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "57" ;
+    ns1:年齢 "56" ;
     ns1:比例区 "近畿" .
 
 [] ns1:facebook "https://www.facebook.com/miyamototooru.jcp" ;
@@ -1010,19 +1061,19 @@
     ns1:twitter "" ;
     ns1:せい "てらだ" ;
     ns1:めい "あきみつ" ;
-    ns1:候補名 "寺田明充" ;
+    ns1:候補名 "寺田あきみつ" ;
     ns1:前回 "新" ;
     ns1:名 "明充" ;
     ns1:名前 "寺田明充" ;
     ns1:姓 "寺田" ;
     ns1:小選挙区 "広島6区" ;
-    ns1:年齢 "66" ;
+    ns1:年齢 "65" ;
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
     ns1:twitter "azuma_united_fc" ;
     ns1:せい "こづつみ" ;
-    ns1:めい "あずま" ;
+    ns1:めい "あづま" ;
     ns1:候補名 "小堤東" ;
     ns1:前回 "新" ;
     ns1:名 "東" ;
@@ -1059,20 +1110,20 @@
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "tokikobayashi" ;
+    ns1:twitter "" ;
     ns1:せい "こばやし" ;
     ns1:めい "ときこ" ;
-    ns1:候補名 "小林解子" ;
-    ns1:前回 "新" ;
-    ns1:名 "解子" ;
-    ns1:名前 "小林解子" ;
+    ns1:候補名 "小林とき子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子解" ;
+    ns1:名前 "小林子解" ;
     ns1:姓 "小林" ;
     ns1:小選挙区 "福岡6区" ;
     ns1:年齢 "37" ;
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "koike_kazunori" ;
+    ns1:twitter "" ;
     ns1:せい "こいけ" ;
     ns1:めい "かずのり" ;
     ns1:候補名 "小池一徳" ;
@@ -1088,7 +1139,7 @@
     ns1:twitter "" ;
     ns1:せい "おのざと" ;
     ns1:めい "ていりょう" ;
-    ns1:候補名 "小野里定良" ;
+    ns1:候補名 "小野里ていりょう" ;
     ns1:前回 "新" ;
     ns1:名 "定良" ;
     ns1:名前 "小野里定良" ;
@@ -1097,11 +1148,11 @@
     ns1:年齢 "70" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/yuka.koganezawa.1" ;
     ns1:twitter "koganezawayuka" ;
     ns1:せい "こがねざわ" ;
     ns1:めい "ゆか" ;
-    ns1:候補名 "小金沢由佳" ;
+    ns1:候補名 "小金沢ゆか" ;
     ns1:前回 "新" ;
     ns1:名 "由佳" ;
     ns1:名前 "小金沢由佳" ;
@@ -1123,14 +1174,14 @@
     ns1:年齢 "64" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/people/Hiroko-Ozaki/100005889727144" ;
     ns1:twitter "" ;
     ns1:せい "おざき" ;
     ns1:めい "ひろこ" ;
-    ns1:候補名 "尾崎宏子" ;
-    ns1:前回 "新" ;
-    ns1:名 "宏子" ;
-    ns1:名前 "尾崎宏子" ;
+    ns1:候補名 "尾崎ひろ子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子宏" ;
+    ns1:名前 "尾崎子宏" ;
     ns1:姓 "尾崎" ;
     ns1:小選挙区 "岡山3区" ;
     ns1:年齢 "61" ;
@@ -1153,7 +1204,7 @@
     ns1:twitter "" ;
     ns1:せい "やました" ;
     ns1:めい "かい" ;
-    ns1:候補名 "山下魁" ;
+    ns1:候補名 "山下かい" ;
     ns1:前回 "新" ;
     ns1:名 "魁" ;
     ns1:名前 "山下魁" ;
@@ -1181,8 +1232,8 @@
     ns1:めい "ゆうた" ;
     ns1:候補名 "山口ゆうた" ;
     ns1:前回 "新" ;
-    ns1:名 "勇太" ;
-    ns1:名前 "山口勇太" ;
+    ns1:名 "ゆうた" ;
+    ns1:名前 "山口ゆうた" ;
     ns1:姓 "山口" ;
     ns1:小選挙区 "鹿児島3区" ;
     ns1:年齢 "26" ;
@@ -1192,7 +1243,7 @@
     ns1:twitter "" ;
     ns1:せい "やまざき" ;
     ns1:めい "としひこ" ;
-    ns1:候補名 "山崎寿彦" ;
+    ns1:候補名 "山崎としひこ" ;
     ns1:前回 "新" ;
     ns1:名 "寿彦" ;
     ns1:名前 "山崎寿彦" ;
@@ -1201,7 +1252,7 @@
     ns1:年齢 "64" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/100005512444252" ;
     ns1:twitter "yamachi0102" ;
     ns1:せい "やまもと" ;
     ns1:めい "ちよこ" ;
@@ -1214,7 +1265,7 @@
     ns1:年齢 "68" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/yamakoshi.toru" ;
     ns1:twitter "" ;
     ns1:せい "やまこし" ;
     ns1:めい "とおる" ;
@@ -1227,11 +1278,11 @@
     ns1:年齢 "47" ;
     ns1:比例区 "東海" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/jcp20141214okazaki/" ;
     ns1:twitter "" ;
     ns1:せい "おかざき" ;
     ns1:めい "ゆたか" ;
-    ns1:候補名 "岡崎裕" ;
+    ns1:候補名 "岡崎ゆたか" ;
     ns1:前回 "新" ;
     ns1:名 "裕" ;
     ns1:名前 "岡崎裕" ;
@@ -1244,7 +1295,7 @@
     ns1:twitter "" ;
     ns1:せい "いわもと" ;
     ns1:めい "こうじ" ;
-    ns1:候補名 "岩本康嗣" ;
+    ns1:候補名 "岩本こうじ" ;
     ns1:前回 "新" ;
     ns1:名 "康嗣" ;
     ns1:名前 "岩本康嗣" ;
@@ -1253,11 +1304,11 @@
     ns1:年齢 "52" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100009478396899" ;
     ns1:twitter "" ;
     ns1:せい "いわた" ;
     ns1:めい "たけし" ;
-    ns1:候補名 "岩田剛" ;
+    ns1:候補名 "岩田たけし" ;
     ns1:前回 "新" ;
     ns1:名 "剛" ;
     ns1:名前 "岩田剛" ;
@@ -1266,11 +1317,11 @@
     ns1:年齢 "36" ;
     ns1:比例区 "中国" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100011583347889" ;
     ns1:twitter "kisskisi" ;
     ns1:せい "きし" ;
     ns1:めい "たけし" ;
-    ns1:候補名 "岸武志" ;
+    ns1:候補名 "岸たけし" ;
     ns1:前回 "新" ;
     ns1:名 "武志" ;
     ns1:名前 "岸武志" ;
@@ -1279,8 +1330,8 @@
     ns1:年齢 "46" ;
     ns1:比例区 "東京" .
 
-[] ns1:facebook "https://www.facebook.com/岸良信-衆院東京10区日本共産党予定候補-472646869788614" ;
-    ns1:twitter "jcptokyo10" ;
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:せい "きし" ;
     ns1:めい "よしのぶ" ;
     ns1:候補名 "岸良信" ;
@@ -1292,7 +1343,7 @@
     ns1:年齢 "62" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/shimazu.yukihiro.jcp" ;
     ns1:twitter "u_simazu" ;
     ns1:せい "しまづ" ;
     ns1:めい "ゆきひろ" ;
@@ -1305,7 +1356,7 @@
     ns1:年齢 "60" ;
     ns1:比例区 "東海" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/jcpshizu.seibu" ;
     ns1:twitter "" ;
     ns1:せい "しまだ" ;
     ns1:めい "はつえ" ;
@@ -1322,7 +1373,7 @@
     ns1:twitter "" ;
     ns1:せい "かわかみ" ;
     ns1:めい "ひとし" ;
-    ns1:候補名 "川上均" ;
+    ns1:候補名 "川上ひとし" ;
     ns1:前回 "新" ;
     ns1:名 "均" ;
     ns1:名前 "川上均" ;
@@ -1335,20 +1386,20 @@
     ns1:twitter "jyuuouumare" ;
     ns1:せい "かわさき" ;
     ns1:めい "あつこ" ;
-    ns1:候補名 "川崎篤子" ;
-    ns1:前回 "新" ;
-    ns1:名 "篤子" ;
-    ns1:名前 "川崎篤子" ;
+    ns1:候補名 "川崎あつ子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子篤" ;
+    ns1:名前 "川崎子篤" ;
     ns1:姓 "川崎" ;
     ns1:小選挙区 "茨城5区" ;
     ns1:年齢 "64" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100005961273973" ;
     ns1:twitter "ryujikawabe" ;
     ns1:せい "かわべ" ;
     ns1:めい "りゅうじ" ;
-    ns1:候補名 "川部竜二" ;
+    ns1:候補名 "かわべ竜二" ;
     ns1:前回 "新" ;
     ns1:名 "竜二" ;
     ns1:名前 "川部竜二" ;
@@ -1361,7 +1412,7 @@
     ns1:twitter "" ;
     ns1:せい "たいら" ;
     ns1:めい "よしひこ" ;
-    ns1:候補名 "平善彦" ;
+    ns1:候補名 "たいら善彦" ;
     ns1:前回 "新" ;
     ns1:名 "善彦" ;
     ns1:名前 "平善彦" ;
@@ -1374,7 +1425,7 @@
     ns1:twitter "" ;
     ns1:せい "ひらかわ" ;
     ns1:めい "みちや" ;
-    ns1:候補名 "平川道也" ;
+    ns1:候補名 "平川みちや" ;
     ns1:前回 "新" ;
     ns1:名 "道也" ;
     ns1:名前 "平川道也" ;
@@ -1409,14 +1460,14 @@
     ns1:年齢 "62" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/setuko2525" ;
     ns1:twitter "" ;
     ns1:せい "たなはし" ;
     ns1:めい "せつこ" ;
-    ns1:候補名 "店橋世津子" ;
-    ns1:前回 "新" ;
-    ns1:名 "世津子" ;
-    ns1:名前 "店橋世津子" ;
+    ns1:候補名 "たなはしせつ子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子世津" ;
+    ns1:名前 "店橋子世津" ;
     ns1:姓 "店橋" ;
     ns1:小選挙区 "群馬1区" ;
     ns1:年齢 "56" ;
@@ -1432,7 +1483,7 @@
     ns1:名前 "志位和夫" ;
     ns1:姓 "志位" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "63" ;
+    ns1:年齢 "62" ;
     ns1:比例区 "南関東" .
 
 [] ns1:facebook "" ;
@@ -1448,11 +1499,11 @@
     ns1:年齢 "67" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100009276016244" ;
     ns1:twitter "" ;
     ns1:せい "ところ" ;
     ns1:めい "すすむ" ;
-    ns1:候補名 "所進" ;
+    ns1:候補名 "所すすむ" ;
     ns1:前回 "新" ;
     ns1:名 "進" ;
     ns1:名前 "所進" ;
@@ -1478,13 +1529,13 @@
     ns1:twitter "" ;
     ns1:せい "さいとう" ;
     ns1:めい "さちこ" ;
-    ns1:候補名 "斉藤幸子" ;
+    ns1:候補名 "さいとう幸子" ;
     ns1:前回 "新" ;
     ns1:名 "幸子" ;
     ns1:名前 "斉藤幸子" ;
     ns1:姓 "斉藤" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "60" ;
+    ns1:年齢 "59" ;
     ns1:比例区 "近畿" .
 
 [] ns1:facebook "" ;
@@ -1513,11 +1564,11 @@
     ns1:年齢 "57" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/jcp.munakasu" ;
     ns1:twitter "" ;
     ns1:せい "しんどめ" ;
     ns1:めい "きよたか" ;
-    ns1:候補名 "新留清隆" ;
+    ns1:候補名 "しんどめ清隆" ;
     ns1:前回 "新" ;
     ns1:名 "清隆" ;
     ns1:名前 "新留清隆" ;
@@ -1528,22 +1579,22 @@
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
-    ns1:せい "ほしの" ;
-    ns1:めい "ふみお" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
     ns1:候補名 "星野文雄" ;
     ns1:前回 "新" ;
-    ns1:名 "文雄" ;
+    ns1:名 "" ;
     ns1:名前 "星野文雄" ;
-    ns1:姓 "星野" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "茨城2区" ;
     ns1:年齢 "69" ;
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "jcpniiza" ;
+    ns1:twitter "" ;
     ns1:せい "あさか" ;
     ns1:めい "ひでよし" ;
-    ns1:候補名 "朝賀英義" ;
+    ns1:候補名 "あさか英義" ;
     ns1:前回 "新" ;
     ns1:名 "英義" ;
     ns1:名前 "朝賀英義" ;
@@ -1553,10 +1604,10 @@
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "hondanobuhiro" ;
+    ns1:twitter "" ;
     ns1:せい "ほんだ" ;
     ns1:めい "のぶひろ" ;
-    ns1:候補名 "本多信弘" ;
+    ns1:候補名 "本多のぶひろ" ;
     ns1:前回 "新" ;
     ns1:名 "信弘" ;
     ns1:名前 "本多信弘" ;
@@ -1565,7 +1616,7 @@
     ns1:年齢 "41" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/nobuko.motomura.3" ;
     ns1:twitter "motomura_nobuko" ;
     ns1:せい "もとむら" ;
     ns1:めい "のぶこ" ;
@@ -1591,17 +1642,17 @@
     ns1:年齢 "68" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/honmakatsumi" ;
+[] ns1:facebook "https://www.facebook.com/honmakatsumi/" ;
     ns1:twitter "" ;
     ns1:せい "ほんま" ;
     ns1:めい "かつみ" ;
-    ns1:候補名 "本間勝美" ;
+    ns1:候補名 "本間かつみ" ;
     ns1:前回 "新" ;
     ns1:名 "勝美" ;
     ns1:名前 "本間勝美" ;
     ns1:姓 "本間" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "48" ;
+    ns1:年齢 "49" ;
     ns1:比例区 "北海道" .
 
 [] ns1:facebook "" ;
@@ -1617,20 +1668,20 @@
     ns1:年齢 "68" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "" ;
+[] ns1:facebook "https://www.facebook.com/jcpsuitasettsukuntan/" ;
+    ns1:twitter "hanahanaexpress" ;
     ns1:せい "むらぐち" ;
     ns1:めい "くみこ" ;
-    ns1:候補名 "村口久美子" ;
-    ns1:前回 "新" ;
-    ns1:名 "久美子" ;
-    ns1:名前 "村口久美子" ;
+    ns1:候補名 "村口くみ子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子久美" ;
+    ns1:名前 "村口子久美" ;
     ns1:姓 "村口" ;
     ns1:小選挙区 "大阪7区" ;
     ns1:年齢 "45" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/matsuihideakijimusyo/" ;
     ns1:twitter "" ;
     ns1:せい "まつい" ;
     ns1:めい "ひであき" ;
@@ -1656,14 +1707,14 @@
     ns1:年齢 "55" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/rm11104" ;
     ns1:twitter "matsuo_ritsuko" ;
     ns1:せい "まつお" ;
     ns1:めい "りつこ" ;
-    ns1:候補名 "松尾律子" ;
-    ns1:前回 "新" ;
-    ns1:名 "律子" ;
-    ns1:名前 "松尾律子" ;
+    ns1:候補名 "松尾りつ子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子律" ;
+    ns1:名前 "松尾子律" ;
     ns1:姓 "松尾" ;
     ns1:小選挙区 "福岡2区" ;
     ns1:年齢 "45" ;
@@ -1682,7 +1733,7 @@
     ns1:年齢 "58" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/mtok48" ;
     ns1:twitter "monsmsg2" ;
     ns1:せい "まつおか" ;
     ns1:めい "まさる" ;
@@ -1695,7 +1746,7 @@
     ns1:年齢 "44" ;
     ns1:比例区 "九州" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100013854768106" ;
     ns1:twitter "matsu_en" ;
     ns1:せい "まつもと" ;
     ns1:めい "けんじ" ;
@@ -1708,11 +1759,11 @@
     ns1:年齢 "33" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/ryosuke.matsumura.79" ;
     ns1:twitter "akahigepapa" ;
     ns1:せい "まつむら" ;
     ns1:めい "りょうすけ" ;
-    ns1:候補名 "松村亮佑" ;
+    ns1:候補名 "松村りょうすけ" ;
     ns1:前回 "新" ;
     ns1:名 "亮佑" ;
     ns1:名前 "松村亮佑" ;
@@ -1721,11 +1772,11 @@
     ns1:年齢 "37" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100008887746844" ;
     ns1:twitter "" ;
     ns1:せい "まつはし" ;
     ns1:めい "ちはる" ;
-    ns1:候補名 "松橋千春" ;
+    ns1:候補名 "松橋ちはる" ;
     ns1:前回 "新" ;
     ns1:名 "千春" ;
     ns1:名前 "松橋千春" ;
@@ -1734,7 +1785,7 @@
     ns1:年齢 "35" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/KazushiMatsuda53" ;
     ns1:twitter "" ;
     ns1:せい "まつだ" ;
     ns1:めい "かずし" ;
@@ -1747,9 +1798,8 @@
     ns1:年齢 "60" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "2b28a3370fae4f9",
-        "itakurama" ;
+[] ns1:facebook "https://www.facebook.com/masafumi.itakura.5" ;
+    ns1:twitter "2b28a3370fae4f9" ;
     ns1:せい "いたくら" ;
     ns1:めい "まさふみ" ;
     ns1:候補名 "いたくら正文" ;
@@ -1765,7 +1815,7 @@
     ns1:twitter "" ;
     ns1:せい "はやし" ;
     ns1:めい "みやこ" ;
-    ns1:候補名 "林京" ;
+    ns1:候補名 "林みやこ" ;
     ns1:前回 "新" ;
     ns1:名 "京" ;
     ns1:名前 "林京" ;
@@ -1775,10 +1825,10 @@
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "hayashihide22" ;
+    ns1:twitter "" ;
     ns1:せい "はやし" ;
     ns1:めい "ひでひろ" ;
-    ns1:候補名 "林秀洋" ;
+    ns1:候補名 "林ひでひろ" ;
     ns1:前回 "新" ;
     ns1:名 "秀洋" ;
     ns1:名前 "林秀洋" ;
@@ -1787,11 +1837,11 @@
     ns1:年齢 "43" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/shibaoka.yuma" ;
     ns1:twitter "shibaoka_yuma" ;
     ns1:せい "しばおか" ;
     ns1:めい "ゆうま" ;
-    ns1:候補名 "柴岡祐真" ;
+    ns1:候補名 "柴岡ゆうま" ;
     ns1:前回 "新" ;
     ns1:名 "祐真" ;
     ns1:名前 "柴岡祐真" ;
@@ -1801,24 +1851,23 @@
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "140829haruking" ;
-    ns1:せい "かきぬま" ;
-    ns1:めい "はるき" ;
-    ns1:候補名 "柿沼遥輝" ;
-    ns1:前回 "新" ;
-    ns1:名 "遥輝" ;
-    ns1:名前 "柿沼遥輝" ;
-    ns1:姓 "柿沼" ;
+    ns1:twitter "" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
+    ns1:候補名 "柿沼はるき" ;
+    ns1:前回 "新" ;
+    ns1:名 "" ;
+    ns1:名前 "柿沼はるき" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "埼玉10区" ;
     ns1:年齢 "34" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/umemurasaeko" ;
     ns1:twitter "umemura_saeko" ;
     ns1:せい "うめむら" ;
     ns1:めい "さえこ" ;
-    ns1:候補名 "梅村さえこ",
-        "関口早江子" ;
+    ns1:候補名 "梅村さえこ" ;
     ns1:前回 "前" ;
     ns1:名 "早江子" ;
     ns1:名前 "梅村早江子" ;
@@ -1840,7 +1889,7 @@
     ns1:年齢 "60" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/fumiaki.shiina" ;
     ns1:twitter "" ;
     ns1:せい "しいな" ;
     ns1:めい "ふみあき" ;
@@ -1853,11 +1902,11 @@
     ns1:年齢 "60" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100012701765862" ;
     ns1:twitter "" ;
     ns1:せい "くすもと" ;
     ns1:めい "ふみろう" ;
-    ns1:候補名 "楠本文郎" ;
+    ns1:候補名 "くすもと文郎" ;
     ns1:前回 "新" ;
     ns1:名 "文郎" ;
     ns1:名前 "楠本文郎" ;
@@ -1870,7 +1919,7 @@
     ns1:twitter "" ;
     ns1:せい "まき" ;
     ns1:めい "しょうぞう" ;
-    ns1:候補名 "槙昌三" ;
+    ns1:候補名 "まき昌三" ;
     ns1:前回 "新" ;
     ns1:名 "昌三" ;
     ns1:名前 "槙昌三" ;
@@ -1879,11 +1928,11 @@
     ns1:年齢 "74" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/seigo.yokoyama.1/" ;
     ns1:twitter "" ;
     ns1:せい "よこやま" ;
     ns1:めい "せいご" ;
-    ns1:候補名 "横山征吾" ;
+    ns1:候補名 "横山せいご" ;
     ns1:前回 "新" ;
     ns1:名 "征吾" ;
     ns1:名前 "横山征吾" ;
@@ -1905,7 +1954,7 @@
     ns1:年齢 "73" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100004574320067" ;
     ns1:twitter "" ;
     ns1:せい "よこた" ;
     ns1:めい "えいじ" ;
@@ -1922,7 +1971,7 @@
     ns1:twitter "" ;
     ns1:せい "はしもと" ;
     ns1:めい "けんじ" ;
-    ns1:候補名 "橋本健二" ;
+    ns1:候補名 "橋本けんじ" ;
     ns1:前回 "新" ;
     ns1:名 "健二" ;
     ns1:名前 "橋本健二" ;
@@ -1931,7 +1980,7 @@
     ns1:年齢 "69" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/eimouri" ;
     ns1:twitter "eimouri" ;
     ns1:せい "もうり" ;
     ns1:めい "えいこ" ;
@@ -1944,11 +1993,11 @@
     ns1:年齢 "66" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/100008827811511" ;
     ns1:twitter "omuta_eguchi" ;
     ns1:せい "えぐち" ;
     ns1:めい "まなぶ" ;
-    ns1:候補名 "江口学" ;
+    ns1:候補名 "江口まなぶ" ;
     ns1:前回 "新" ;
     ns1:名 "学" ;
     ns1:名前 "江口学" ;
@@ -1957,9 +2006,8 @@
     ns1:年齢 "43" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/saorioshitai" ;
-    ns1:twitter "ikeuchi_saori",
-        "saorijimusyo" ;
+[] ns1:facebook "https://www.facebook.com/saolyn.ikeuchi" ;
+    ns1:twitter "ikeuchi_saori" ;
     ns1:せい "いけうち" ;
     ns1:めい "さおり" ;
     ns1:候補名 "池内さおり" ;
@@ -1981,14 +2029,14 @@
     ns1:名前 "池田博英" ;
     ns1:姓 "池田" ;
     ns1:小選挙区 "神奈川16区" ;
-    ns1:年齢 "50" ;
-    ns1:比例区 "" .
-
-[] ns1:facebook "" ;
+    ns1:年齢 "54" ;
+    ns1:比例区 "" .
+
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100008264835156" ;
     ns1:twitter "" ;
     ns1:せい "かわむら" ;
     ns1:めい "ただし" ;
-    ns1:候補名 "河村整" ;
+    ns1:候補名 "河村ただし" ;
     ns1:前回 "新" ;
     ns1:名 "整" ;
     ns1:名前 "河村整" ;
@@ -2001,7 +2049,7 @@
     ns1:twitter "" ;
     ns1:せい "ふかつ" ;
     ns1:めい "としろう" ;
-    ns1:候補名 "深津俊郎" ;
+    ns1:候補名 "深津としろう" ;
     ns1:前回 "新" ;
     ns1:名 "俊郎" ;
     ns1:名前 "深津俊郎" ;
@@ -2010,7 +2058,7 @@
     ns1:年齢 "69" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/shimizu.tadashi.jcp" ;
+[] ns1:facebook "https://www.facebook.com/tadashi.shimizu.90" ;
     ns1:twitter "tadashishimizu" ;
     ns1:せい "しみず" ;
     ns1:めい "ただし" ;
@@ -2027,7 +2075,7 @@
     ns1:twitter "" ;
     ns1:せい "わたなべ" ;
     ns1:めい "ひろし" ;
-    ns1:候補名 "渡辺裕" ;
+    ns1:候補名 "渡辺ひろし" ;
     ns1:前回 "新" ;
     ns1:名 "裕" ;
     ns1:名前 "渡辺裕" ;
@@ -2036,7 +2084,7 @@
     ns1:年齢 "37" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/yui.watanabe.jcp.osk/" ;
     ns1:twitter "yuiwatanabe_osk" ;
     ns1:せい "わたなべ" ;
     ns1:めい "ゆい" ;
@@ -2053,7 +2101,7 @@
     ns1:twitter "" ;
     ns1:せい "わたなべ" ;
     ns1:めい "たかお" ;
-    ns1:候補名 "渡部隆夫" ;
+    ns1:候補名 "わたなべ隆夫" ;
     ns1:前回 "新" ;
     ns1:名 "隆夫" ;
     ns1:名前 "渡部隆夫" ;
@@ -2066,7 +2114,7 @@
     ns1:twitter "" ;
     ns1:せい "せと" ;
     ns1:めい "かずひろ" ;
-    ns1:候補名 "瀬戸和弘" ;
+    ns1:候補名 "せと和弘" ;
     ns1:前回 "新" ;
     ns1:名 "和弘" ;
     ns1:名前 "瀬戸和弘" ;
@@ -2075,11 +2123,11 @@
     ns1:年齢 "65" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/kumagaitomo3/" ;
     ns1:twitter "h0use3" ;
     ns1:せい "くまがい" ;
     ns1:めい "とも" ;
-    ns1:候補名 "熊谷智" ;
+    ns1:候補名 "くまがい智" ;
     ns1:前回 "新" ;
     ns1:名 "智" ;
     ns1:名前 "熊谷智" ;
@@ -2088,7 +2136,7 @@
     ns1:年齢 "37" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100009414735635" ;
     ns1:twitter "" ;
     ns1:せい "まきやま" ;
     ns1:めい "たかし" ;
@@ -2105,7 +2153,7 @@
     ns1:twitter "" ;
     ns1:せい "さるはし" ;
     ns1:めい "たくみ" ;
-    ns1:候補名 "猿橋たくみ" ;
+    ns1:候補名 "さるはし巧" ;
     ns1:前回 "新" ;
     ns1:名 "巧" ;
     ns1:名前 "猿橋巧" ;
@@ -2116,22 +2164,35 @@
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
+    ns1:せい "たまき" ;
+    ns1:めい "デニー" ;
+    ns1:候補名 "玉城デニー" ;
+    ns1:前回 "前" ;
+    ns1:名 "デニー" ;
+    ns1:名前 "玉城デニー" ;
+    ns1:姓 "玉城" ;
+    ns1:小選挙区 "沖縄3区(推薦、オール沖縄)" ;
+    ns1:年齢 "58" ;
+    ns1:比例区 "" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:せい "たがみ" ;
     ns1:めい "みつのり" ;
-    ns1:候補名 "田上光徳" ;
+    ns1:候補名 "田上みつのり" ;
     ns1:前回 "新" ;
     ns1:名 "光徳" ;
     ns1:名前 "田上光徳" ;
     ns1:姓 "田上" ;
     ns1:小選挙区 "愛知6区" ;
-    ns1:年齢 "65" ;
-    ns1:比例区 "" .
-
-[] ns1:facebook "" ;
+    ns1:年齢 "66" ;
+    ns1:比例区 "" .
+
+[] ns1:facebook "https://www.facebook.com/people/Yoji-Tanaka/100007628881907" ;
     ns1:twitter "tanakayoji" ;
     ns1:せい "たなか" ;
     ns1:めい "ようじ" ;
-    ns1:候補名 "田中陽二" ;
+    ns1:候補名 "田中ようじ" ;
     ns1:前回 "新" ;
     ns1:名 "陽二" ;
     ns1:名前 "田中陽二" ;
@@ -2144,7 +2205,7 @@
     ns1:twitter "tamuratakaaki" ;
     ns1:せい "たむら" ;
     ns1:めい "たかあき" ;
-    ns1:候補名 "田村貴昭" ;
+    ns1:候補名 "田村たかあき" ;
     ns1:前回 "前" ;
     ns1:名 "貴昭" ;
     ns1:名前 "田村貴昭" ;
@@ -2153,7 +2214,7 @@
     ns1:年齢 "56" ;
     ns1:比例区 "九州" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/kimie.hatano.9" ;
     ns1:twitter "hatanokimie" ;
     ns1:せい "はたの" ;
     ns1:めい "きみえ" ;
@@ -2166,7 +2227,7 @@
     ns1:年齢 "60" ;
     ns1:比例区 "南関東" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/kazuya.hatayama.1" ;
     ns1:twitter "hatayama_kazuya" ;
     ns1:せい "はたやま" ;
     ns1:めい "かずや" ;
@@ -2176,33 +2237,33 @@
     ns1:名前 "畠山和也" ;
     ns1:姓 "畠山" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "46" ;
+    ns1:年齢 "45" ;
     ns1:比例区 "北海道" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/SirakawaYouko" ;
     ns1:twitter "" ;
     ns1:せい "しらかわ" ;
     ns1:めい "ようこ" ;
-    ns1:候補名 "白川容子" ;
-    ns1:前回 "新" ;
-    ns1:名 "容子" ;
-    ns1:名前 "白川容子" ;
+    ns1:候補名 "白川よう子" ;
+    ns1:前回 "新" ;
+    ns1:名 "子容" ;
+    ns1:名前 "白川子容" ;
     ns1:姓 "白川" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "51" ;
+    ns1:年齢 "50" ;
     ns1:比例区 "四国" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/masuda.jcp" ;
     ns1:twitter "" ;
     ns1:せい "ますだ" ;
     ns1:めい "まきこ" ;
-    ns1:候補名 "益田牧子" ;
+    ns1:候補名 "ますだ牧子" ;
     ns1:前回 "新" ;
     ns1:名 "牧子" ;
     ns1:名前 "益田牧子" ;
     ns1:姓 "益田" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "67" ;
+    ns1:年齢 "66" ;
     ns1:比例区 "九州" .
 
 [] ns1:facebook "https://www.facebook.com/majimasyouzou.jcp" ;
@@ -2218,7 +2279,7 @@
     ns1:年齢 "54" ;
     ns1:比例区 "九州" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/tadasige.yano.7" ;
     ns1:twitter "yanotadasige" ;
     ns1:せい "やの" ;
     ns1:めい "ただしげ" ;
@@ -2244,11 +2305,11 @@
     ns1:年齢 "68" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "jcp_ishidoh" ;
+[] ns1:facebook "https://www.facebook.com/atsushi.ishidoh" ;
+    ns1:twitter "" ;
     ns1:せい "いしどう" ;
     ns1:めい "あつし" ;
-    ns1:候補名 "石堂淳士" ;
+    ns1:候補名 "石堂あつし" ;
     ns1:前回 "新" ;
     ns1:名 "淳士" ;
     ns1:名前 "石堂淳士" ;
@@ -2261,16 +2322,16 @@
     ns1:twitter "" ;
     ns1:せい "いしじま" ;
     ns1:めい "いわお" ;
-    ns1:候補名 "石嶋巌" ;
+    ns1:候補名 "石嶋いわお" ;
     ns1:前回 "新" ;
     ns1:名 "巌" ;
     ns1:名前 "石嶋巌" ;
     ns1:姓 "石嶋" ;
     ns1:小選挙区 "茨城7区" ;
-    ns1:年齢 "57" ;
-    ns1:比例区 "" .
-
-[] ns1:facebook "" ;
+    ns1:年齢 "63" ;
+    ns1:比例区 "" .
+
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100008387827735" ;
     ns1:twitter "tukinokawasator" ;
     ns1:せい "いしかわ" ;
     ns1:めい "さとる" ;
@@ -2280,7 +2341,7 @@
     ns1:名前 "石川悟" ;
     ns1:姓 "石川" ;
     ns1:小選挙区 "長崎4区" ;
-    ns1:年齢 "64" ;
+    ns1:年齢 "65" ;
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
@@ -2296,8 +2357,8 @@
     ns1:年齢 "66" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "sho_ishikawa38" ;
+[] ns1:facebook "https://www.facebook.com/shoishikawa1973/" ;
+    ns1:twitter "" ;
     ns1:せい "いしかわ" ;
     ns1:めい "しょう" ;
     ns1:候補名 "石川渉" ;
@@ -2323,10 +2384,10 @@
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "noriyuki_ishi" ;
+    ns1:twitter "" ;
     ns1:せい "いしざわ" ;
     ns1:めい "のりゆき" ;
-    ns1:候補名 "石沢憲之" ;
+    ns1:候補名 "石沢のりゆき" ;
     ns1:前回 "新" ;
     ns1:名 "憲之" ;
     ns1:名前 "石沢憲之" ;
@@ -2336,7 +2397,7 @@
     ns1:比例区 "東京" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "sobuemotoki" ;
+    ns1:twitter "sobuemotoki/media" ;
     ns1:せい "そぶえ" ;
     ns1:めい "もとき" ;
     ns1:候補名 "そぶえ元希" ;
@@ -2374,7 +2435,7 @@
     ns1:年齢 "63" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/fukuzumih" ;
     ns1:twitter "fukuzumih" ;
     ns1:せい "ふくずみ" ;
     ns1:めい "ひでゆき" ;
@@ -2391,7 +2452,7 @@
     ns1:twitter "" ;
     ns1:せい "いなむら" ;
     ns1:めい "さとる" ;
-    ns1:候補名 "稲村知" ;
+    ns1:候補名 "稲村さとる" ;
     ns1:前回 "新" ;
     ns1:名 "知" ;
     ns1:名前 "稲村知" ;
@@ -2400,11 +2461,11 @@
     ns1:年齢 "57" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/kokutakeiji" ;
     ns1:twitter "kokutakeiji" ;
     ns1:せい "こくた" ;
     ns1:めい "けいじ" ;
-    ns1:候補名 "穀田恵二" ;
+    ns1:候補名 "こくた恵二" ;
     ns1:前回 "前" ;
     ns1:名 "恵二" ;
     ns1:名前 "穀田恵二" ;
@@ -2413,7 +2474,7 @@
     ns1:年齢 "70" ;
     ns1:比例区 "近畿" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/takahiko.tachikawa" ;
     ns1:twitter "" ;
     ns1:せい "たちかわ" ;
     ns1:めい "たかひこ" ;
@@ -2426,7 +2487,7 @@
     ns1:年齢 "66" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/akira.kasai1" ;
     ns1:twitter "akibacsi" ;
     ns1:せい "かさい" ;
     ns1:めい "あきら" ;
@@ -2454,22 +2515,9 @@
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
-    ns1:せい "こみやま" ;
-    ns1:めい "えみこ" ;
-    ns1:候補名 "籠山恵美子" ;
-    ns1:前回 "新" ;
-    ns1:名 "恵美子" ;
-    ns1:名前 "籠山恵美子" ;
-    ns1:姓 "籠山" ;
-    ns1:小選挙区 "岐阜4区" ;
-    ns1:年齢 "62" ;
-    ns1:比例区 "" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "" ;
     ns1:せい "みいみ" ;
     ns1:めい "よしあき" ;
-    ns1:候補名 "美見芳明" ;
+    ns1:候補名 "みいみ芳明" ;
     ns1:前回 "新" ;
     ns1:名 "芳明" ;
     ns1:名前 "美見芳明" ;
@@ -2478,21 +2526,21 @@
     ns1:年齢 "60" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/funayama.yumi.haiji" ;
     ns1:twitter "" ;
     ns1:せい "ふなやま" ;
     ns1:めい "ゆみ" ;
-    ns1:候補名 "舩山由美" ;
+    ns1:候補名 "ふなやま由美" ;
     ns1:前回 "新" ;
     ns1:名 "由美" ;
     ns1:名前 "舩山由美" ;
     ns1:姓 "舩山" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "49" ;
+    ns1:年齢 "48" ;
     ns1:比例区 "東北" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "kyokonae" ;
+[] ns1:facebook "https://www.facebook.com/kyoko.nae" ;
+    ns1:twitter "" ;
     ns1:せい "なえむら" ;
     ns1:めい "きょうこ" ;
     ns1:候補名 "苗村京子" ;
@@ -2504,7 +2552,7 @@
     ns1:年齢 "58" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/wakabayashi8" ;
+[] ns1:facebook "https://www.facebook.com/wakabayashi8/" ;
     ns1:twitter "wakayasu1985" ;
     ns1:せい "わかばやし" ;
     ns1:めい "やすひさ" ;
@@ -2517,11 +2565,11 @@
     ns1:年齢 "32" ;
     ns1:比例区 "南関東" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "http://www.facebook.com/makoto.sugawara.33" ;
     ns1:twitter "sugawara_makoto" ;
     ns1:せい "すがわら" ;
     ns1:めい "まこと" ;
-    ns1:候補名 "菅原誠" ;
+    ns1:候補名 "菅原まこと" ;
     ns1:前回 "新" ;
     ns1:名 "誠" ;
     ns1:名前 "菅原誠" ;
@@ -2534,7 +2582,7 @@
     ns1:twitter "" ;
     ns1:せい "はぎわら" ;
     ns1:めい "さだお" ;
-    ns1:候補名 "萩原貞夫" ;
+    ns1:候補名 "はぎわら貞夫" ;
     ns1:前回 "新" ;
     ns1:名 "貞夫" ;
     ns1:名前 "萩原貞夫" ;
@@ -2543,11 +2591,11 @@
     ns1:年齢 "68" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/kazunori.fujimoto.587" ;
     ns1:twitter "1218fuji" ;
     ns1:せい "ふじもと" ;
     ns1:めい "かずのり" ;
-    ns1:候補名 "藤本一規" ;
+    ns1:候補名 "藤本かずのり" ;
     ns1:前回 "新" ;
     ns1:名 "一規" ;
     ns1:名前 "藤本一規" ;
@@ -2556,7 +2604,7 @@
     ns1:年齢 "52" ;
     ns1:比例区 "中国" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/yuri.fupo" ;
     ns1:twitter "fupo_cgsm" ;
     ns1:せい "ふじもと" ;
     ns1:めい "ゆり" ;
@@ -2582,7 +2630,7 @@
     ns1:年齢 "42" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100006891949363" ;
     ns1:twitter "" ;
     ns1:せい "ふじもと" ;
     ns1:めい "さとし" ;
@@ -2592,27 +2640,27 @@
     ns1:名前 "藤本聡志" ;
     ns1:姓 "藤本" ;
     ns1:小選挙区 "広島2区" ;
-    ns1:年齢 "62" ;
-    ns1:比例区 "" .
-
-[] ns1:facebook "" ;
+    ns1:年齢 "63" ;
+    ns1:比例区 "" .
+
+[] ns1:facebook "https://www.facebook.com/fujino.yasufumi" ;
     ns1:twitter "fujinofujinooo" ;
     ns1:せい "ふじの" ;
     ns1:めい "やすふみ" ;
-    ns1:候補名 "藤野保史" ;
+    ns1:候補名 "藤野やすふみ" ;
     ns1:前回 "前" ;
     ns1:名 "保史" ;
     ns1:名前 "藤野保史" ;
     ns1:姓 "藤野" ;
     ns1:小選挙区 "" ;
-    ns1:年齢 "47" ;
+    ns1:年齢 "46" ;
     ns1:比例区 "北陸信越" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:せい "にしなか" ;
     ns1:めい "たかお" ;
-    ns1:候補名 "西中孝男" ;
+    ns1:候補名 "西中たかお" ;
     ns1:前回 "新" ;
     ns1:名 "孝男" ;
     ns1:名前 "西中孝男" ;
@@ -2634,7 +2682,7 @@
     ns1:年齢 "60" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/jcp.yamaguti.seibu" ;
     ns1:twitter "" ;
     ns1:せい "にしおか" ;
     ns1:めい "ひろのぶ" ;
@@ -2647,11 +2695,11 @@
     ns1:年齢 "54" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "nissyzaaaawa" ;
+[] ns1:facebook "https://www.facebook.com/koichi.nishizawa.758" ;
+    ns1:twitter "" ;
     ns1:せい "にしざわ" ;
     ns1:めい "こういち" ;
-    ns1:候補名 "西沢耕一" ;
+    ns1:候補名 "西沢こういち" ;
     ns1:前回 "新" ;
     ns1:名 "耕一" ;
     ns1:名前 "西沢耕一" ;
@@ -2662,31 +2710,31 @@
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
-    ns1:せい "にしだ" ;
-    ns1:めい "としこ" ;
-    ns1:候補名 "西田敏子" ;
-    ns1:前回 "新" ;
-    ns1:名 "敏子" ;
-    ns1:名前 "西田敏子" ;
-    ns1:姓 "西田" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
+    ns1:候補名 "西田とし子" ;
+    ns1:前回 "新" ;
+    ns1:名 "" ;
+    ns1:名前 "西田とし子" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "愛知4区" ;
     ns1:年齢 "63" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100005911840765" ;
     ns1:twitter "" ;
     ns1:せい "たになか" ;
     ns1:めい "みよし" ;
-    ns1:候補名 "谷中三好" ;
-    ns1:前回 "新" ;
-    ns1:名 "三好" ;
-    ns1:名前 "谷中三好" ;
+    ns1:候補名 "谷中みよし" ;
+    ns1:前回 "新" ;
+    ns1:名 "みよし" ;
+    ns1:名前 "谷中みよし" ;
     ns1:姓 "谷中" ;
     ns1:小選挙区 "三重4区" ;
     ns1:年齢 "57" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/JCP.TANet" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100001997661120" ;
     ns1:twitter "ttanigawa3" ;
     ns1:せい "たにがわ" ;
     ns1:めい "ともゆき" ;
@@ -2699,7 +2747,7 @@
     ns1:年齢 "46" ;
     ns1:比例区 "東京" .
 
-[] ns1:facebook "https://www.facebook.com/akamineseiken/" ;
+[] ns1:facebook "https://www.facebook.com/akamineseiken" ;
     ns1:twitter "akamineseiken" ;
     ns1:せい "あかみね" ;
     ns1:めい "せいけん" ;
@@ -2708,15 +2756,15 @@
     ns1:名 "政賢" ;
     ns1:名前 "赤嶺政賢" ;
     ns1:姓 "赤嶺" ;
-    ns1:小選挙区 "沖縄1区" ;
+    ns1:小選挙区 "沖縄1区(党公認、オール沖縄)" ;
     ns1:年齢 "69" ;
     ns1:比例区 "九州" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "akahira1990" ;
+    ns1:twitter "" ;
     ns1:せい "あかひら" ;
     ns1:めい "ゆうと" ;
-    ns1:候補名 "赤平勇人" ;
+    ns1:候補名 "赤平ゆうと" ;
     ns1:前回 "新" ;
     ns1:名 "勇人" ;
     ns1:名前 "赤平勇人" ;
@@ -2726,10 +2774,10 @@
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "tsujijcp" ;
+    ns1:twitter "" ;
     ns1:せい "つじ" ;
-    ns1:めい "もとみ" ;
-    ns1:候補名 "辻源巳" ;
+    ns1:めい "モトミ" ;
+    ns1:候補名 "つじモトミ" ;
     ns1:前回 "新" ;
     ns1:名 "源巳" ;
     ns1:名前 "辻源巳" ;
@@ -2751,11 +2799,11 @@
     ns1:年齢 "68" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "iakas_oratnek" ;
+[] ns1:facebook "https://www.facebook.com/kentaro.sakai.12" ;
+    ns1:twitter "" ;
     ns1:せい "さかい" ;
-    ns1:めい "けんたろう" ;
-    ns1:候補名 "酒井健太朗" ;
+    ns1:めい "ケンタロー" ;
+    ns1:候補名 "酒井ケンタロー" ;
     ns1:前回 "新" ;
     ns1:名 "健太朗" ;
     ns1:名前 "酒井健太朗" ;
@@ -2768,7 +2816,7 @@
     ns1:twitter "koji_1951" ;
     ns1:せい "しげむら" ;
     ns1:めい "こうじ" ;
-    ns1:候補名 "重村幸司" ;
+    ns1:候補名 "しげむら幸司" ;
     ns1:前回 "新" ;
     ns1:名 "幸司" ;
     ns1:名前 "重村幸司" ;
@@ -2781,7 +2829,7 @@
     ns1:twitter "" ;
     ns1:せい "のむら" ;
     ns1:めい "まさひろ" ;
-    ns1:候補名 "野村真弘" ;
+    ns1:候補名 "野村まさひろ" ;
     ns1:前回 "新" ;
     ns1:名 "真弘" ;
     ns1:名前 "野村真弘" ;
@@ -2790,20 +2838,7 @@
     ns1:年齢 "61" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
-    ns1:twitter "jkabucp" ;
-    ns1:せい "のざわ" ;
-    ns1:めい "やすゆき" ;
-    ns1:候補名 "野沢康幸" ;
-    ns1:前回 "新" ;
-    ns1:名 "康幸" ;
-    ns1:名前 "野沢康幸" ;
-    ns1:姓 "野沢" ;
-    ns1:小選挙区 "愛知15区" ;
-    ns1:年齢 "29" ;
-    ns1:比例区 "" .
-
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/kanakuramasatoshi" ;
     ns1:twitter "kanakuramasato" ;
     ns1:せい "かなくら" ;
     ns1:めい "まさとし" ;
@@ -2816,11 +2851,11 @@
     ns1:年齢 "43" ;
     ns1:比例区 "北海道" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100005810323550" ;
     ns1:twitter "jcp_555" ;
     ns1:せい "かねもと" ;
     ns1:めい "ゆきえ" ;
-    ns1:候補名 "金元幸枝" ;
+    ns1:候補名 "かねもと幸枝" ;
     ns1:前回 "新" ;
     ns1:名 "幸枝" ;
     ns1:名前 "金元幸枝" ;
@@ -2833,7 +2868,7 @@
     ns1:twitter "" ;
     ns1:せい "きんぱら" ;
     ns1:めい "のぶゆき" ;
-    ns1:候補名 "金原信之" ;
+    ns1:候補名 "金原のぶゆき" ;
     ns1:前回 "新" ;
     ns1:名 "信之" ;
     ns1:名前 "金原信之" ;
@@ -2842,7 +2877,7 @@
     ns1:年齢 "57" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/tooru.kanamori" ;
     ns1:twitter "mimu_kanamori" ;
     ns1:せい "かなもり" ;
     ns1:めい "とおる" ;
@@ -2855,7 +2890,7 @@
     ns1:年齢 "61" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/kanedamineo" ;
     ns1:twitter "mineo_kaneda" ;
     ns1:せい "かねだ" ;
     ns1:めい "みねお" ;
@@ -2868,7 +2903,7 @@
     ns1:年齢 "52" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/susumu.kugimaru" ;
     ns1:twitter "kugimarususumu" ;
     ns1:せい "くぎまる" ;
     ns1:めい "すすむ" ;
@@ -2881,7 +2916,7 @@
     ns1:年齢 "44" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/chikachikagogo/" ;
     ns1:twitter "chikachika_gogo" ;
     ns1:せい "すずき" ;
     ns1:めい "ちか" ;
@@ -2920,11 +2955,11 @@
     ns1:年齢 "29" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/tadahiro.nagatomo" ;
     ns1:twitter "clfcde1eljomceq" ;
     ns1:せい "ながとも" ;
     ns1:めい "ただひろ" ;
-    ns1:候補名 "長友忠弘" ;
+    ns1:候補名 "長友ただひろ" ;
     ns1:前回 "新" ;
     ns1:名 "忠弘" ;
     ns1:名前 "長友忠弘" ;
@@ -2947,13 +2982,13 @@
     ns1:比例区 "" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "tinenaganuma" ;
+    ns1:twitter "" ;
     ns1:せい "ながぬま" ;
     ns1:めい "ちね" ;
-    ns1:候補名 "長沼チネ" ;
-    ns1:前回 "新" ;
-    ns1:名 "チネ" ;
-    ns1:名前 "長沼チネ" ;
+    ns1:候補名 "長沼ちね" ;
+    ns1:前回 "新" ;
+    ns1:名 "ちね" ;
+    ns1:名前 "長沼ちね" ;
     ns1:姓 "長沼" ;
     ns1:小選挙区 "埼玉7区" ;
     ns1:年齢 "67" ;
@@ -2972,7 +3007,7 @@
     ns1:年齢 "49" ;
     ns1:比例区 "北陸信越" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/naoyuki.haseda" ;
     ns1:twitter "takijidakkan" ;
     ns1:せい "はせだ" ;
     ns1:めい "なおゆき" ;
@@ -3011,7 +3046,7 @@
     ns1:年齢 "46" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/Tokyo22abemakoto" ;
+[] ns1:facebook "https://www.facebook.com/makoto.abe.22" ;
     ns1:twitter "rhein2011" ;
     ns1:せい "あべ" ;
     ns1:めい "まこと" ;
@@ -3024,11 +3059,11 @@
     ns1:年齢 "43" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/junichi.shimotori.1" ;
     ns1:twitter "" ;
     ns1:せい "しもとり" ;
     ns1:めい "じゅんいち" ;
-    ns1:候補名 "霜鳥純一" ;
+    ns1:候補名 "しもとり純一" ;
     ns1:前回 "新" ;
     ns1:名 "純一" ;
     ns1:名前 "霜鳥純一" ;
@@ -3039,22 +3074,22 @@
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
-    ns1:せい "あおやま" ;
-    ns1:めい "りょうすけ" ;
+    ns1:せい "" ;
+    ns1:めい "" ;
     ns1:候補名 "青山了介" ;
     ns1:前回 "新" ;
-    ns1:名 "了介" ;
+    ns1:名 "" ;
     ns1:名前 "青山了介" ;
-    ns1:姓 "青山" ;
+    ns1:姓 "" ;
     ns1:小選挙区 "富山1区" ;
     ns1:年齢 "42" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/kohei.aoyama.90" ;
     ns1:twitter "iyaiya_55" ;
     ns1:せい "あおやま" ;
-    ns1:めい "こうへい" ;
-    ns1:候補名 "青山昂平" ;
+    ns1:めい "コウヘイ" ;
+    ns1:候補名 "青山コウヘイ" ;
     ns1:前回 "新" ;
     ns1:名 "昂平" ;
     ns1:名前 "青山昂平" ;
@@ -3067,7 +3102,7 @@
     ns1:twitter "" ;
     ns1:せい "あおき" ;
     ns1:めい "ひろし" ;
-    ns1:候補名 "青木弘" ;
+    ns1:候補名 "青木ひろし" ;
     ns1:前回 "新" ;
     ns1:名 "弘" ;
     ns1:名前 "青木弘" ;
@@ -3076,11 +3111,11 @@
     ns1:年齢 "56" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/八法亭-みややっこ-729898467032255" ;
-    ns1:twitter "" ;
+[] ns1:facebook "https://www.facebook.com/729898467032255/" ;
+    ns1:twitter "iidamiyako1" ;
     ns1:せい "いいだ" ;
     ns1:めい "みやこ" ;
-    ns1:候補名 "飯田美弥子" ;
+    ns1:候補名 "飯田みやこ" ;
     ns1:前回 "新" ;
     ns1:名 "美弥子" ;
     ns1:名前 "飯田美弥子" ;
@@ -3089,14 +3124,14 @@
     ns1:年齢 "57" ;
     ns1:比例区 "東京" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/katusuke.kouzai" ;
     ns1:twitter "kouzai2007" ;
     ns1:せい "こうざい" ;
     ns1:めい "かつすけ" ;
-    ns1:候補名 "香西克介" ;
-    ns1:前回 "新" ;
-    ns1:名 "克介" ;
-    ns1:名前 "香西克介" ;
+    ns1:候補名 "香西かつ介" ;
+    ns1:前回 "新" ;
+    ns1:名 "介克" ;
+    ns1:名前 "香西介克" ;
     ns1:姓 "香西" ;
     ns1:小選挙区 "東京3区" ;
     ns1:年齢 "41" ;
@@ -3106,7 +3141,7 @@
     ns1:twitter "" ;
     ns1:せい "たかやなぎ" ;
     ns1:めい "ひろあき" ;
-    ns1:候補名 "高柳博明" ;
+    ns1:候補名 "高柳ひろあき" ;
     ns1:前回 "新" ;
     ns1:名 "博明" ;
     ns1:名前 "高柳博明" ;
@@ -3115,7 +3150,7 @@
     ns1:年齢 "47" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/megumi.toba.5" ;
     ns1:twitter "toba_megumi" ;
     ns1:せい "とば" ;
     ns1:めい "めぐみ" ;
@@ -3128,7 +3163,7 @@
     ns1:年齢 "58" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/yasusiro.kamosida" ;
     ns1:twitter "ky60954585" ;
     ns1:せい "かもしだ" ;
     ns1:めい "やすしろ" ;
@@ -3154,7 +3189,7 @@
     ns1:年齢 "69" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100004194765892" ;
     ns1:twitter "" ;
     ns1:せい "くろき" ;
     ns1:めい "まんじ" ;
