--- GrayDB
+++ 政治ナビ
@@ -7,61 +7,57 @@
 [] ns1:twitter "abetoshiko" ;
     ns1:wikidata "Q7827692" ;
     ns1:名 "俊子" ;
-    ns1:名前 "阿部俊子" ;
-    ns1:姓 "阿部" ;
+    ns1:名前 "あべ俊子" ;
+    ns1:姓 "あべ" ;
     ns1:性別 "女性" ;
     ns1:生年月日 "1959-05-19" .
 
 [] ns1:twitter "saitou_ken" ;
     ns1:wikidata "Q11679075" ;
     ns1:名 "健" ;
-    ns1:名前 "齋藤健" ;
-    ns1:姓 "齋藤" ;
+    ns1:名前 "斎藤健" ;
+    ns1:姓 "斎藤" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1959-06-14" .
 
 [] ns1:twitter "naomi_tokashiki" ;
     ns1:wikidata "Q11561908" ;
-    ns1:名 "奈緒美" ;
-    ns1:名前 "渡嘉敷奈緒美" ;
-    ns1:姓 "渡嘉敷" ;
+    ns1:名 "なおみ" ;
+    ns1:名前 "とかしきなおみ" ;
+    ns1:姓 "とかしき" ;
     ns1:性別 "女性" ;
     ns1:生年月日 "1962-07-16" .
 
 [] ns1:twitter "hiromitsubayasi" ;
     ns1:wikidata "Q11353993" ;
     ns1:名 "裕巳" ;
-    ns1:名前 "三ッ林裕巳" ;
-    ns1:姓 "三ッ林" ;
-    ns1:性別 "女性" ;
+    ns1:名前 "三ツ林裕巳" ;
+    ns1:姓 "三ツ林" ;
+    ns1:性別 "男性" ;
     ns1:生年月日 "1955-09-07" .
 
 [] ns1:wikidata "Q11455005" ;
     ns1:名 "博行" ;
-    ns1:名前 "宮澤博行" ;
-    ns1:姓 "宮澤" ;
+    ns1:名前 "宮沢博行" ;
+    ns1:姓 "宮沢" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1975-01-10" .
 
-[] ns1:twitter "mitsuya_office" ;
-    ns1:wikidata "Q7051315" ;
+[] ns1:wikidata "Q7051315" ;
     ns1:名 "憲生" ;
     ns1:名前 "三ツ矢憲生" ;
     ns1:姓 "三ツ矢" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1950-12-13" .
 
-[] ns1:twitter "ezzdxpqwoyfkdvh",
-        "miharaasahiko" ;
-    ns1:wikidata "Q4803390" ;
+[] ns1:wikidata "Q4803390" ;
     ns1:名 "朝彦" ;
     ns1:名前 "三原朝彦" ;
     ns1:姓 "三原" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1947-05-23" .
 
-[] ns1:twitter "kamikawa_yoko" ;
-    ns1:wikidata "Q276275" ;
+[] ns1:wikidata "Q276275" ;
     ns1:名 "陽子" ;
     ns1:名前 "上川陽子" ;
     ns1:姓 "上川" ;
@@ -91,24 +87,21 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1965-08-03" .
 
-[] ns1:twitter "mikioshimoji" ;
-    ns1:wikidata "Q1761759" ;
+[] ns1:wikidata "Q1761759" ;
     ns1:名 "幹郎" ;
     ns1:名前 "下地幹郎" ;
     ns1:姓 "下地" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1961-08-14" .
 
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q1655220" ;
+[] ns1:wikidata "Q1655220" ;
     ns1:名 "博文" ;
     ns1:名前 "下村博文" ;
     ns1:姓 "下村" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1954-05-23" .
 
-[] ns1:twitter "norihiron" ;
-    ns1:wikidata "Q11364144" ;
+[] ns1:wikidata "Q11364144" ;
     ns1:名 "展宏" ;
     ns1:名前 "中山展宏" ;
     ns1:姓 "中山" ;
@@ -138,8 +131,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1970-04-25" .
 
-[] ns1:twitter "nakagawa_komei" ;
-    ns1:wikidata "Q20041786" ;
+[] ns1:wikidata "Q20041786" ;
     ns1:名 "康洋" ;
     ns1:名前 "中川康洋" ;
     ns1:姓 "中川" ;
@@ -161,16 +153,14 @@
     ns1:性別 "女性" ;
     ns1:生年月日 "1958-12-22" .
 
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q6416606" ;
+[] ns1:wikidata "Q6416606" ;
     ns1:名 "喜四郎" ;
     ns1:名前 "中村喜四郎" ;
     ns1:姓 "中村" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1949-04-10" .
 
-[] ns1:twitter "nakamurahiro3" ;
-    ns1:wikidata "Q11365989" ;
+[] ns1:wikidata "Q11365989" ;
     ns1:名 "裕之" ;
     ns1:名前 "中村裕之" ;
     ns1:姓 "中村" ;
@@ -229,8 +219,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1972-12-20" .
 
-[] ns1:twitter "niwayuyaibaraki" ;
-    ns1:wikidata "Q861808" ;
+[] ns1:wikidata "Q861808" ;
     ns1:名 "雄哉" ;
     ns1:名前 "丹羽雄哉" ;
     ns1:姓 "丹羽" ;
@@ -260,8 +249,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1939-02-17" .
 
-[] ns1:twitter "shinrai_seiji" ;
-    ns1:wikidata "Q7497583" ;
+[] ns1:wikidata "Q7497583" ;
     ns1:名 "信治" ;
     ns1:名前 "井上信治" ;
     ns1:姓 "井上" ;
@@ -313,8 +301,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1976-07-18" .
 
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q11373778" ;
+[] ns1:wikidata "Q11373778" ;
     ns1:名 "俊郎" ;
     ns1:名前 "井野俊郎" ;
     ns1:姓 "井野" ;
@@ -344,8 +331,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1984-02-18" .
 
-[] ns1:twitter "imazu164" ;
-    ns1:wikidata "Q11377294" ;
+[] ns1:wikidata "Q11377294" ;
     ns1:名 "寛" ;
     ns1:名前 "今津寛" ;
     ns1:姓 "今津" ;
@@ -419,8 +405,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1969-11-13" .
 
-[] ns1:twitter "tatsuyaito0501" ;
-    ns1:wikidata "Q614476" ;
+[] ns1:wikidata "Q614476" ;
     ns1:名 "達也" ;
     ns1:名前 "伊藤達也" ;
     ns1:姓 "伊藤" ;
@@ -524,8 +509,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1969-04-15" .
 
-[] ns1:twitter "maehara201" ;
-    ns1:wikidata "Q132280" ;
+[] ns1:wikidata "Q132280" ;
     ns1:名 "誠司" ;
     ns1:名前 "前原誠司" ;
     ns1:姓 "前原" ;
@@ -539,8 +523,7 @@
     ns1:性別 "女性" ;
     ns1:生年月日 "1975-08-16" .
 
-[] ns1:twitter "donanmaeda" ;
-    ns1:wikidata "Q11397382" ;
+[] ns1:wikidata "Q11397382" ;
     ns1:名 "一男" ;
     ns1:名前 "前田一男" ;
     ns1:姓 "前田" ;
@@ -562,8 +545,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1946-04-24" .
 
-[] ns1:twitter "katoayuko_and_s" ;
-    ns1:wikidata "Q18701018" ;
+[] ns1:wikidata "Q18701018" ;
     ns1:名 "鮎子" ;
     ns1:名前 "加藤鮎子" ;
     ns1:姓 "加藤" ;
@@ -614,6 +596,13 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1951-11-08" .
 
+[] ns1:wikidata "Q7496361" ;
+    ns1:名 "茂男" ;
+    ns1:名前 "北村茂男" ;
+    ns1:姓 "北村" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1945-11-08" .
+
 [] ns1:wikidata "Q7446686" ;
     ns1:名 "誠吾" ;
     ns1:名前 "北村誠吾" ;
@@ -724,7 +713,7 @@
     ns1:名前 "吉川元" ;
     ns1:姓 "吉川" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1966-9-28" .
+    ns1:生年月日 "1966-09-28" .
 
 [] ns1:wikidata "Q7677156" ;
     ns1:名 "貴盛" ;
@@ -733,13 +722,12 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1950-10-20" .
 
-[] ns1:twitter "yoshida_komei" ;
-    ns1:wikidata "Q21703015" ;
+[] ns1:wikidata "Q21703015" ;
     ns1:名 "宣弘" ;
     ns1:名前 "吉田宣弘" ;
     ns1:姓 "吉田" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1967-12-8" .
+    ns1:生年月日 "1967-12-08" .
 
 [] ns1:wikidata "Q6102348" ;
     ns1:名 "泉" ;
@@ -748,8 +736,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1949-02-26" .
 
-[] ns1:twitter "toyo23office" ;
-    ns1:wikidata "Q21655857" ;
+[] ns1:wikidata "Q21655857" ;
     ns1:名 "豊史" ;
     ns1:名前 "吉田豊史" ;
     ns1:姓 "吉田" ;
@@ -770,8 +757,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1948-08-08" .
 
-[] ns1:twitter "ywada_hokkaido5" ;
-    ns1:wikidata "Q23907516" ;
+[] ns1:wikidata "Q23907516" ;
     ns1:名 "義明" ;
     ns1:名前 "和田義明" ;
     ns1:姓 "和田" ;
@@ -781,16 +767,16 @@
 [] ns1:twitter "kokubkonosuke" ;
     ns1:wikidata "Q11422694" ;
     ns1:名 "幸之助" ;
-    ns1:名前 "國場幸之助" ;
-    ns1:姓 "國場" ;
-    ns1:性別 "男性" ;
-    ns1:生年月日 "1973-1-10" .
+    ns1:名前 "国場幸之助" ;
+    ns1:姓 "国場" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1973-01-10" .
 
 [] ns1:twitter "torukunishige" ;
     ns1:wikidata "Q11422753" ;
     ns1:名 "徹" ;
-    ns1:名前 "國重徹" ;
-    ns1:姓 "國重" ;
+    ns1:名前 "国重徹" ;
+    ns1:姓 "国重" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1974-11-23" .
 
@@ -816,8 +802,7 @@
     ns1:性別 "女性" ;
     ns1:生年月日 "1952-02-09" .
 
-[] ns1:twitter "tsuchiyamasatad" ;
-    ns1:wikidata "Q6782686" ;
+[] ns1:wikidata "Q6782686" ;
     ns1:名 "正忠" ;
     ns1:名前 "土屋正忠" ;
     ns1:姓 "土屋" ;
@@ -861,8 +846,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1972-02-19" .
 
-[] ns1:twitter "teruhoriuchi" ;
-    ns1:wikidata "Q18818101" ;
+[] ns1:wikidata "Q18818101" ;
     ns1:名 "照文" ;
     ns1:名前 "堀内照文" ;
     ns1:姓 "堀内" ;
@@ -899,8 +883,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1950-02-18" .
 
-[] ns1:twitter "oogushihiroshi" ;
-    ns1:wikidata "Q5771003" ;
+[] ns1:wikidata "Q5771003" ;
     ns1:名 "博志" ;
     ns1:名前 "大串博志" ;
     ns1:姓 "大串" ;
@@ -948,7 +931,7 @@
     ns1:名前 "大島敦" ;
     ns1:姓 "大島" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1956-01-01" .
+    ns1:生年月日 "1956-12-21" .
 
 [] ns1:twitter "tadamori_oshima" ;
     ns1:wikidata "Q49998" ;
@@ -958,16 +941,14 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1946-09-06" .
 
-[] ns1:twitter "tekuteku_diary" ;
-    ns1:wikidata "Q30144920" ;
+[] ns1:wikidata "Q30144920" ;
     ns1:名 "喜信" ;
     ns1:名前 "大平喜信" ;
     ns1:姓 "大平" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1978-02-28" .
 
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q418119" ;
+[] ns1:wikidata "Q418119" ;
     ns1:名 "章宏" ;
     ns1:名前 "大畠章宏" ;
     ns1:姓 "大畠" ;
@@ -982,28 +963,27 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1971-04-13" .
 
-[] ns1:twitter "onishi_hiroyuki" ;
-    ns1:wikidata "Q20039110" ;
+[] ns1:wikidata "Q20039110" ;
     ns1:名 "宏幸" ;
     ns1:名前 "大西宏幸" ;
     ns1:姓 "大西" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1967-07-15" .
 
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q11438916" ;
+[] ns1:wikidata "Q11438916" ;
     ns1:名 "英男" ;
     ns1:名前 "大西英男" ;
     ns1:姓 "大西" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1946-08-28" .
 
-[] ns1:wikidata "Q11438945" ;
+[] ns1:twitter "ohmisei" ;
+    ns1:wikidata "Q11438945" ;
     ns1:名 "正" ;
     ns1:名前 "大見正" ;
     ns1:姓 "大見" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1958-6-20" .
+    ns1:生年月日 "1958-06-20" .
 
 [] ns1:wikidata "Q11439619" ;
     ns1:名 "敬太郎" ;
@@ -1019,16 +999,14 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1969-08-23" .
 
-[] ns1:twitter "ota_kazumi" ;
-    ns1:wikidata "Q6381578" ;
+[] ns1:wikidata "Q6381578" ;
     ns1:名 "和美" ;
     ns1:名前 "太田和美" ;
     ns1:姓 "太田" ;
     ns1:性別 "女性" ;
     ns1:生年月日 "1979-08-28" .
 
-[] ns1:twitter "teamohta" ;
-    ns1:wikidata "Q337406" ;
+[] ns1:wikidata "Q337406" ;
     ns1:名 "昭宏" ;
     ns1:名前 "太田昭宏" ;
     ns1:姓 "太田" ;
@@ -1072,8 +1050,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1965-03-28" .
 
-[] ns1:twitter "munekiyooffice" ;
-    ns1:wikidata "Q18818207" ;
+[] ns1:wikidata "Q18818207" ;
     ns1:名 "皇一" ;
     ns1:名前 "宗清皇一" ;
     ns1:姓 "宗清" ;
@@ -1102,8 +1079,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1970-02-14" .
 
-[] ns1:twitter "miyazaki_kirin" ;
-    ns1:wikidata "Q11454333" ;
+[] ns1:wikidata "Q11454333" ;
     ns1:名 "政久" ;
     ns1:名前 "宮崎政久" ;
     ns1:姓 "宮崎" ;
@@ -1124,8 +1100,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1959-12-25" .
 
-[] ns1:twitter "miyamototooru" ;
-    ns1:wikidata "Q18818200" ;
+[] ns1:wikidata "Q18818200" ;
     ns1:名 "徹" ;
     ns1:名前 "宮本徹" ;
     ns1:姓 "宮本" ;
@@ -1144,7 +1119,7 @@
     ns1:名前 "宮路拓馬" ;
     ns1:姓 "宮路" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1979-12-6" .
+    ns1:生年月日 "1979-12-06" .
 
 [] ns1:wikidata "Q7496423" ;
     ns1:名 "茂之" ;
@@ -1153,13 +1128,12 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1953-10-01" .
 
-[] ns1:twitter "teratamanabu" ;
-    ns1:wikidata "Q6746585" ;
+[] ns1:wikidata "Q6746585" ;
     ns1:名 "学" ;
     ns1:名前 "寺田学" ;
     ns1:姓 "寺田" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "-" .
+    ns1:生年月日 "1976-09-20" .
 
 [] ns1:twitter "terada124" ;
     ns1:wikidata "Q6869306" ;
@@ -1171,8 +1145,8 @@
 
 [] ns1:twitter "masanobu_ogura" ;
     ns1:wikidata "Q11458942" ;
-    ns1:名 "將信" ;
-    ns1:名前 "小倉將信" ;
+    ns1:名 "将信" ;
+    ns1:名前 "小倉将信" ;
     ns1:姓 "小倉" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1981-05-30" .
@@ -1197,7 +1171,7 @@
     ns1:名前 "小島敏文" ;
     ns1:姓 "小島" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1950-9-7" .
+    ns1:生年月日 "1950-09-07" .
 
 [] ns1:twitter "junyaog" ;
     ns1:wikidata "Q6314413" ;
@@ -1268,8 +1242,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1952-09-17" .
 
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q455848" ;
+[] ns1:wikidata "Q455848" ;
     ns1:名 "優子" ;
     ns1:名前 "小渕優子" ;
     ns1:姓 "小渕" ;
@@ -1286,8 +1259,8 @@
 
 [] ns1:twitter "odawarakiyoshi" ;
     ns1:wikidata "Q11462993" ;
-    ns1:名 "潔" ;
-    ns1:名前 "小田原潔" ;
+    ns1:名 "きよし" ;
+    ns1:名前 "小田原きよし" ;
     ns1:姓 "小田原" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1964-05-23" .
@@ -1367,8 +1340,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1947-09-04" .
 
-[] ns1:twitter "yamamotogiin" ;
-    ns1:wikidata "Q6435368" ;
+[] ns1:wikidata "Q6435368" ;
     ns1:名 "幸三" ;
     ns1:名前 "山本幸三" ;
     ns1:姓 "山本" ;
@@ -1381,7 +1353,7 @@
     ns1:名前 "山本拓" ;
     ns1:姓 "山本" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1952-7-7" .
+    ns1:生年月日 "1952-07-07" .
 
 [] ns1:wikidata "Q8060529" ;
     ns1:名 "有二" ;
@@ -1429,8 +1401,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1975-03-08" .
 
-[] ns1:twitter "okamotooffice" ;
-    ns1:wikidata "Q16264927" ;
+[] ns1:wikidata "Q16264927" ;
     ns1:名 "三成" ;
     ns1:名前 "岡本三成" ;
     ns1:姓 "岡本" ;
@@ -1452,8 +1423,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1953-07-14" .
 
-[] ns1:twitter "takeshi108" ;
-    ns1:wikidata "Q1654639" ;
+[] ns1:wikidata "Q1654639" ;
     ns1:名 "毅" ;
     ns1:名前 "岩屋毅" ;
     ns1:姓 "岩屋" ;
@@ -1466,7 +1436,7 @@
     ns1:名前 "岩田和親" ;
     ns1:姓 "岩田" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1973-9-20" .
+    ns1:生年月日 "1973-09-20" .
 
 [] ns1:twitter "kishinobuo" ;
     ns1:wikidata "Q1995125" ;
@@ -1491,13 +1461,12 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1957-07-29" .
 
-[] ns1:twitter "u_simazu" ;
-    ns1:wikidata "Q18817650" ;
+[] ns1:wikidata "Q18817650" ;
     ns1:名 "幸広" ;
     ns1:名前 "島津幸広" ;
     ns1:姓 "島津" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1956-10-19" .
+    ns1:生年月日 "1956-10-01" .
 
 [] ns1:wikidata "Q17159494" ;
     ns1:名 "佳和" ;
@@ -1513,8 +1482,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1947-11-15" .
 
-[] ns1:twitter "tatsuo_kawabata" ;
-    ns1:wikidata "Q1131518" ;
+[] ns1:wikidata "Q1131518" ;
     ns1:名 "達夫" ;
     ns1:名前 "川端達夫" ;
     ns1:姓 "川端" ;
@@ -1539,8 +1507,8 @@
 
 [] ns1:twitter "hiratakuchan" ;
     ns1:wikidata "Q7678622" ;
-    ns1:名 "卓也" ;
-    ns1:名前 "平井卓也" ;
+    ns1:名 "たくや" ;
+    ns1:名前 "平井たくや" ;
     ns1:姓 "平井" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1958-01-25" .
@@ -1575,16 +1543,14 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1939-08-03" .
 
-[] ns1:twitter "hiranohirofumi" ;
-    ns1:wikidata "Q1155448" ;
+[] ns1:wikidata "Q1155448" ;
     ns1:名 "博文" ;
     ns1:名前 "平野博文" ;
     ns1:姓 "平野" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1949-03-19" .
 
-[] ns1:twitter "gotodamasazumi" ;
-    ns1:wikidata "Q6782819" ;
+[] ns1:wikidata "Q6782819" ;
     ns1:名 "正純" ;
     ns1:名前 "後藤田正純" ;
     ns1:姓 "後藤田" ;
@@ -1721,8 +1687,7 @@
     ns1:性別 "女性" ;
     ns1:生年月日 "1965-08-11" .
 
-[] ns1:twitter "motomura_nobuko" ;
-    ns1:wikidata "Q18818221" ;
+[] ns1:wikidata "Q18818221" ;
     ns1:名 "伸子" ;
     ns1:名前 "本村伸子" ;
     ns1:姓 "本村" ;
@@ -1775,8 +1740,7 @@
     ns1:性別 "女性" ;
     ns1:生年月日 "1956-07-15" .
 
-[] ns1:twitter "kenkou64865187" ;
-    ns1:wikidata "Q6389412" ;
+[] ns1:wikidata "Q6389412" ;
     ns1:名 "謙公" ;
     ns1:名前 "松木謙公" ;
     ns1:姓 "松木" ;
@@ -1856,8 +1820,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1964-05-31" .
 
-[] ns1:twitter "yunoki_m" ;
-    ns1:wikidata "Q6837954" ;
+[] ns1:wikidata "Q6837954" ;
     ns1:名 "道義" ;
     ns1:名前 "柚木道義" ;
     ns1:姓 "柚木" ;
@@ -1910,14 +1873,12 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1951-04-03" .
 
-[] ns1:twitter "umemura_saeko" ;
-    ns1:wikidata "Q18700742" ;
+[] ns1:wikidata "Q18700742" ;
     ns1:名 "早江子" ;
     ns1:名前 "梅村早江子" ;
     ns1:姓 "梅村" ;
     ns1:性別 "女性" ;
-    ns1:生年月日 "1964-06-28",
-        "1964-6-28" .
+    ns1:生年月日 "1964-06-28" .
 
 [] ns1:twitter "kajiyamahiroshi" ;
     ns1:wikidata "Q5770884" ;
@@ -1935,11 +1896,11 @@
     ns1:生年月日 "1963-02-11" .
 
 [] ns1:wikidata "Q337762" ;
-    ns1:名 "𥙿" ;
-    ns1:名前 "森山𥙿" ;
+    ns1:名 "裕" ;
+    ns1:名前 "森山裕" ;
     ns1:姓 "森山" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1945-4-8" .
+    ns1:生年月日 "1945-04-08" .
 
 [] ns1:wikidata "Q1133194" ;
     ns1:名 "英介" ;
@@ -1948,8 +1909,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1948-08-31" .
 
-[] ns1:twitter "shiiki_tamotsu_" ;
-    ns1:wikidata "Q11647123" ;
+[] ns1:wikidata "Q11647123" ;
     ns1:名 "保" ;
     ns1:名前 "椎木保" ;
     ns1:姓 "椎木" ;
@@ -1969,6 +1929,13 @@
     ns1:姓 "横山" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1950-11-06" .
+
+[] ns1:wikidata "Q130856" ;
+    ns1:名 "孝弘" ;
+    ns1:名前 "横路孝弘" ;
+    ns1:姓 "横路" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1941-01-03" .
 
 [] ns1:twitter "ga9_h" ;
     ns1:wikidata "Q5517695" ;
@@ -2105,15 +2072,13 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1960-07-01" .
 
-[] ns1:twitter "ikeuchi_saori",
-        "saorijimusyo" ;
+[] ns1:twitter "ikeuchi_saori" ;
     ns1:wikidata "Q18669603" ;
     ns1:名 "沙織" ;
     ns1:名前 "池内沙織" ;
     ns1:姓 "池内" ;
     ns1:性別 "女性" ;
-    ns1:生年月日 "1982-09-15",
-        "1982-9-15" .
+    ns1:生年月日 "1982-09-15" .
 
 [] ns1:twitter "ikeda_0620" ;
     ns1:wikidata "Q11551703" ;
@@ -2137,8 +2102,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1963-03-11" .
 
-[] ns1:twitter "takeokawamura" ;
-    ns1:wikidata "Q382311" ;
+[] ns1:wikidata "Q382311" ;
     ns1:名 "建夫" ;
     ns1:名前 "河村建夫" ;
     ns1:姓 "河村" ;
@@ -2153,16 +2117,14 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1963-01-10" .
 
-[] ns1:twitter "mkawano2416" ;
-    ns1:wikidata "Q11554229" ;
+[] ns1:wikidata "Q11554229" ;
     ns1:名 "正美" ;
     ns1:名前 "河野正美" ;
     ns1:姓 "河野" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1961-6-30" .
-
-[] ns1:twitter "office50824963" ;
-    ns1:wikidata "Q6391984" ;
+    ns1:生年月日 "1961-06-30" .
+
+[] ns1:wikidata "Q6391984" ;
     ns1:名 "健太" ;
     ns1:名前 "泉健太" ;
     ns1:姓 "泉" ;
@@ -2177,8 +2139,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1966-10-18" .
 
-[] ns1:twitter "tsumura_keisuke" ;
-    ns1:wikidata "Q6383911" ;
+[] ns1:wikidata "Q6383911" ;
     ns1:名 "啓介" ;
     ns1:名前 "津村啓介" ;
     ns1:姓 "津村" ;
@@ -2196,10 +2157,10 @@
 [] ns1:twitter "hamachi_m" ;
     ns1:wikidata "Q17220479" ;
     ns1:名 "雅一" ;
-    ns1:名前 "濱地雅一" ;
-    ns1:姓 "濱地" ;
-    ns1:性別 "男性" ;
-    ns1:生年月日 "1970-5-8" .
+    ns1:名前 "浜地雅一" ;
+    ns1:姓 "浜地" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1970-05-08" .
 
 [] ns1:wikidata "Q1132836" ;
     ns1:名 "靖一" ;
@@ -2218,16 +2179,15 @@
 
 [] ns1:twitter "tomo_ukishima" ;
     ns1:wikidata "Q7820173" ;
-    ns1:名 "智子" ;
-    ns1:名前 "浮島智子" ;
+    ns1:名 "とも子" ;
+    ns1:名前 "浮島とも子" ;
     ns1:姓 "浮島" ;
     ns1:性別 "女性" ;
     ns1:生年月日 "1963-02-01" .
 
-[] ns1:twitter "tadashishimizu" ;
-    ns1:wikidata "Q11560960" ;
-    ns1:名 "忠史" ;
-    ns1:名前 "清水忠史" ;
+[] ns1:wikidata "Q11560960" ;
+    ns1:名 "ただし" ;
+    ns1:名前 "清水ただし" ;
     ns1:姓 "清水" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1968-05-05" .
@@ -2366,16 +2326,14 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1963-11-11" .
 
-[] ns1:twitter "tanaka_hide" ;
-    ns1:wikidata "Q17160872" ;
+[] ns1:wikidata "Q17160872" ;
     ns1:名 "英之" ;
     ns1:名前 "田中英之" ;
     ns1:姓 "田中" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1970-07-11" .
 
-[] ns1:twitter "tajimaissei" ;
-    ns1:wikidata "Q6088170" ;
+[] ns1:wikidata "Q6088170" ;
     ns1:名 "一成" ;
     ns1:名前 "田島一成" ;
     ns1:姓 "田島" ;
@@ -2405,7 +2363,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1964-12-15" .
 
-[] ns1:twitter "tamuratakaaki" ;
+[] ns1:twitter "@tamuratakaaki" ;
     ns1:wikidata "Q18817798" ;
     ns1:名 "貴昭" ;
     ns1:名前 "田村貴昭" ;
@@ -2434,16 +2392,14 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1974-07-04" .
 
-[] ns1:twitter "hatanokimie" ;
-    ns1:wikidata "Q11578505" ;
+[] ns1:wikidata "Q11578505" ;
     ns1:名 "君枝" ;
     ns1:名前 "畑野君枝" ;
     ns1:姓 "畑野" ;
     ns1:性別 "女性" ;
     ns1:生年月日 "1957-01-19" .
 
-[] ns1:twitter "hatayama_kazuya" ;
-    ns1:wikidata "Q18817972" ;
+[] ns1:wikidata "Q18817972" ;
     ns1:名 "和也" ;
     ns1:名前 "畠山和也" ;
     ns1:姓 "畠山" ;
@@ -2503,8 +2459,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1964-06-19" .
 
-[] ns1:twitter "tohruishizaki" ;
-    ns1:wikidata "Q11585604" ;
+[] ns1:wikidata "Q11585604" ;
     ns1:名 "徹" ;
     ns1:名前 "石崎徹" ;
     ns1:姓 "石崎" ;
@@ -2525,8 +2480,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1952-04-11" .
 
-[] ns1:twitter "ishidanoritoshi" ;
-    ns1:wikidata "Q7051352" ;
+[] ns1:wikidata "Q7051352" ;
     ns1:名 "祝稔" ;
     ns1:名前 "石田祝稔" ;
     ns1:姓 "石田" ;
@@ -2552,6 +2506,7 @@
     ns1:名 "佐市" ;
     ns1:名前 "神山佐市" ;
     ns1:姓 "神山" ;
+    ns1:性別 "男性" ;
     ns1:生年月日 "1954-08-17" .
 
 [] ns1:twitter "kamiyama_yosuke" ;
@@ -2605,16 +2560,14 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1964-04-08" .
 
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q3435725" ;
+[] ns1:wikidata "Q3435725" ;
     ns1:名 "昭夫" ;
     ns1:名前 "福田昭夫" ;
     ns1:姓 "福田" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1948-04-17" .
 
-[] ns1:twitter "tatsuofukuda" ;
-    ns1:wikidata "Q2506910" ;
+[] ns1:wikidata "Q2506910" ;
     ns1:名 "達夫" ;
     ns1:名前 "福田達夫" ;
     ns1:姓 "福田" ;
@@ -2645,8 +2598,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1962-07-03" .
 
-[] ns1:twitter "inatsuhisashi" ;
-    ns1:wikidata "Q11596723" ;
+[] ns1:wikidata "Q11596723" ;
     ns1:名 "久" ;
     ns1:名前 "稲津久" ;
     ns1:姓 "稲津" ;
@@ -2688,7 +2640,7 @@
     ns1:名前 "竹内譲" ;
     ns1:姓 "竹内" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1958-6-25" .
+    ns1:生年月日 "1958-06-25" .
 
 [] ns1:twitter "naokazutakemoto" ;
     ns1:wikidata "Q614464" ;
@@ -2713,8 +2665,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1965-01-03" .
 
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q11602573" ;
+[] ns1:wikidata "Q11602573" ;
     ns1:名 "博義" ;
     ns1:名前 "笹川博義" ;
     ns1:姓 "笹川" ;
@@ -2728,16 +2679,14 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1948-07-17" .
 
-[] ns1:twitter "go_shinohara" ;
-    ns1:wikidata "Q20040516" ;
+[] ns1:wikidata "Q20040516" ;
     ns1:名 "豪" ;
     ns1:名前 "篠原豪" ;
     ns1:姓 "篠原" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1975-2-12" .
-
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q11603860" ;
+    ns1:生年月日 "1975-02-12" .
+
+[] ns1:wikidata "Q11603860" ;
     ns1:名 "和生" ;
     ns1:名前 "簗和生" ;
     ns1:姓 "簗" ;
@@ -2798,8 +2747,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1961-09-02" .
 
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q11616618" ;
+[] ns1:wikidata "Q11616618" ;
     ns1:名 "勝" ;
     ns1:名前 "若狭勝" ;
     ns1:姓 "若狭" ;
@@ -2814,8 +2762,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1955-10-07" .
 
-[] ns1:twitter "satoshisatoshia" ;
-    ns1:wikidata "Q1478311" ;
+[] ns1:wikidata "Q1478311" ;
     ns1:名 "聰" ;
     ns1:名前 "荒井聰" ;
     ns1:姓 "荒井" ;
@@ -2846,24 +2793,21 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1946-10-10" .
 
-[] ns1:twitter "sugawitter" ;
-    ns1:wikidata "Q122465" ;
+[] ns1:wikidata "Q122465" ;
     ns1:名 "義偉" ;
     ns1:名前 "菅義偉" ;
     ns1:姓 "菅" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1948-12-06" .
 
-[] ns1:twitter "kikuta_official" ;
-    ns1:wikidata "Q6739516" ;
+[] ns1:wikidata "Q6739516" ;
     ns1:名 "真紀子" ;
     ns1:名前 "菊田真紀子" ;
     ns1:姓 "菊田" ;
     ns1:性別 "女性" ;
     ns1:生年月日 "1969-10-24" .
 
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q6426211" ;
+[] ns1:wikidata "Q6426211" ;
     ns1:名 "光一" ;
     ns1:名前 "萩生田光一" ;
     ns1:姓 "萩生田" ;
@@ -2916,8 +2860,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1983-08-02" .
 
-[] ns1:twitter "fujinofujinooo" ;
-    ns1:wikidata "Q18818065" ;
+[] ns1:wikidata "Q18818065" ;
     ns1:名 "保史" ;
     ns1:名前 "藤野保史" ;
     ns1:姓 "藤野" ;
@@ -2954,8 +2897,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1960-07-16" .
 
-[] ns1:twitter "chinami_niigata" ;
-    ns1:wikidata "Q5100105" ;
+[] ns1:wikidata "Q5100105" ;
     ns1:名 "智奈美" ;
     ns1:名前 "西村智奈美" ;
     ns1:姓 "西村" ;
@@ -2982,6 +2924,14 @@
     ns1:姓 "谷" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1952-01-28" .
+
+[] ns1:twitter "tanigaki_s" ;
+    ns1:wikidata "Q463331" ;
+    ns1:名 "禎一" ;
+    ns1:名前 "谷垣禎一" ;
+    ns1:姓 "谷垣" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1945-03-07" .
 
 [] ns1:wikidata "Q20041303" ;
     ns1:名 "とむ" ;
@@ -3037,8 +2987,8 @@
 [] ns1:twitter "ryosei_akazawa" ;
     ns1:wikidata "Q7385320" ;
     ns1:名 "亮正" ;
-    ns1:名前 "赤澤亮正" ;
-    ns1:姓 "赤澤" ;
+    ns1:名前 "赤沢亮正" ;
+    ns1:姓 "赤沢" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1960-12-18" .
 
@@ -3068,22 +3018,20 @@
 
 [] ns1:twitter "adachiyasushi" ;
     ns1:wikidata "Q11637002" ;
-    ns1:名 "康史" ;
-    ns1:名前 "足立康史" ;
+    ns1:名 "やすし" ;
+    ns1:名前 "足立やすし" ;
     ns1:姓 "足立" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1965-10-14" .
 
-[] ns1:twitter "koshimizu_k1" ;
-    ns1:wikidata "Q11637774" ;
+[] ns1:wikidata "Q11637774" ;
     ns1:名 "恵一" ;
     ns1:名前 "輿水恵一" ;
     ns1:姓 "輿水" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1962-02-04" .
 
-[] ns1:twitter "tsujimotokiyomi" ;
-    ns1:wikidata "Q405533" ;
+[] ns1:wikidata "Q405533" ;
     ns1:名 "清美" ;
     ns1:名前 "辻元清美" ;
     ns1:姓 "辻元" ;
@@ -3105,16 +3053,14 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1958-05-26" .
 
-[] ns1:twitter "kondoyosuke21" ;
-    ns1:wikidata "Q49943" ;
+[] ns1:wikidata "Q49943" ;
     ns1:名 "洋介" ;
     ns1:名前 "近藤洋介" ;
     ns1:姓 "近藤" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1965-05-19" .
 
-[] ns1:twitter "seiji_ohsaka" ;
-    ns1:wikidata "Q7446737" ;
+[] ns1:wikidata "Q7446737" ;
     ns1:名 "誠二" ;
     ns1:名前 "逢坂誠二" ;
     ns1:姓 "逢坂" ;
@@ -3166,8 +3112,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1976-11-17" .
 
-[] ns1:twitter "nodayoshi55" ;
-    ns1:wikidata "Q57647" ;
+[] ns1:wikidata "Q57647" ;
     ns1:名 "佳彦" ;
     ns1:名前 "野田佳彦" ;
     ns1:姓 "野田" ;
@@ -3194,6 +3139,13 @@
     ns1:姓 "野間" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1958-10-08" .
+
+[] ns1:wikidata "Q382231" ;
+    ns1:名 "一義" ;
+    ns1:名前 "金子一義" ;
+    ns1:姓 "金子" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1942-12-20" .
 
 [] ns1:wikidata "Q16770033" ;
     ns1:名 "万寿夫" ;
@@ -3210,7 +3162,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1961-02-27" .
 
-[] ns1:twitter "megumi_kaneko4" ;
+[] ns1:twitter "kanekomegu" ;
     ns1:wikidata "Q11646784",
         "Q5371065" ;
     ns1:名 "恵美" ;
@@ -3278,8 +3230,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1977-08-08" .
 
-[] ns1:twitter "-" ;
-    ns1:wikidata "Q6383909" ;
+[] ns1:wikidata "Q6383909" ;
     ns1:名 "馨祐" ;
     ns1:名前 "鈴木馨祐" ;
     ns1:姓 "鈴木" ;
@@ -3301,8 +3252,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1960-06-14" .
 
-[] ns1:twitter "takashinagao" ;
-    ns1:wikidata "Q11651845" ;
+[] ns1:wikidata "Q11651845" ;
     ns1:名 "敬" ;
     ns1:名前 "長尾敬" ;
     ns1:姓 "長尾" ;
@@ -3385,8 +3335,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1944-01-11" .
 
-[] ns1:twitter "baba_ishin" ;
-    ns1:wikidata "Q11667848" ;
+[] ns1:wikidata "Q11667848" ;
     ns1:名 "伸幸" ;
     ns1:名前 "馬場伸幸" ;
     ns1:姓 "馬場" ;
@@ -3408,8 +3357,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1961-05-05" .
 
-[] ns1:twitter "t_takai" ;
-    ns1:wikidata "Q11668691" ;
+[] ns1:wikidata "Q11668691" ;
     ns1:名 "崇志" ;
     ns1:名前 "高井崇志" ;
     ns1:姓 "高井" ;
@@ -3423,10 +3371,9 @@
     ns1:性別 "女性" ;
     ns1:生年月日 "1961-03-07" .
 
-[] ns1:twitter "takagi_hirohisa" ;
-    ns1:wikidata "Q11670038" ;
-    ns1:名 "宏壽" ;
-    ns1:名前 "高木宏壽" ;
+[] ns1:wikidata "Q11670038" ;
+    ns1:名 "宏寿" ;
+    ns1:名前 "高木宏寿" ;
     ns1:姓 "高木" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1960-04-09" .
@@ -3467,8 +3414,8 @@
     ns1:生年月日 "1942-03-15" .
 
 [] ns1:wikidata "Q11671375" ;
-    ns1:名 "比奈子" ;
-    ns1:名前 "高橋比奈子" ;
+    ns1:名 "ひなこ" ;
+    ns1:名前 "高橋ひなこ" ;
     ns1:姓 "高橋" ;
     ns1:性別 "女性" ;
     ns1:生年月日 "1958-01-19" .
@@ -3501,18 +3448,16 @@
     ns1:名前 "鳩山二郎" ;
     ns1:姓 "鳩山" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1979-01-13" .
-
-[] ns1:twitter "dr_kamo" ;
-    ns1:wikidata "Q730991" ;
+    ns1:生年月日 "1979-01-01" .
+
+[] ns1:wikidata "Q730991" ;
     ns1:名 "一郎" ;
     ns1:名前 "鴨下一郎" ;
     ns1:姓 "鴨下" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1949-01-16" .
 
-[] ns1:twitter "washioeiichirou" ;
-    ns1:wikidata "Q5349200" ;
+[] ns1:wikidata "Q5349200" ;
     ns1:名 "英一郎" ;
     ns1:名前 "鷲尾英一郎" ;
     ns1:姓 "鷲尾" ;
@@ -3535,8 +3480,14 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1970-10-13" .
 
-[] ns1:twitter "kuroiwatakahir1" ;
-    ns1:wikidata "Q11678114" ;
+[] ns1:wikidata "Q7827415" ;
+    ns1:名 "徹" ;
+    ns1:名前 "黄川田徹" ;
+    ns1:姓 "黄川田" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1953-10-14" .
+
+[] ns1:wikidata "Q11678114" ;
     ns1:名 "宇洋" ;
     ns1:名前 "黒岩宇洋" ;
     ns1:姓 "黒岩" ;
