--- GrayDB
+++ 公明党公式
@@ -6,7 +6,7 @@
 
 [] ns1:facebook "https://www.facebook.com/hamachi.masakazu" ;
     ns1:twitter "hamachi_m" ;
-    ns1:候補名 "浜地雅一" ;
+    ns1:候補名 "濵地雅一" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "九州" .
 
@@ -16,8 +16,7 @@
     ns1:小選挙区 "大阪6区" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/inatsu.komei",
-        "https://www.facebook.com/inatsu.news" ;
+[] ns1:facebook "https://www.facebook.com/inatsu.komei" ;
     ns1:twitter "inatsuhisashi" ;
     ns1:候補名 "いなつ久" ;
     ns1:小選挙区 "北海道10区" ;
@@ -42,7 +41,7 @@
     ns1:比例区 "東海" .
 
 [] ns1:facebook "https://www.facebook.com/nakanohiromasa.office" ;
-    ns1:twitter "hiro_nakano_" ;
+    ns1:twitter "" ;
     ns1:候補名 "中野ひろまさ" ;
     ns1:小選挙区 "兵庫8区" ;
     ns1:比例区 "" .
@@ -53,8 +52,8 @@
     ns1:小選挙区 "" ;
     ns1:比例区 "東北" .
 
-[] ns1:facebook "https://www.facebook.com/yasuo.ikawa.1" ;
-    ns1:twitter "yamanashi_yasuo" ;
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:候補名 "井川泰雄" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" .
@@ -65,7 +64,7 @@
     ns1:小選挙区 "" ;
     ns1:比例区 "東海" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/komei.itowataru" ;
     ns1:twitter "ito_wataru" ;
     ns1:候補名 "伊藤渉" ;
     ns1:小選挙区 "" ;
@@ -95,7 +94,7 @@
     ns1:小選挙区 "大阪16区" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/noriko.furuya514" ;
+[] ns1:facebook "https://www.facebook.com/noriko.furuya.jp" ;
     ns1:twitter "noriko_furuya" ;
     ns1:候補名 "古屋範子" ;
     ns1:小選挙区 "" ;
@@ -107,7 +106,7 @@
     ns1:小選挙区 "" ;
     ns1:比例区 "九州" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/nobuhiro.yoshida.940" ;
     ns1:twitter "yoshida_komei" ;
     ns1:候補名 "吉田宣弘" ;
     ns1:小選挙区 "" ;
@@ -119,7 +118,7 @@
     ns1:小選挙区 "大阪5区" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/OguchiYoshinori" ;
     ns1:twitter "" ;
     ns1:候補名 "大口善徳" ;
     ns1:小選挙区 "" ;
@@ -143,14 +142,14 @@
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/okamoto.office" ;
     ns1:twitter "okamotooffice" ;
     ns1:候補名 "岡本三成" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "北関東" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "papiya396" ;
+    ns1:twitter "" ;
     ns1:候補名 "布施正行" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" .
@@ -191,7 +190,7 @@
     ns1:小選挙区 "" ;
     ns1:比例区 "北海道" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/edayasuyuki.official" ;
     ns1:twitter "" ;
     ns1:候補名 "江田康幸" ;
     ns1:小選挙区 "" ;
@@ -239,7 +238,7 @@
     ns1:小選挙区 "" ;
     ns1:比例区 "北関東" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/noritoshi.ishida.komei" ;
     ns1:twitter "ishidanoritoshi" ;
     ns1:候補名 "石田祝稔" ;
     ns1:小選挙区 "" ;
@@ -257,9 +256,9 @@
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" .
 
-[] ns1:facebook "https://www.facebook.com/kazuyoshi.akaba" ;
+[] ns1:facebook "https://www.facebook.com/akabakazuyoshi" ;
     ns1:twitter "akbhyogo2ku" ;
-    ns1:候補名 "赤羽一嘉" ;
+    ns1:候補名 "赤羽かずよし" ;
     ns1:小選挙区 "兵庫2区" ;
     ns1:比例区 "" .
 
@@ -281,7 +280,7 @@
     ns1:小選挙区 "" ;
     ns1:比例区 "九州" .
 
-[] ns1:facebook "" ;
+[] ns1:facebook "https://www.facebook.com/michiyo.takagi.komei" ;
     ns1:twitter "" ;
     ns1:候補名 "高木美智代" ;
     ns1:小選挙区 "" ;
