--- GrayDB
+++ 幸福公式
@@ -9,658 +9,551 @@
     ns1:候補名 "いき愛子" ;
     ns1:小選挙区 "神奈川3区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "コージャパン(株)" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党神奈川第3選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/profile.php?id=100014235768852" ;
     ns1:twitter "" ;
     ns1:候補名 "いしわた剛" ;
     ns1:小選挙区 "栃木3区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "塾経営",
-        "自動車販売会社員" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党栃木第3選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/tosimitu.egasira" ;
     ns1:twitter "egasiratosimitu" ;
     ns1:候補名 "えがしら俊満" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東海" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "東海比例ブロック副支部長",
-        "航空自衛隊第2術科学校" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "東海比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/profile.php?id=100008449007902" ;
     ns1:twitter "" ;
-    ns1:候補名 "縁田有紀" ;
+    ns1:候補名 "えんだゆき" ;
     ns1:小選挙区 "京都4区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "保育士" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党京都第4選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/miwakohrp" ;
     ns1:twitter "miwako_hrp" ;
     ns1:候補名 "かわい美和子" ;
     ns1:小選挙区 "山口1区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "山口放送興産(株)",
-        "明治安田生命保険相互会社" .
-
-[] ns1:facebook "https://www.facebook.com/kawabekenichiofficial",
-        "https://www.facebook.com/profile.php?id=100002921750076" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党山口第1選挙区支部長" .
+
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100002921750076" ;
     ns1:twitter "kawabekenichi" ;
     ns1:候補名 "かわべ賢一" ;
     ns1:小選挙区 "茨城1区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "エバ・ジャパン(株)",
-        "会社役員" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党茨城県本部代表(兼)茨城第1選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/kouno.ichirou" ;
     ns1:twitter "hrpkonoichiro" ;
     ns1:候補名 "こうの一郎" ;
     ns1:小選挙区 "宮崎2区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
-
-[] ns1:facebook "https://www.facebook.com/ryota.shirotori",
-        "https://www.facebook.com/shirotori.ryota" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党宮崎第2選挙区支部長" .
+
+[] ns1:facebook "https://www.facebook.com/ryota.shirotori" ;
     ns1:twitter "ryotashirotori" ;
     ns1:候補名 "しろとり良太" ;
     ns1:小選挙区 "山形3区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党山形県本部代表(兼)山形県第3支部長" .
 
 [] ns1:facebook "https://www.facebook.com/seii.miho" ;
     ns1:twitter "mihotanii" ;
     ns1:候補名 "せいいみほ" ;
     ns1:小選挙区 "東京14区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "教員",
-        "音楽教室経営" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党東京第14選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/yuji.tanabe.501" ;
     ns1:twitter "tanabe_yuji" ;
     ns1:候補名 "たなべ雄治" ;
     ns1:小選挙区 "岡山2区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)キーエンス" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党岡山県本部代表（兼）岡山第2選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/ai.nishiwaki.hrp" ;
     ns1:twitter "ainishiwaki" ;
     ns1:候補名 "にしわき愛" ;
     ns1:小選挙区 "山梨1区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党山梨第1選挙区支部長" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:候補名 "のはらのりこ" ;
+    ns1:小選挙区 "岐阜1区" ;
+    ns1:比例区 "" ;
+    ns1:立候補 "党発表" ;
     ns1:職歴 "" .
 
-[] ns1:facebook "https://www.facebook.com/norico.nohara" ;
-    ns1:twitter "noharanoriko" ;
-    ns1:候補名 "野原典子" ;
-    ns1:小選挙区 "岐阜1区" ;
-    ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "医療法人野原クリニック理事" .
-
-[] ns1:facebook "https://www.facebook.com/akemi.mitsubachi" ;
-    ns1:twitter "akemimitsubachi" ;
-    ns1:候補名 "三觜明美" ;
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:候補名 "みつはし明美" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "北関東" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)三友",
-        "英語塾講師" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "北関東比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/miw.wld" ;
-    ns1:twitter "minatoyuko" ;
-    ns1:候補名 "湊侑子" ;
+    ns1:twitter "" ;
+    ns1:候補名 "みなと侑子" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "近畿" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "近畿比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/satoshi.miyamoto.52" ;
     ns1:twitter "satoshimiyamoto" ;
     ns1:候補名 "みやもとさとし" ;
     ns1:小選挙区 "石川2区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "金沢大学非常勤講師" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党石川県本部代表代行（兼）石川県第2支部長" .
 
 [] ns1:facebook "https://www.facebook.com/yanaifm" ;
     ns1:twitter "yanaihissho" ;
-    ns1:候補名 "矢内筆勝" ;
+    ns1:候補名 "やない筆勝" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東北" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福実現党総務会長" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "" ;
-    ns1:候補名 "ゆあさようこ" ;
-    ns1:小選挙区 "" ;
-    ns1:比例区 "近畿" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "近畿比例ブロック副支部長" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党総務会長（兼）出版局長" .
 
 [] ns1:facebook "https://www.facebook.com/kazue.yoshitomi" ;
     ns1:twitter "yoshitomi_kazue" ;
-    ns1:候補名 "吉冨和枝" ;
+    ns1:候補名 "よしとみ和枝" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "九州" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "公立小学校教諭" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "九州比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/nanami.hiroko" ;
     ns1:twitter "hiroko_nanami" ;
     ns1:候補名 "七海ひろこ" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東京" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福実現党財務局長",
-        "（株）NTTデータ" .
-
-[] ns1:facebook "https://www.facebook.com/mikuniyuki392" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党財務局長" .
+
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100011203660602" ;
     ns1:twitter "mikuni_yuki" ;
-    ns1:候補名 "三国佑貴" ;
+    ns1:候補名 "三国ゆうき" ;
     ns1:小選挙区 "青森3区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党青森第4選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/profile.php?id=100005729313897" ;
     ns1:twitter "xdauib5hhmwisqv" ;
     ns1:候補名 "上田あつこ" ;
     ns1:小選挙区 "大分2区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)大分銀行" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党大分第2選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/profile.php?id=100008282590315" ;
     ns1:twitter "reikoshimoji" ;
     ns1:候補名 "下地れいこ" ;
     ns1:小選挙区 "沖縄1区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "医療法人青天会下地内科理事" .
-
-[] ns1:facebook "https://www.facebook.com/profile.php?id=100013473602579" ;
-    ns1:twitter "yokunihideyuki" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党沖縄県本部代表（兼）沖縄第1選挙区支部長" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:候補名 "与国秀行" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "中華そば中川社長",
-        "幸福の科学伊勢原支部長" .
-
-[] ns1:facebook "https://www.facebook.com/maki.nakaoka.98",
-        "https://www.facebook.com/nakaoka.maki.97",
-        "https://www.facebook.com/profile.php?id=100014180830459" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "南関東比例ブロック副支部長、幸福の科学伊勢原支部長" .
+
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100014180830459" ;
     ns1:twitter "nakaokamaki" ;
     ns1:候補名 "中岡まき" ;
     ns1:小選挙区 "東京6区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "教員" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党東京第6選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/sagakoufukunakashima" ;
-    ns1:twitter "thinkbig_toru" ;
+    ns1:twitter "toru_nakashima_" ;
     ns1:候補名 "中島とおる" ;
     ns1:小選挙区 "佐賀1区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "住友林業(株)" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党佐賀第1選挙区支部長" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "中島秀雄" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "北海道" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "北海道比例ブロック副支部長",
-        "私立高校保健体育科教諭" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "" ;
-    ns1:候補名 "中村勝" ;
-    ns1:小選挙区 "東京12区" ;
-    ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "北海道比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/nakane.hiromi" ;
     ns1:twitter "nakanehiromi" ;
-    ns1:候補名 "中根裕美" ;
+    ns1:候補名 "中根ひろみ" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東海" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福実現党愛知県本部代表",
-        "私立保育園園長" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党愛知県本部代表（兼）東海比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/nakanishirie.hrp" ;
-    ns1:twitter "rie_nakanishi77" ;
-    ns1:候補名 "中西利恵" ;
+    ns1:twitter "" ;
+    ns1:候補名 "中西りえ" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "四国" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福実現党香川県本部副代表" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "四国比例ブロック支部長" .
 
 [] ns1:facebook "https://www.facebook.com/profile.php?id=100007874922516" ;
     ns1:twitter "" ;
     ns1:候補名 "佐伯知子" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "中国" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "中国比例ブロック副支部長" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党広島県本部副代表" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "佐藤純郎" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東北" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "学習塾非常勤講師",
-        "幸福実現党秋田県本部代表" .
-
-[] ns1:facebook "https://www.facebook.com/koichi.maeda.10" ;
-    ns1:twitter "maedakouichi777" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党秋田県本部代表(兼)東北比例ブロック副支部長" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:候補名 "前田浩一" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "九州" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福の科学南九州本部長" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "九州比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/yukihiko.kanoh" ;
     ns1:twitter "asebora" ;
     ns1:候補名 "加納有輝彦" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東海" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "古田酒造(資)",
-        "国際電信電話(株)",
-        "幸福実現党岐阜県本部代表" .
-
-[] ns1:facebook "https://www.facebook.com/ppp.ggg.ddm.w" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党岐阜県本部代表（兼）東海比例ブロック副支部長" .
+
+[] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "加藤健太" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "中国" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "中国比例ブロック副支部長",
-        "陸上自衛隊自衛官" .
-
-[] ns1:facebook "https://www.facebook.com/Tatsuyoshi.Kato.HS" ;
-    ns1:twitter "tatsuyoshikato" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "中国比例ブロック副支部長" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:候補名 "加藤龍義" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東海" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "東海比例ブロック副支部長",
-        "電力会社勤務",
-        "電機メーカー" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "東海比例ブロック副支部長" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "原伸次" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "北陸信越" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "学習塾経営",
-        "幸福実現党新潟県本部幹事長" .
-
-[] ns1:facebook "https://www.facebook.com/Haraguchi.Miki.HRP",
-        "https://www.facebook.com/profile.php?id=100022300025896" ;
-    ns1:twitter "mikiharaguchi" ;
-    ns1:候補名 "原口実季" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党新潟県本部幹事長（兼）広報部長（兼）北信越比例ブロック副支部長" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:候補名 "原口みき" ;
     ns1:小選挙区 "東京1区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "tw_matayoshi" ;
-    ns1:候補名 "又吉光雄" ;
-    ns1:小選挙区 "東京1区" ;
-    ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "hrpnagano3",
-        "yukiatkanagawa" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党東京第1選挙区支部長" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "yukiatkanagawa" ;
     ns1:候補名 "及川ゆきひさ" ;
     ns1:小選挙区 "長野3区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党外務局長幸福実現党長野県本部代表（兼）長野第3選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/aini.tekinashi.yuzo" ;
     ns1:twitter "yuzofurukawa" ;
-    ns1:候補名 "古川裕三" ;
+    ns1:候補名 "古川ゆうぞう" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" ;
-    ns1:立候補 "確定" ;
+    ns1:立候補 "党発表" ;
     ns1:職歴 "南関東比例ブロック副支部長" .
 
-[] ns1:facebook "https://www.facebook.com/toshimitsu.yoshii.7",
-        "https://www.facebook.com/yoshiitoshimitsu" ;
+[] ns1:facebook "https://www.facebook.com/toshimitsu.yoshii.7" ;
     ns1:twitter "toshi_yoshii777" ;
-    ns1:候補名 "吉井利光" ;
+    ns1:候補名 "吉井としみつ" ;
     ns1:小選挙区 "東京10区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党東京都本部代表（兼）東京第10選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/kaworu.yoshida.9" ;
-    ns1:twitter "kawoyosida" ;
+    ns1:twitter "" ;
     ns1:候補名 "吉田かをる" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "北陸信越" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "富山県立高等学校講師",
-        "幸福実現党富山県本部代表" .
-
-[] ns1:facebook "https://www.facebook.com/profile.php?id=100012396186318" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党富山県本部代表（兼）北信越比例ブロック支部長" .
+
+[] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "国領ぶんた" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "中国" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "宗教法人幸福の科学",
-        "幸福実現党青年局長" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "" ;
-    ns1:候補名 "圷孝行" ;
-    ns1:小選挙区 "神奈川11区" ;
-    ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党青年局長（兼）選対委員長代理（兼）中国比例ブロック支部長" .
 
 [] ns1:facebook "https://www.facebook.com/maki.sakamoto.52" ;
     ns1:twitter "maki1868" ;
     ns1:候補名 "坂本マキ" ;
     ns1:小選挙区 "三重3区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)NTTフィールドテクノ" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党三重県本部代表代行（兼）三重第3選挙区支部長" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "坂本隆一" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東京" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)坂本建設代表取締役社長",
-        "(株)大林組",
-        "幸福実現党東京都本部幹事長" .
-
-[] ns1:facebook "https://www.facebook.com/tsune0024",
-        "https://www.facebook.com/tsuneki.ohnishi" ;
-    ns1:twitter "tsune0024" ;
-    ns1:候補名 "大西恒樹" ;
-    ns1:小選挙区 "神奈川8区" ;
-    ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "amakinaoto" ;
-    ns1:候補名 "天木直人" ;
-    ns1:小選挙区 "東京21区" ;
-    ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
-
-[] ns1:facebook "https://www.facebook.com/tomikawayasumasa",
-        "https://www.facebook.com/yasumasa.tomikawa" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党東京都本部幹事長" .
+
+[] ns1:facebook "https://www.facebook.com/yasumasa.tomikawa" ;
     ns1:twitter "_tomikawa" ;
-    ns1:候補名 "富川泰全" ;
+    ns1:候補名 "富川やすまさ" ;
     ns1:小選挙区 "沖縄4区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党沖縄第4選挙区支部長" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "富川昇" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "九州" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "NTT支店長" .
-
-[] ns1:facebook "https://www.facebook.com/kojima.ichiro",
-        "https://www.facebook.com/幸福幸福実現党小島一郎-幹事長代理-129543174365259" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "九州比例ブロック副支部長" .
+
+[] ns1:facebook "https://www.facebook.com/kojima.ichiro" ;
     ns1:twitter "kojimaichiro" ;
     ns1:候補名 "小島一郎" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "北関東" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福実現党総務会部長" .
-
-[] ns1:facebook "https://www.facebook.com/skkokumi" ;
-    ns1:twitter "" ;
-    ns1:候補名 "小泉修平" ;
-    ns1:小選挙区 "大阪1区" ;
-    ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党総務会部長(兼)北関東比例ブロック支部長" .
 
 [] ns1:facebook "https://www.facebook.com/keigo.kazumori" ;
     ns1:twitter "kazumokeigo" ;
-    ns1:候補名 "数森圭吾" ;
+    ns1:候補名 "数森けいご" ;
     ns1:小選挙区 "大阪5区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "JFE商事株式会社" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "" ;
-    ns1:候補名 "斉藤芳男" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党大阪第5選挙区支部長" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:候補名 "斎藤芳男" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "北関東" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)志村産業役員",
-        "パイオニア(株)",
-        "幸福実現党埼玉県本部幹事長" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "zengakuren" ;
-    ns1:候補名 "斎藤郁真" ;
-    ns1:小選挙区 "東京8区" ;
-    ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党埼玉県本部幹事長(兼)北関東比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/profile.php?id=100008594358938" ;
     ns1:twitter "" ;
     ns1:候補名 "木下じゅん子" ;
     ns1:小選挙区 "熊本2区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)花プラス道の駅おおむた" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "" ;
-    ns1:候補名 "松島弘典" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党熊本第2選挙区支部長" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:候補名 "松島ひろのり" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "近畿" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "三菱商事(株)",
-        "宗教法人幸福の科学",
-        "幸福実現党幹事長" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党幹事長（兼）選対委員長（兼）近畿比例ブロック支部長" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "桃田妙子" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "四国" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福の科学・高知支部上級支部長" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "" ;
-    ns1:候補名 "森国英和" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "四国比例ブロック副支部長" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:候補名 "森國英和" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東海" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福実現党党首特別補佐" .
-
-[] ns1:facebook "https://www.facebook.com/moriyama.hokkaido",
-        "https://www.facebook.com/yoshinori.moriyama.7509" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党党首特別補佐（兼）東海比例ブロック支部長" .
+
+[] ns1:facebook "https://www.facebook.com/yoshinori.moriyama.7509" ;
     ns1:twitter "yosshiimoriyama" ;
-    ns1:候補名 "森山佳則" ;
+    ns1:候補名 "森山よしのり" ;
     ns1:小選挙区 "北海道5区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "新日本製鐵(株)" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党北海道第5選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/morita.kojihrp" ;
     ns1:twitter "" ;
     ns1:候補名 "森田こうじ" ;
     ns1:小選挙区 "愛媛3区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "古川製作所（株）" .
-
-[] ns1:facebook "https://www.facebook.com/MoritaTakayuki777" ;
-    ns1:twitter "paul_morimori" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党愛媛第3選挙区支部長" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:候補名 "森田貴行" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "北関東" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福の科学高崎支部長" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "北関東比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/yoshio.soccer.yoshi" ;
     ns1:twitter "yoshio7_m" ;
     ns1:候補名 "水野善丈" ;
     ns1:小選挙区 "広島2区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党広島県本部代表代行（兼）広島第2選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/Enatsu.Masatoshi.HR" ;
     ns1:twitter "enatsumasatoshi" ;
     ns1:候補名 "江夏正敏" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "九州" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "HS政経塾塾長",
-        "九州比例ブロック支部長",
-        "宗教法人幸福の科学",
-        "幸福実現党政務調査会長" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党政務調査会長（兼）HS政経塾塾長（兼）九州比例ブロック支部長" .
 
 [] ns1:facebook "https://www.facebook.com/tetsushi.yui" ;
     ns1:twitter "yuitetsushi" ;
     ns1:候補名 "油井哲史" ;
     ns1:小選挙区 "宮城1区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)DGコミュニケーションズ" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党宮城県本部代表(兼)宮城第1選挙区支部長" .
+
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
+    ns1:候補名 "油淺頌子" ;
+    ns1:小選挙区 "" ;
+    ns1:比例区 "近畿" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "近畿比例ブロック副支部長" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "泉章子" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "近畿" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "市立小学校校長",
-        "近畿比例ブロック副支部長" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "" ;
-    ns1:候補名 "犬丸光加" ;
-    ns1:小選挙区 "東京1区" ;
-    ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "近畿比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/jitsugen.takako" ;
     ns1:twitter "" ;
-    ns1:候補名 "田中孝子" ;
+    ns1:候補名 "田中たかこ" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "近畿" ;
-    ns1:立候補 "確定" ;
+    ns1:立候補 "党発表" ;
     ns1:職歴 "近畿比例ブロック副支部長" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
-    ns1:候補名 "白川康之" ;
+    ns1:候補名 "白川やすゆき" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "北陸信越" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福実現党福井県本部代表",
-        "建設業(自営)" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党福井県本部代表（兼）北信越比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/profile.php?id=100005122571380" ;
     ns1:twitter "" ;
-    ns1:候補名 "石川幹子" ;
+    ns1:候補名 "石川みき子" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東北" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)シダックス",
-        "東北比例ブロック副支部長",
-        "社会福祉法人本宮会なかの保育園" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "東北比例ブロック副支部長" .
 
 [] ns1:facebook "" ;
     ns1:twitter "realjimmu_hrp" ;
     ns1:候補名 "神武桜子" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福実現党副党首" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党副党首(兼)広報本部長" .
 
 [] ns1:facebook "https://www.facebook.com/masatoshi.fukuyama.1" ;
     ns1:twitter "seitikawashima" ;
-    ns1:候補名 "福山正敏" ;
+    ns1:候補名 "福山まさとし" ;
     ns1:小選挙区 "徳島2区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)クボタ" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党徳島県本部代表代行（兼）徳島第2選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/kasahara.reika" ;
     ns1:twitter "kasahara_reika" ;
     ns1:候補名 "笠原れいか" ;
     ns1:小選挙区 "新潟5区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党新潟第5選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/profile.php?id=100005211330715" ;
     ns1:twitter "mkynet1975" ;
     ns1:候補名 "荒川まさし" ;
     ns1:小選挙区 "滋賀2区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
-
-[] ns1:facebook "https://www.facebook.com/nishihara.tada" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党滋賀第2選挙区支部長" .
+
+[] ns1:facebook "https://www.facebook.com/profile.php?id=100013230707179" ;
     ns1:twitter "nishihara_tada" ;
     ns1:候補名 "西原ただひろ" ;
     ns1:小選挙区 "福岡6区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "バイエル薬品(株)" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党福岡県本部総務会長（兼）福岡第6選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/Atsushi.Nishimoto7" ;
-    ns1:twitter "nishimotoatsush" ;
-    ns1:候補名 "西本篤" ;
+    ns1:twitter "" ;
+    ns1:候補名 "西本あつし" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "近畿" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党和歌山県本部代表（兼）近畿比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/shakuryoko" ;
     ns1:twitter "shakuryoko" ;
     ns1:候補名 "釈量子" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東京" ;
-    ns1:立候補 "確定" ;
+    ns1:立候補 "党発表" ;
     ns1:職歴 "幸福実現党党首" .
 
 [] ns1:facebook "" ;
@@ -668,68 +561,54 @@
     ns1:候補名 "野市裕司" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "北海道" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)みのり",
-        "幸福実現党北海道本部代表" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党北海道本部代表(兼)北海道比例ブロック支部長" .
 
 [] ns1:facebook "https://www.facebook.com/masateru.nomura" ;
     ns1:twitter "nomuramasateru" ;
     ns1:候補名 "野村昌央" ;
     ns1:小選挙区 "広島3区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党広島第3選挙区支部長" .
 
 [] ns1:facebook "https://www.facebook.com/elmichiko" ;
     ns1:twitter "kinjou_tatsurou" ;
     ns1:候補名 "金城タツロー" ;
     ns1:小選挙区 "沖縄3区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "沖縄気象台" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党沖縄第3選挙区支部長" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "鈴木豪" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "東京" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "幸福実現党東京都本部青年局長" .
-
-[] ns1:facebook "https://www.facebook.com/profile.php?id=100017183265706" ;
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福の科学理事(兼)青年局長" .
+
+[] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "長村康博" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "三井生命保険(株)",
-        "南関東比例ブロック副支部長",
-        "神奈川県信用保証協会",
-        "鉄鋼メーカー商社" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "南関東比例ブロック副支部長" .
 
 [] ns1:facebook "https://www.facebook.com/takeshi.iida.90" ;
     ns1:twitter "" ;
     ns1:候補名 "飯田たけし" ;
     ns1:小選挙区 "埼玉3区" ;
     ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "会社役員" .
-
-[] ns1:facebook "" ;
-    ns1:twitter "" ;
-    ns1:候補名 "駒村幸成" ;
-    ns1:小選挙区 "長野1区" ;
-    ns1:比例区 "" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "" .
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "幸福実現党埼玉第3選挙区支部長" .
 
 [] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "高橋彰" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" ;
-    ns1:立候補 "確定" ;
-    ns1:職歴 "(株)タカヒマラ・ユニバース代表取締役",
-        "南関東比例ブロック副支部長",
-        "旭化成工業(株)" .
-
+    ns1:立候補 "党発表" ;
+    ns1:職歴 "南関東比例ブロック副支部長" .
+
