--- GrayDB
+++ 政治ナビ
@@ -4,13 +4,25 @@
 @prefix xml: <http://www.w3.org/XML/1998/namespace> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
+[] ns1:twitter "abetoshiko" ;
+    ns1:wikidata "Q7827692" ;
+    ns1:名前 "あべ俊子" ;
+    ns1:性別 "女性" ;
+    ns1:生年月日 "1959-05-19" .
+
+[] ns1:twitter "naomi_tokashiki" ;
+    ns1:wikidata "Q11561908" ;
+    ns1:名前 "とかしきなおみ" ;
+    ns1:性別 "女性" ;
+    ns1:生年月日 "1962-07-16" .
+
 [] ns1:twitter "hiromitsubayasi" ;
     ns1:wikidata "Q11353993" ;
-    ns1:名前 "三ッ林裕巳" ;
-    ns1:性別 "女性" ;
+    ns1:名前 "三ツ林裕巳" ;
+    ns1:性別 "男性" ;
     ns1:生年月日 "1955-09-07" .
 
-[] ns1:twitter "mitsuya_office" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q7051315" ;
     ns1:名前 "三ツ矢憲生" ;
     ns1:性別 "男性" ;
@@ -413,6 +425,12 @@
     ns1:生年月日 "1955-11-22" .
 
 [] ns1:twitter "" ;
+    ns1:wikidata "Q11399159" ;
+    ns1:名前 "加藤寛治" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1946-04-24" .
+
+[] ns1:twitter "" ;
     ns1:wikidata "Q18701018" ;
     ns1:名前 "加藤鮎子" ;
     ns1:性別 "女性" ;
@@ -428,7 +446,7 @@
     ns1:wikidata "Q7505129" ;
     ns1:名前 "務台俊介" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1956-01-01" .
+    ns1:生年月日 "1956-07-03" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q11400147" ;
@@ -544,11 +562,23 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1972-07-14" .
 
+[] ns1:twitter "yoshikawahajime" ;
+    ns1:wikidata "Q11412937" ;
+    ns1:名前 "吉川元" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1966-09-28" .
+
 [] ns1:twitter "" ;
     ns1:wikidata "Q7677156" ;
     ns1:名前 "吉川貴盛" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1950-01-01" .
+    ns1:生年月日 "1950-10-20" .
+
+[] ns1:twitter "" ;
+    ns1:wikidata "Q21703015" ;
+    ns1:名前 "吉田宣弘" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1967-12-08" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q6102348" ;
@@ -556,7 +586,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1949-02-26" .
 
-[] ns1:twitter "toyo23office" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q21655857" ;
     ns1:名前 "吉田豊史" ;
     ns1:性別 "男性" ;
@@ -580,21 +610,15 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1971-10-10" .
 
-[] ns1:twitter "" ;
+[] ns1:twitter "kokubkonosuke" ;
+    ns1:wikidata "Q11422694" ;
+    ns1:名前 "国場幸之助" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1973-01-10" .
+
+[] ns1:twitter "torukunishige" ;
     ns1:wikidata "Q11422753" ;
     ns1:名前 "国重徹" ;
-    ns1:性別 "" ;
-    ns1:生年月日 "" .
-
-[] ns1:twitter "kokubkonosuke" ;
-    ns1:wikidata "Q11422694" ;
-    ns1:名前 "國場幸之助" ;
-    ns1:性別 "男性" ;
-    ns1:生年月日 "1973-01-10" .
-
-[] ns1:twitter "torukunishige" ;
-    ns1:wikidata "Q11422753" ;
-    ns1:名前 "國重徹" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1974-11-23" .
 
@@ -722,7 +746,7 @@
     ns1:wikidata "Q4817826" ;
     ns1:名前 "大島敦" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1956-01-01" .
+    ns1:生年月日 "1956-12-21" .
 
 [] ns1:twitter "tadamori_oshima" ;
     ns1:wikidata "Q49998" ;
@@ -760,6 +784,12 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1946-08-28" .
 
+[] ns1:twitter "ohmisei" ;
+    ns1:wikidata "Q11438945" ;
+    ns1:名前 "大見正" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1958-06-20" .
+
 [] ns1:twitter "" ;
     ns1:wikidata "Q11439619" ;
     ns1:名前 "大野敬太郎" ;
@@ -778,7 +808,7 @@
     ns1:性別 "女性" ;
     ns1:生年月日 "1979-08-28" .
 
-[] ns1:twitter "teamohta" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q337406" ;
     ns1:名前 "太田昭宏" ;
     ns1:性別 "男性" ;
@@ -864,7 +894,7 @@
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q11455005" ;
-    ns1:名前 "宮澤博行" ;
+    ns1:名前 "宮沢博行" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1975-01-10" .
 
@@ -872,13 +902,13 @@
     ns1:wikidata "Q1656896" ;
     ns1:名前 "宮腰光寛" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1950-01-01" .
+    ns1:生年月日 "1950-12-21" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q21655823" ;
     ns1:名前 "宮路拓馬" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1979-12-6" .
+    ns1:生年月日 "1979-12-06" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q7496423" ;
@@ -890,7 +920,7 @@
     ns1:wikidata "Q6746585" ;
     ns1:名前 "寺田学" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1971-01-01" .
+    ns1:生年月日 "1976-09-20" .
 
 [] ns1:twitter "terada124" ;
     ns1:wikidata "Q6869306" ;
@@ -900,7 +930,7 @@
 
 [] ns1:twitter "masanobu_ogura" ;
     ns1:wikidata "Q11458942" ;
-    ns1:名前 "小倉將信" ;
+    ns1:名前 "小倉将信" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1981-05-30" .
 
@@ -916,11 +946,17 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1975-12-26" .
 
+[] ns1:twitter "" ;
+    ns1:wikidata "Q11460163" ;
+    ns1:名前 "小島敏文" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1950-09-07" .
+
 [] ns1:twitter "junyaog" ;
     ns1:wikidata "Q6314413" ;
     ns1:名前 "小川淳也" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1971-01-01" .
+    ns1:生年月日 "1971-04-18" .
 
 [] ns1:twitter "komatsu_yutaka" ;
     ns1:wikidata "Q11461246" ;
@@ -984,7 +1020,7 @@
 
 [] ns1:twitter "odawarakiyoshi" ;
     ns1:wikidata "Q11462993" ;
-    ns1:名前 "小田原潔" ;
+    ns1:名前 "小田原きよし" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1964-05-23" .
 
@@ -1054,6 +1090,12 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1948-08-08" .
 
+[] ns1:twitter "yamamoto_taku" ;
+    ns1:wikidata "Q7678543" ;
+    ns1:名前 "山本拓" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1952-07-07" .
+
 [] ns1:twitter "" ;
     ns1:wikidata "Q8060529" ;
     ns1:名前 "山本有二" ;
@@ -1108,17 +1150,17 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1953-07-14" .
 
-[] ns1:twitter "takeshi108" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q1654639" ;
     ns1:名前 "岩屋毅" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1957-08-24" .
 
-[] ns1:twitter "" ;
+[] ns1:twitter "kazuchika_iwata" ;
     ns1:wikidata "Q11475456" ;
     ns1:名前 "岩田和親" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1973-9-20" .
+    ns1:生年月日 "1973-09-20" .
 
 [] ns1:twitter "kishinobuo" ;
     ns1:wikidata "Q1995125" ;
@@ -1142,7 +1184,7 @@
     ns1:wikidata "Q18817650" ;
     ns1:名前 "島津幸広" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1956-10-19" .
+    ns1:生年月日 "1956-10-01" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q17159494" ;
@@ -1176,7 +1218,7 @@
 
 [] ns1:twitter "hiratakuchan" ;
     ns1:wikidata "Q7678622" ;
-    ns1:名前 "平井卓也" ;
+    ns1:名前 "平井たくや" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1958-01-25" .
 
@@ -1184,7 +1226,7 @@
     ns1:wikidata "Q5770815" ;
     ns1:名前 "平口洋" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1948-01-01" .
+    ns1:生年月日 "1948-08-01" .
 
 [] ns1:twitter "tairamasaaki" ;
     ns1:wikidata "Q6782264" ;
@@ -1210,7 +1252,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1949-03-19" .
 
-[] ns1:twitter "gotodamasazumi" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q6782819" ;
     ns1:名前 "後藤田正純" ;
     ns1:性別 "男性" ;
@@ -1232,7 +1274,7 @@
     ns1:wikidata "Q49988" ;
     ns1:名前 "御法川信英" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1964-01-01" .
+    ns1:生年月日 "1964-05-25" .
 
 [] ns1:twitter "shiikazuo" ;
     ns1:wikidata "Q337507" ;
@@ -1252,6 +1294,12 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1952-02-05" .
 
+[] ns1:twitter "saitou_ken" ;
+    ns1:wikidata "Q11679075" ;
+    ns1:名前 "斎藤健" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1959-06-14" .
+
 [] ns1:twitter "saito_hiroaki_" ;
     ns1:wikidata "Q17158549" ;
     ns1:名前 "斎藤洋明" ;
@@ -1304,13 +1352,13 @@
     ns1:wikidata "Q11518220" ;
     ns1:名前 "木原稔" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1969-01-01" .
+    ns1:生年月日 "1969-08-12" .
 
 [] ns1:twitter "kihara_seiji" ;
     ns1:wikidata "Q11518222" ;
     ns1:名前 "木原誠二" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1970-01-01" .
+    ns1:生年月日 "1970-06-08" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q20039675" ;
@@ -1406,7 +1454,7 @@
     ns1:wikidata "Q5770251" ;
     ns1:名前 "松野博一" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1962-01-01" .
+    ns1:生年月日 "1962-09-13" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q8055323" ;
@@ -1426,7 +1474,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1964-05-31" .
 
-[] ns1:twitter "yunoki_m" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q6837954" ;
     ns1:名前 "柚木道義" ;
     ns1:性別 "男性" ;
@@ -1460,7 +1508,7 @@
     ns1:wikidata "Q8056290" ;
     ns1:名前 "桜田義孝" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1949-01-01" .
+    ns1:生年月日 "1949-12-20" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q6383605" ;
@@ -1478,13 +1526,19 @@
     ns1:wikidata "Q5770884" ;
     ns1:名前 "梶山弘志" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1955-01-01" .
+    ns1:生年月日 "1955-10-18" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q1340817" ;
     ns1:名前 "棚橋泰文" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1963-02-11" .
+
+[] ns1:twitter "" ;
+    ns1:wikidata "Q337762" ;
+    ns1:名前 "森山裕" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1945-04-08" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q1133194" ;
@@ -1648,7 +1702,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1963-03-11" .
 
-[] ns1:twitter "takeokawamura" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q382311" ;
     ns1:名前 "河村建夫" ;
     ns1:性別 "男性" ;
@@ -1660,11 +1714,11 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1963-01-10" .
 
-[] ns1:twitter "mkawano2416" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q11554229" ;
     ns1:名前 "河野正美" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1961-6-30" .
+    ns1:生年月日 "1961-06-30" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q6391984" ;
@@ -1672,11 +1726,17 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1974-07-29" .
 
-[] ns1:twitter "tsumura_keisuke" ;
+[] ns1:twitter "tsushimajun" ;
+    ns1:wikidata "Q11556075" ;
+    ns1:名前 "津島淳" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1966-10-18" .
+
+[] ns1:twitter "" ;
     ns1:wikidata "Q6383911" ;
     ns1:名前 "津村啓介" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1971-01-01" .
+    ns1:生年月日 "1971-10-27" .
 
 [] ns1:twitter "asao_keiichiro" ;
     ns1:wikidata "Q1737907" ;
@@ -1684,6 +1744,12 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1964-02-11" .
 
+[] ns1:twitter "hamachi_m" ;
+    ns1:wikidata "Q17220479" ;
+    ns1:名前 "浜地雅一" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1970-05-08" .
+
 [] ns1:twitter "" ;
     ns1:wikidata "Q1132836" ;
     ns1:名前 "浜田靖一" ;
@@ -1698,21 +1764,15 @@
 
 [] ns1:twitter "tomo_ukishima" ;
     ns1:wikidata "Q7820173" ;
-    ns1:名前 "浮島智子" ;
+    ns1:名前 "浮島とも子" ;
     ns1:性別 "女性" ;
     ns1:生年月日 "1963-02-01" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q11560960" ;
-    ns1:名前 "清水忠史" ;
+    ns1:名前 "清水ただし" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1968-05-05" .
-
-[] ns1:twitter "naomi_tokashiki" ;
-    ns1:wikidata "Q11561908" ;
-    ns1:名前 "渡嘉敷奈緒美" ;
-    ns1:性別 "女性" ;
-    ns1:生年月日 "1962-07-16" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q1133136" ;
@@ -1772,7 +1832,7 @@
     ns1:wikidata "Q5752395" ;
     ns1:名前 "牧原秀樹" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1971-01-01" .
+    ns1:生年月日 "1971-06-04" .
 
 [] ns1:twitter "makishimakaren" ;
     ns1:wikidata "Q11570409" ;
@@ -1838,7 +1898,7 @@
     ns1:wikidata "Q6360573" ;
     ns1:名前 "田嶋要" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1961-01-01" .
+    ns1:生年月日 "1961-09-22" .
 
 [] ns1:twitter "tadokoro_yoshi" ;
     ns1:wikidata "Q11576787" ;
@@ -1852,7 +1912,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1964-12-15" .
 
-[] ns1:twitter "tamuratakaaki" ;
+[] ns1:twitter "@tamuratakaaki" ;
     ns1:wikidata "Q18817798" ;
     ns1:名前 "田村貴昭" ;
     ns1:性別 "男性" ;
@@ -1880,9 +1940,9 @@
     ns1:wikidata "Q11578505" ;
     ns1:名前 "畑野君枝" ;
     ns1:性別 "女性" ;
-    ns1:生年月日 "1957-01-28" .
-
-[] ns1:twitter "hatayama_kazuya" ;
+    ns1:生年月日 "1957-01-19" .
+
+[] ns1:twitter "" ;
     ns1:wikidata "Q18817972" ;
     ns1:名前 "畠山和也" ;
     ns1:性別 "男性" ;
@@ -1946,7 +2006,7 @@
     ns1:wikidata "Q6782738" ;
     ns1:名前 "石田真敏" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1952-01-01" .
+    ns1:生年月日 "1952-04-11" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q7051352" ;
@@ -1966,6 +2026,12 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1972-02-18" .
 
+[] ns1:twitter "" ;
+    ns1:wikidata "Q11589528" ;
+    ns1:名前 "神山佐市" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1954-08-17" .
+
 [] ns1:twitter "kamiyama_yosuke" ;
     ns1:wikidata "Q11589542" ;
     ns1:名前 "神山洋介" ;
@@ -1988,7 +2054,7 @@
     ns1:wikidata "Q7705291" ;
     ns1:名前 "福井照" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1953-01-01" .
+    ns1:生年月日 "1953-12-14" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q11592023" ;
@@ -2006,7 +2072,7 @@
     ns1:wikidata "Q6864590" ;
     ns1:名前 "福田峰之" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1964-01-01" .
+    ns1:生年月日 "1964-04-08" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q3435725" ;
@@ -2014,7 +2080,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1948-04-17" .
 
-[] ns1:twitter "tatsuofukuda" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q2506910" ;
     ns1:名前 "福田達夫" ;
     ns1:性別 "男性" ;
@@ -2038,7 +2104,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1962-07-03" .
 
-[] ns1:twitter "inatsuhisashi" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q11596723" ;
     ns1:名前 "稲津久" ;
     ns1:性別 "男性" ;
@@ -2068,6 +2134,12 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1946-11-03" .
 
+[] ns1:twitter "takeuchi_yuzuru" ;
+    ns1:wikidata "Q11598974" ;
+    ns1:名前 "竹内譲" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1958-06-25" .
+
 [] ns1:twitter "naokazutakemoto" ;
     ns1:wikidata "Q614464" ;
     ns1:名前 "竹本直一" ;
@@ -2096,7 +2168,13 @@
     ns1:wikidata "Q7677369" ;
     ns1:名前 "篠原孝" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1948-01-01" .
+    ns1:生年月日 "1948-07-17" .
+
+[] ns1:twitter "" ;
+    ns1:wikidata "Q20040516" ;
+    ns1:名前 "篠原豪" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1975-02-12" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q11603860" ;
@@ -2138,13 +2216,13 @@
     ns1:wikidata "Q5639959" ;
     ns1:名前 "船田元" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1953-01-01" .
+    ns1:生年月日 "1953-11-22" .
 
 [] ns1:twitter "wakamiya7788" ;
     ns1:wikidata "Q6389383" ;
     ns1:名前 "若宮健嗣" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1961-01-01" .
+    ns1:生年月日 "1961-09-02" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q11616618" ;
@@ -2158,7 +2236,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1955-10-07" .
 
-[] ns1:twitter "satoshisatoshia" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q1478311" ;
     ns1:名前 "荒井聰" ;
     ns1:性別 "男性" ;
@@ -2182,7 +2260,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1946-10-10" .
 
-[] ns1:twitter "sugawitter" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q122465" ;
     ns1:名前 "菅義偉" ;
     ns1:性別 "男性" ;
@@ -2210,7 +2288,7 @@
     ns1:wikidata "Q8049977" ;
     ns1:名前 "葉梨康弘" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1959-01-01" .
+    ns1:生年月日 "1959-10-12" .
 
 [] ns1:twitter "sonourakentaro" ;
     ns1:wikidata "Q1965622" ;
@@ -2340,7 +2418,7 @@
 
 [] ns1:twitter "ryosei_akazawa" ;
     ns1:wikidata "Q7385320" ;
-    ns1:名前 "赤澤亮正" ;
+    ns1:名前 "赤沢亮正" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1960-12-18" .
 
@@ -2348,7 +2426,7 @@
     ns1:wikidata "Q6381788" ;
     ns1:名前 "赤羽一嘉" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1958-01-01" .
+    ns1:生年月日 "1958-05-07" .
 
 [] ns1:twitter "akama_jiro" ;
     ns1:wikidata "Q6203082" ;
@@ -2364,7 +2442,7 @@
 
 [] ns1:twitter "adachiyasushi" ;
     ns1:wikidata "Q11637002" ;
-    ns1:名前 "足立康史" ;
+    ns1:名前 "足立やすし" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1965-10-14" .
 
@@ -2374,7 +2452,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1962-02-04" .
 
-[] ns1:twitter "tsujimotokiyomi" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q405533" ;
     ns1:名前 "辻元清美" ;
     ns1:性別 "女性" ;
@@ -2392,7 +2470,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1958-05-26" .
 
-[] ns1:twitter "kondoyosuke21" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q49943" ;
     ns1:名前 "近藤洋介" ;
     ns1:性別 "男性" ;
@@ -2420,7 +2498,7 @@
     ns1:wikidata "Q7827620" ;
     ns1:名前 "遠藤利明" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1950-01-01" .
+    ns1:生年月日 "1950-01-17" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q11642258" ;
@@ -2440,7 +2518,7 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1976-11-17" .
 
-[] ns1:twitter "nodayoshi55" ;
+[] ns1:twitter "" ;
     ns1:wikidata "Q57647" ;
     ns1:名前 "野田佳彦" ;
     ns1:性別 "男性" ;
@@ -2482,7 +2560,8 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1961-02-27" .
 
-[] ns1:twitter "" ;
+[] ns1:twitter "",
+        "kanekomegu" ;
     ns1:wikidata "Q11646784",
         "Q5371065" ;
     ns1:名前 "金子恵美" ;
@@ -2592,12 +2671,6 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1965-06-07" .
 
-[] ns1:twitter "abetoshiko" ;
-    ns1:wikidata "Q7827692" ;
-    ns1:名前 "阿部俊子" ;
-    ns1:性別 "女性" ;
-    ns1:生年月日 "1959-05-19" .
-
 [] ns1:twitter "abe_tomoko" ;
     ns1:wikidata "Q516232" ;
     ns1:名前 "阿部知子" ;
@@ -2608,7 +2681,7 @@
     ns1:wikidata "Q49875" ;
     ns1:名前 "階猛" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1966-01-01" .
+    ns1:生年月日 "1966-10-07" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q11661236" ;
@@ -2660,7 +2733,7 @@
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q11670038" ;
-    ns1:名前 "高木宏壽" ;
+    ns1:名前 "高木宏寿" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1960-04-09" .
 
@@ -2668,7 +2741,7 @@
     ns1:wikidata "Q7850389" ;
     ns1:名前 "高木毅" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1956-01-01" .
+    ns1:生年月日 "1956-01-16" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q6837950" ;
@@ -2693,6 +2766,12 @@
     ns1:名前 "高村正彦" ;
     ns1:性別 "男性" ;
     ns1:生年月日 "1942-03-15" .
+
+[] ns1:twitter "" ;
+    ns1:wikidata "Q11671375" ;
+    ns1:名前 "高橋ひなこ" ;
+    ns1:性別 "女性" ;
+    ns1:生年月日 "1958-01-19" .
 
 [] ns1:twitter "chiduko916" ;
     ns1:wikidata "Q5102671" ;
@@ -2700,23 +2779,23 @@
     ns1:性別 "女性" ;
     ns1:生年月日 "1959-09-16" .
 
-[] ns1:twitter "" ;
-    ns1:wikidata "Q11671375" ;
-    ns1:名前 "高橋比奈子" ;
-    ns1:性別 "女性" ;
-    ns1:生年月日 "1958-01-19" .
-
 [] ns1:twitter "takatorishuichi" ;
     ns1:wikidata "Q7504756" ;
     ns1:名前 "高鳥修一" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1960-01-01" .
+    ns1:生年月日 "1960-09-29" .
+
+[] ns1:twitter "" ;
+    ns1:wikidata "Q11673487" ;
+    ns1:名前 "鬼木誠" ;
+    ns1:性別 "男性" ;
+    ns1:生年月日 "1972-10-16" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q17219387" ;
     ns1:名前 "鳩山二郎" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1979-01-13" .
+    ns1:生年月日 "1979-01-01" .
 
 [] ns1:twitter "" ;
     ns1:wikidata "Q730991" ;
@@ -2728,7 +2807,7 @@
     ns1:wikidata "Q5349200" ;
     ns1:名前 "鷲尾英一郎" ;
     ns1:性別 "男性" ;
-    ns1:生年月日 "1977-01-01" .
+    ns1:生年月日 "1977-01-03" .
 
 [] ns1:twitter "taro_aso_" ;
     ns1:wikidata "Q132596" ;
@@ -2754,9 +2833,3 @@
     ns1:性別 "男性" ;
     ns1:生年月日 "1966-10-13" .
 
-[] ns1:twitter "saitou_ken" ;
-    ns1:wikidata "Q11679075" ;
-    ns1:名前 "齋藤健" ;
-    ns1:性別 "男性" ;
-    ns1:生年月日 "1959-06-14" .
-
