--- GrayDB
+++ 公明党公式
@@ -6,7 +6,7 @@
 
 [] ns1:facebook "https://www.facebook.com/hamachi.masakazu" ;
     ns1:twitter "hamachi_m" ;
-    ns1:候補名 "浜地雅一" ;
+    ns1:候補名 "濵地雅一" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "九州" .
 
@@ -16,8 +16,7 @@
     ns1:小選挙区 "大阪6区" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/inatsu.komei",
-        "https://www.facebook.com/inatsu.news" ;
+[] ns1:facebook "https://www.facebook.com/inatsu.komei" ;
     ns1:twitter "inatsuhisashi" ;
     ns1:候補名 "いなつ久" ;
     ns1:小選挙区 "北海道10区" ;
@@ -42,7 +41,7 @@
     ns1:比例区 "東海" .
 
 [] ns1:facebook "https://www.facebook.com/nakanohiromasa.office" ;
-    ns1:twitter "hiro_nakano_" ;
+    ns1:twitter "" ;
     ns1:候補名 "中野ひろまさ" ;
     ns1:小選挙区 "兵庫8区" ;
     ns1:比例区 "" .
@@ -53,8 +52,8 @@
     ns1:小選挙区 "" ;
     ns1:比例区 "東北" .
 
-[] ns1:facebook "https://www.facebook.com/yasuo.ikawa.1" ;
-    ns1:twitter "yamanashi_yasuo" ;
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:候補名 "井川泰雄" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" .
@@ -95,8 +94,7 @@
     ns1:小選挙区 "大阪16区" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/noriko.furuya.jp",
-        "https://www.facebook.com/noriko.furuya514" ;
+[] ns1:facebook "https://www.facebook.com/noriko.furuya.jp" ;
     ns1:twitter "noriko_furuya" ;
     ns1:候補名 "古屋範子" ;
     ns1:小選挙区 "" ;
@@ -132,8 +130,8 @@
     ns1:小選挙区 "東京12区" ;
     ns1:比例区 "" .
 
-[] ns1:facebook "https://www.facebook.com/ota.masataka" ;
-    ns1:twitter "ootamasataka" ;
+[] ns1:facebook "" ;
+    ns1:twitter "" ;
     ns1:候補名 "太田昌孝" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "北陸信越" .
@@ -151,7 +149,7 @@
     ns1:比例区 "北関東" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "papiya396" ;
+    ns1:twitter "" ;
     ns1:候補名 "布施正行" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" .
@@ -258,10 +256,9 @@
     ns1:小選挙区 "" ;
     ns1:比例区 "南関東" .
 
-[] ns1:facebook "https://www.facebook.com/akabakazuyoshi",
-        "https://www.facebook.com/kazuyoshi.akaba" ;
+[] ns1:facebook "https://www.facebook.com/akabakazuyoshi" ;
     ns1:twitter "akbhyogo2ku" ;
-    ns1:候補名 "赤羽一嘉" ;
+    ns1:候補名 "赤羽かずよし" ;
     ns1:小選挙区 "兵庫2区" ;
     ns1:比例区 "" .
 
@@ -295,7 +292,7 @@
     ns1:小選挙区 "" ;
     ns1:比例区 "東京" .
 
-[] ns1:facebook "https://www.facebook.com/muneo.takamura" ;
+[] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "高村宗男" ;
     ns1:小選挙区 "" ;
@@ -314,12 +311,12 @@
     ns1:比例区 "近畿" .
 
 [] ns1:facebook "" ;
-    ns1:twitter "wanibuchiyoko" ;
+    ns1:twitter "" ;
     ns1:候補名 "鰐淵洋子" ;
     ns1:小選挙区 "" ;
     ns1:比例区 "近畿" .
 
-[] ns1:facebook "https://www.facebook.com/hideaki.washioka" ;
+[] ns1:facebook "" ;
     ns1:twitter "" ;
     ns1:候補名 "鷲岡秀明" ;
     ns1:小選挙区 "" ;
